<!DOCTYPE html><html lang="en"><head><title>Static News</title><meta charSet="utf-8"/><meta name="description" content="Static delayed Hacker News."/><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"/><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1d1f21"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><meta name="application-name" content="Static News"/><meta name="apple-mobile-web-app-title" content="Static News"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="#1d1f21"/><link rel="preload" href="styles.css?v=1692954061573" as="style"/><link rel="stylesheet" href="styles.css?v=1692954061573"/></head><body><div id="container"><div id="inner"><header><a href="/">Static News</a><a href="/about">about</a></header><div id="content"><div><div id="title"><a href="https://www.nature.com/articles/s41586-023-06457-y">The human Y chromosome has been completely sequenced</a> <span class="domain">(<a href="https://www.nature.com">www.nature.com</a>)</span></div><div class="subtext"><span>birriel</span> | <span>145 comments</span></div><br/><div><div id="37257371" class="c"><input type="checkbox" id="c-37257371" checked=""/><div class="controls bullet"><span class="by">10g1k</span><span>|</span><a href="#37257407">next</a><span>|</span><label class="collapse" for="c-37257371">[-]</label><label class="expand" for="c-37257371">[28 more]</label></div><br/><div class="children"><div class="content">For those who don&#x27;t recall: Back in the Dark Ages, there was a race to decode the human genome.  The leading competitiors (wealthiest) were Celera Genomics and the Human Genome Project.  After some time, Celera (headed by Craig Venter), announced they had done the deed.  However, what Celera had actually done was used what they called a &quot;shotgun method&quot;, which meant they took small samples here and there, then built a model of the genome with various statistical shenanigans.  About five years later, after all the hype, they admitted they had not sequenced the entire genome.<p>Ref: <a href="https:&#x2F;&#x2F;www.technologyreview.com&#x2F;2007&#x2F;09&#x2F;04&#x2F;223919&#x2F;craig-venters-genome&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.technologyreview.com&#x2F;2007&#x2F;09&#x2F;04&#x2F;223919&#x2F;craig-ven...</a></div><br/><div id="37258250" class="c"><input type="checkbox" id="c-37258250" checked=""/><div class="controls bullet"><span class="by">jhbadger</span><span>|</span><a href="#37257371">parent</a><span>|</span><a href="#37257695">next</a><span>|</span><label class="collapse" for="c-37258250">[-]</label><label class="expand" for="c-37258250">[3 more]</label></div><br/><div class="children"><div class="content">As noted, the shotgun sequencing wasn&#x27;t any worse than the traditional chromosome walking method used by the other human genome project, and neither created a truly complete sequence, which is only possible today with improvements in technology.<p>And it is worth noting that despite all the critiques of shotgun sequencing at the time as &quot;cheating&quot;, chromosome walking sequencing is dead, and shotgun is what we use today. If anything replaces that it will be long read technology like that done by Oxford Nanopore sequencing, although we aren&#x27;t really there yet (you still need to assemble that data and don&#x27;t really get end-to-end sequences yet).</div><br/><div id="37258757" class="c"><input type="checkbox" id="c-37258757" checked=""/><div class="controls bullet"><span class="by">kazinator</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258250">parent</a><span>|</span><a href="#37257695">next</a><span>|</span><label class="collapse" for="c-37258757">[-]</label><label class="expand" for="c-37258757">[2 more]</label></div><br/><div class="children"><div class="content">I feel a joke coming on.<p>Mother: &quot;Amy, dear, you hardly touched any of your vegetables. Most of them are still on your plate.&quot;<p>Amy: &quot;That&#x27;s not true, Mom; I completely sequenced them into my tummy using the shotgun method.&quot;</div><br/><div id="37258889" class="c"><input type="checkbox" id="c-37258889" checked=""/><div class="controls bullet"><span class="by">udkl</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258757">parent</a><span>|</span><a href="#37257695">next</a><span>|</span><label class="collapse" for="c-37258889">[-]</label><label class="expand" for="c-37258889">[1 more]</label></div><br/><div class="children"><div class="content">Someone submit this to smbc comics !</div><br/></div></div></div></div></div></div><div id="37257695" class="c"><input type="checkbox" id="c-37257695" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257371">parent</a><span>|</span><a href="#37258250">prev</a><span>|</span><a href="#37257596">next</a><span>|</span><label class="collapse" for="c-37257695">[-]</label><label class="expand" for="c-37257695">[13 more]</label></div><br/><div class="children"><div class="content">Yes, but the HGP didn&#x27;t make a full sequence either, using their scaffold-based contig assembly.  Both groups declared a truce and announced they were &quot;finished with the first draft&quot;<p>35 years ago:
<a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;1987&#x2F;12&#x2F;13&#x2F;magazine&#x2F;the-genome-project.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;1987&#x2F;12&#x2F;13&#x2F;magazine&#x2F;the-genome-proje...</a><p>33 years ago:
<a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;1990&#x2F;06&#x2F;05&#x2F;science&#x2F;great-15-year-project-to-decipher-genes-stirs-opposition.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;1990&#x2F;06&#x2F;05&#x2F;science&#x2F;great-15-year-pro...</a><p>29 years ago: the competition gets fierce
<a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;1994&#x2F;02&#x2F;22&#x2F;science&#x2F;scientist-at-work-j-craig-venter-a-bold-short-cut-to-human-genes.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;1994&#x2F;02&#x2F;22&#x2F;science&#x2F;scientist-at-work...</a><p>24 years ago: the race is alive!
<a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;1999&#x2F;03&#x2F;23&#x2F;science&#x2F;who-ll-sequence-human-genome-first-it-s-up-to-phred.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;1999&#x2F;03&#x2F;23&#x2F;science&#x2F;who-ll-sequence-h...</a><p>23 years ago: draft is complete
<a href="https:&#x2F;&#x2F;archive.nytimes.com&#x2F;www.nytimes.com&#x2F;library&#x2F;national&#x2F;science&#x2F;062600sci-human-genome.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;archive.nytimes.com&#x2F;www.nytimes.com&#x2F;library&#x2F;national...</a><p>20 years ago:
<a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;2003&#x2F;04&#x2F;15&#x2F;science&#x2F;once-again-scientists-say-human-genome-is-complete.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;2003&#x2F;04&#x2F;15&#x2F;science&#x2F;once-again-scient...</a><p>2 years ago:
<a href="https:&#x2F;&#x2F;archive.nytimes.com&#x2F;www.nytimes.com&#x2F;library&#x2F;national&#x2F;science&#x2F;062600sci-human-genome.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;archive.nytimes.com&#x2F;www.nytimes.com&#x2F;library&#x2F;national...</a><p>The celera approach using shotgun was tricky because assembling the bits required a great deal of computational finesse and horsepower; &quot;The final assembly computations were run on Compaq’s new AlphaServer GS160 because the algorithms and data required 64 gigabytes of shared memory to run successfully.&quot;<p>At the time the GS160 was the monster machine, a classic big iron UNIX, which could be tightly clustered- unified IO&#x2F;filesystem between all the machines.  The primary author of the shotgun assembly was Gene Myers, who previously had written BLAST, and invented the Suffix Array with Udi Manber.<p>The public project had its own issues, as many of the teams assigned to work on it still had the &quot;cottage industry&#x2F;artisinal&#x2F;academic&quot; approach, then Eric Lander came along and turned it into an industrial process, and parlayed that into running the Broad Institute, a privately funded MIT&#x2F;Harvard research institute in Boston.<p>These days petabytes of sequence data are generated every day and stored in clouds.  The genome has been a fundamental tool for shaping our studies of humans, although its true potential for understanding complex phenotypes remains elusive.</div><br/><div id="37259569" class="c"><input type="checkbox" id="c-37259569" checked=""/><div class="controls bullet"><span class="by">dotancohen</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37257695">parent</a><span>|</span><a href="#37258002">next</a><span>|</span><label class="collapse" for="c-37259569">[-]</label><label class="expand" for="c-37259569">[1 more]</label></div><br/><div class="children"><div class="content"><p><pre><code>  &gt; The primary author of the shotgun assembly was Gene Myers
</code></pre>
No conflict of interest there!</div><br/></div></div><div id="37258002" class="c"><input type="checkbox" id="c-37258002" checked=""/><div class="controls bullet"><span class="by">jghn</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37257695">parent</a><span>|</span><a href="#37259569">prev</a><span>|</span><a href="#37257994">next</a><span>|</span><label class="collapse" for="c-37258002">[-]</label><label class="expand" for="c-37258002">[10 more]</label></div><br/><div class="children"><div class="content">Lets not forget how Perl Saved The Human Genome Project [1], which I remember seeing in a print magazine at the time.<p>As someone who is in the genomics world now as a software person, I find it amusing that I&#x27;ve gotten more into perl over the last year or so. It has its place, in the way that grep&#x2F;sed&#x2F;awk&#x2F;etc does.<p>[1] <a href="https:&#x2F;&#x2F;news.ycombinator.com&#x2F;item?id=30327812">https:&#x2F;&#x2F;news.ycombinator.com&#x2F;item?id=30327812</a></div><br/><div id="37258070" class="c"><input type="checkbox" id="c-37258070" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258002">parent</a><span>|</span><a href="#37257994">next</a><span>|</span><label class="collapse" for="c-37258070">[-]</label><label class="expand" for="c-37258070">[9 more]</label></div><br/><div class="children"><div class="content">Lincoln Stein is great, and perl was certainly critical to many processes, but it was and remains fairly niche in genomics, which used much more C++, Java, and later Python.<p>IMHO the person who &quot;saved the genome project&quot; was WJ Kent, who developed the assembler, BLAT, that the public project needed.  I strive to point out that he wasn&#x27;t a sole hero, nor was Lincoln.  What I really like about BLAT is that while Celera was using Big Iron UNIX (massive 64-bit 64GB machines with 10s of terabytes of central high performance storage), BLAT ran on a cluster of linux machines, right around the time that people were waking up to the fact that linux was becoming a useful tool for scientific data processing.  BLAT&#x27;s design allowed it to work on a cluster of cheaper&#x2F;smaller machines, while celera&#x27;s algorithm really needed a massive shared memory single machine.  it was sort of a microcosm of the larger battle being fought between Big iron UNIX and little intel linux at the time.</div><br/><div id="37258501" class="c"><input type="checkbox" id="c-37258501" checked=""/><div class="controls bullet"><span class="by">ryandv</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258070">parent</a><span>|</span><a href="#37258202">next</a><span>|</span><label class="collapse" for="c-37258501">[-]</label><label class="expand" for="c-37258501">[1 more]</label></div><br/><div class="children"><div class="content">I had the privilege of working a co-op term at Stein&#x27;s lab at the OICR. I encountered quite a bit of Perl during my short time there, and have yet to see it elsewhere in the modern enterprise tech world. BioPerl in particular stands out as a fairly substantial project in the bioinformatics space.</div><br/></div></div><div id="37258202" class="c"><input type="checkbox" id="c-37258202" checked=""/><div class="controls bullet"><span class="by">iten</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258070">parent</a><span>|</span><a href="#37258501">prev</a><span>|</span><a href="#37258188">next</a><span>|</span><label class="collapse" for="c-37258202">[-]</label><label class="expand" for="c-37258202">[1 more]</label></div><br/><div class="children"><div class="content">Small correction: BLAT is a local alignment tool Jim Kent also wrote. I think his assembler you&#x27;re referring to is GigAssembler (<a href="https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC311095&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC311095&#x2F;</a>).</div><br/></div></div><div id="37258188" class="c"><input type="checkbox" id="c-37258188" checked=""/><div class="controls bullet"><span class="by">jghn</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258070">parent</a><span>|</span><a href="#37258202">prev</a><span>|</span><a href="#37258174">next</a><span>|</span><label class="collapse" for="c-37258188">[-]</label><label class="expand" for="c-37258188">[1 more]</label></div><br/><div class="children"><div class="content">I agree from the human genome sequencing perspective and the superlatives of saving the HGP. But at the time I was a standard software dev who thought the topic was cool, as was this new fan dangled language named perl.<p>However, I got into the genomics world in the early aughts, and the perl hung on and on and on. I remember starting a new job in the mid-teens, and one of the first tasks I had was to port over a legacy perl script. Such is the world of scientific software. 99% of the people I know who have touched perl since ~2003ish are in the bioinformatics space.</div><br/></div></div><div id="37258174" class="c"><input type="checkbox" id="c-37258174" checked=""/><div class="controls bullet"><span class="by">jfoutz</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258070">parent</a><span>|</span><a href="#37258188">prev</a><span>|</span><a href="#37257994">next</a><span>|</span><label class="collapse" for="c-37258174">[-]</label><label class="expand" for="c-37258174">[5 more]</label></div><br/><div class="children"><div class="content">I know hn isn&#x27;t the place for memes, but, I&#x27;m obligated.
Imagine what they could do with a beowulf cluster of pentium pros?</div><br/><div id="37258289" class="c"><input type="checkbox" id="c-37258289" checked=""/><div class="controls bullet"><span class="by">epistasis</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258174">parent</a><span>|</span><a href="#37258660">next</a><span>|</span><label class="collapse" for="c-37258289">[-]</label><label class="expand" for="c-37258289">[1 more]</label></div><br/><div class="children"><div class="content">There was a NYTimes profile at the time of Kent&#x27;s efforts, and they IIRC literally got the UC Santa Cruz administration to hand over desktop computers destined to go to various people for a few months to assemble into a cluster. So it was likely a bunch of pentium pros or similar. Not a Beowulf cluster, of course, but whatever clustering software Kent had hand rolled out of C for the purpose.</div><br/></div></div><div id="37258660" class="c"><input type="checkbox" id="c-37258660" checked=""/><div class="controls bullet"><span class="by">jacquesm</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258174">parent</a><span>|</span><a href="#37258289">prev</a><span>|</span><a href="#37258237">next</a><span>|</span><label class="collapse" for="c-37258660">[-]</label><label class="expand" for="c-37258660">[2 more]</label></div><br/><div class="children"><div class="content">They weren&#x27;t using pentium pros, they were using Alpha&#x27;s because the memory interconnect between a beowulf cluster of pentium pros would be orders of magnitude slower. So likely &#x27;the same, but much slower&#x27;.</div><br/><div id="37258868" class="c"><input type="checkbox" id="c-37258868" checked=""/><div class="controls bullet"><span class="by">jfoutz</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258660">parent</a><span>|</span><a href="#37258237">next</a><span>|</span><label class="collapse" for="c-37258868">[-]</label><label class="expand" for="c-37258868">[1 more]</label></div><br/><div class="children"><div class="content">oh, ha. yeah, that was sort of the root of the joke. companies like Celera, or whoever had the money would write a (large) check for big iron.<p>smaller organizations were realizing they could get slower computers power pretty cheap, and they could get a whole lot of them. There was a little flurry of activity with flocking algorithms, objective-c had a little renaissance due to swarm computing. probably the most famous and lasting was map-reduce, from alphabet, but they were called google back then.<p>there were a bunch of clever little tricks, like channel bonded network cards to make multiple cards look like one fast card, so you could double or triple bandwidth.<p>The beowulf cluster joke was kind of the spirit of the scrappy, make something cool out of junk approach while calling companies like sun, hp, compaq dinosaurs. and it continued for a while with people building stuff like this - <a href="https:&#x2F;&#x2F;ncsa30.ncsa.illinois.edu&#x2F;2003&#x2F;05&#x2F;ncsa-creates-sony-playstation2-cluster&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;ncsa30.ncsa.illinois.edu&#x2F;2003&#x2F;05&#x2F;ncsa-creates-sony-p...</a> I think there was some weirdness with export controls of ps2&#x27;s for this kind of stuff.<p>I don&#x27;t remember what pc chip was the new hotness back then, it was a good 20 years ago and my memory is dim.<p>but that, I think, captures the gist of the meme.</div><br/></div></div></div></div><div id="37258237" class="c"><input type="checkbox" id="c-37258237" checked=""/><div class="controls bullet"><span class="by">jghn</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37258174">parent</a><span>|</span><a href="#37258660">prev</a><span>|</span><a href="#37257994">next</a><span>|</span><label class="collapse" for="c-37258237">[-]</label><label class="expand" for="c-37258237">[1 more]</label></div><br/><div class="children"><div class="content">I feel like this is the one domain where the beowfulf cluster meme is still funny</div><br/></div></div></div></div></div></div></div></div><div id="37257994" class="c"><input type="checkbox" id="c-37257994" checked=""/><div class="controls bullet"><span class="by">sublinear</span><span>|</span><a href="#37257371">root</a><span>|</span><a href="#37257695">parent</a><span>|</span><a href="#37258002">prev</a><span>|</span><a href="#37257596">next</a><span>|</span><label class="collapse" for="c-37257994">[-]</label><label class="expand" for="c-37257994">[1 more]</label></div><br/><div class="children"><div class="content">&gt; The primary author of the shotgun assembly was Gene Myers, who previously had written BLAST...<p>So... um... BLAST processing?</div><br/></div></div></div></div></div></div><div id="37257407" class="c"><input type="checkbox" id="c-37257407" checked=""/><div class="controls bullet"><span class="by">jshorty</span><span>|</span><a href="#37257371">prev</a><span>|</span><a href="#37257555">next</a><span>|</span><label class="collapse" for="c-37257407">[-]</label><label class="expand" for="c-37257407">[68 more]</label></div><br/><div class="children"><div class="content">Could someone explain exactly what it means to be &quot;completely sequence&quot; the human genome when all humans have distinct genetic makeup (ie, different sequences of nucleobases in their DNA&#x2F;RNA)?</div><br/><div id="37258814" class="c"><input type="checkbox" id="c-37258814" checked=""/><div class="controls bullet"><span class="by">diekhans</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37257742">next</a><span>|</span><label class="collapse" for="c-37258814">[-]</label><label class="expand" for="c-37258814">[1 more]</label></div><br/><div class="children"><div class="content">We really should say &quot;a human genome&quot;.  Reference genomes serve as a Rosetta Stone of genomics.    So we can take DNA&#x2F;RNA sequences from other individuals and align (pattern match) them to the reference as a way of understanding and comparing individuals.<p>It is not perfect, as a references can be missing or have large variability in DNA regions.  The goal of the Human Pangenome Reference Consortium (HPRC) <a href="https:&#x2F;&#x2F;humanpangenome.org&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;humanpangenome.org&#x2F;</a> is to sequence individuals from different populations to address this issue.  We are also working to develop new computation models to support analysis of data across populations.</div><br/></div></div><div id="37257742" class="c"><input type="checkbox" id="c-37257742" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37258814">prev</a><span>|</span><a href="#37257467">next</a><span>|</span><label class="collapse" for="c-37257742">[-]</label><label class="expand" for="c-37257742">[1 more]</label></div><br/><div class="children"><div class="content">The public Human Genome Project used a group of people but most of the sequence library was derived from a single individual in Buffalo, NY.  The celera project also used a group of people but it was mostly Venter&#x27;s genome<p><a href="https:&#x2F;&#x2F;www.nytimes.com&#x2F;2002&#x2F;04&#x2F;27&#x2F;us&#x2F;scientist-reveals-secret-of-genome-it-s-his.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nytimes.com&#x2F;2002&#x2F;04&#x2F;27&#x2F;us&#x2F;scientist-reveals-secr...</a><p>I believe more recent sequencing projects have used a wider pool of individuals.  I think some projects pool all the individuals and sequence them together, while others sequence each individual separately.  This isn&#x27;t really so much of a problem since the large-scale structure is highly similar across all humans and we have developed sophisticated approaches to model the variations in individuals, see <a href="https:&#x2F;&#x2F;www.biomedcentral.com&#x2F;collections&#x2F;graphgenomes" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.biomedcentral.com&#x2F;collections&#x2F;graphgenomes</a> for an explanation of the &quot;graph structure&quot; used to reprsent alternatives in the reference, which can include individual single nucleobase differences, as well as more complex ones such as large deletions in one individual, to rearrangements and even inversions.</div><br/></div></div><div id="37257467" class="c"><input type="checkbox" id="c-37257467" checked=""/><div class="controls bullet"><span class="by">pdonis</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37257742">prev</a><span>|</span><a href="#37257439">next</a><span>|</span><label class="collapse" for="c-37257467">[-]</label><label class="expand" for="c-37257467">[1 more]</label></div><br/><div class="children"><div class="content">They mean they have obtained the complete sequence for a particular Y chromosome that is considered to be a &quot;reference&quot; chromosome. This is similar to what was done for all the other chromosomes.</div><br/></div></div><div id="37257439" class="c"><input type="checkbox" id="c-37257439" checked=""/><div class="controls bullet"><span class="by">chihuahua</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37257467">prev</a><span>|</span><a href="#37258863">next</a><span>|</span><label class="collapse" for="c-37257439">[-]</label><label class="expand" for="c-37257439">[55 more]</label></div><br/><div class="children"><div class="content">I&#x27;ve never understood this either. I assume the genome is many megabytes of [ATCG]+. If we have that sequence, what does it tell us? Do we look at it and say &quot;Ah, yes, ...ATGCTACGACTACGACTAGCG... very interesting?&quot;</div><br/><div id="37257470" class="c"><input type="checkbox" id="c-37257470" checked=""/><div class="controls bullet"><span class="by">jiggawatts</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37259486">next</a><span>|</span><label class="collapse" for="c-37257470">[-]</label><label class="expand" for="c-37257470">[5 more]</label></div><br/><div class="children"><div class="content">Many genes are highly conserved or consistent enough. E.g.: if there&#x27;s a 1% difference between two people, then it&#x27;s a bit like two very unique sentences that have a couple of small typos. They&#x27;re sill recognisable, and it&#x27;s also still pretty obvious that they&#x27;re the &quot;same&quot;.<p>A gene sequence allows researchers to determine the amino acids that are coded for, and from those, which proteins match which genes.<p>This can be matched up with genetic diseases. If you know that damage to a certain location in a chromosome causes a problem with a certain biological process, then ergo, the associated protein is needed for that process!<p>So: genetic illness -&gt; gene sequence -&gt; protein -&gt; role in the body<p>Without sequencing, that chain can&#x27;t be built.</div><br/><div id="37258380" class="c"><input type="checkbox" id="c-37258380" checked=""/><div class="controls bullet"><span class="by">dclowd9901</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257470">parent</a><span>|</span><a href="#37259486">next</a><span>|</span><label class="collapse" for="c-37258380">[-]</label><label class="expand" for="c-37258380">[4 more]</label></div><br/><div class="children"><div class="content">But you can only know that by having a large sample of very “stable” (have few genetic irregularities) gene samples compared to a large pool of samples from people with very narrow and pronounced gene irregularities, right?<p>Is this why it’s so hard? This feels more like a healthcare records keeping people and less like an “actually reading the data problem”.<p>I can’t help but feel like some form of single payer healthcare is truly the way out of this problem. One where all disease record keeping is uniform and complete.</div><br/><div id="37258695" class="c"><input type="checkbox" id="c-37258695" checked=""/><div class="controls bullet"><span class="by">Earw0rm</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258380">parent</a><span>|</span><a href="#37258678">next</a><span>|</span><label class="collapse" for="c-37258695">[-]</label><label class="expand" for="c-37258695">[2 more]</label></div><br/><div class="children"><div class="content">Single payer healthcare here (UK) is still subject to privacy controls in a way which would make it very difficult to do that.<p>(Also our health system&#x27;s IT is a hellscape, but one reason for that is that people would literally rather not have a working system at all, than one with less than impeccable privacy controls.<p>Personally I&#x27;d gladly sacrifice a fair bit of medical privacy in return for giving scientists greater insight into disease processes, but the average citizen here wants advanced healthcare without giving their data to research scientists. &#x2F;facepalm )</div><br/><div id="37259576" class="c"><input type="checkbox" id="c-37259576" checked=""/><div class="controls bullet"><span class="by">spookie</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258695">parent</a><span>|</span><a href="#37258678">next</a><span>|</span><label class="collapse" for="c-37259576">[-]</label><label class="expand" for="c-37259576">[1 more]</label></div><br/><div class="children"><div class="content">I trust the scientists, the problem isn&#x27;t them.
Look at the whole abortion data scandal in the U.S.</div><br/></div></div></div></div></div></div></div></div><div id="37259486" class="c"><input type="checkbox" id="c-37259486" checked=""/><div class="controls bullet"><span class="by">dwattttt</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37257470">prev</a><span>|</span><a href="#37257590">next</a><span>|</span><label class="collapse" for="c-37259486">[-]</label><label class="expand" for="c-37259486">[1 more]</label></div><br/><div class="children"><div class="content">You could say exactly the same of all data; it&#x27;s just 1s and 0s, but when I look I just see blonde, brunette.</div><br/></div></div><div id="37257590" class="c"><input type="checkbox" id="c-37257590" checked=""/><div class="controls bullet"><span class="by">bglazer</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37259486">prev</a><span>|</span><a href="#37257544">next</a><span>|</span><label class="collapse" for="c-37257590">[-]</label><label class="expand" for="c-37257590">[1 more]</label></div><br/><div class="children"><div class="content">It’s actually about 3 gigabases (ATCG). There are some recurrent features of the genome whose function we’ve worked out. For example the TATA box is a classic sequence that typically indicates the start of a part of the genome that codes for a protein. The vast majority of the genome doesn’t code for proteins. The function of these genome regions are much more murky. Some of these regions function like scaffolds for proteins to assemble into complexes. These protein complexes then start transcribing the genome into into mRNA. So the genome regulates its own expression, in a sense. Many of the sequences that function in this way are known. There are also just a bunch of parts of the genome that probably don’t do anything. There are also many regions of the genome that are basically self replicating sequences. They code for proteins that are capable of inserting their own genetic sequence back into the genome. These are transposons.<p>In short, a lot of very painstaking genetics and molecular biology work has gone into characterizing the function of certain sequences.</div><br/></div></div><div id="37257544" class="c"><input type="checkbox" id="c-37257544" checked=""/><div class="controls bullet"><span class="by">throw3823423</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37257590">prev</a><span>|</span><a href="#37257575">next</a><span>|</span><label class="collapse" for="c-37257544">[-]</label><label class="expand" for="c-37257544">[1 more]</label></div><br/><div class="children"><div class="content">For the same reason Monsanto sequences basically anything: Because we can tell what proteins are encoded in there, and what is near them, and we can have good ideas of what proteins are expressed together. When dealing with genetic modification, we get to see whether our modification went in, and where it landed: Having a protein in a genome isn&#x27;t enough. Its expression might be having an effect on other things, depending on where it is.<p>When we have baselines, we can compare different individuals, and eventually make predictions of how they are going to be based solely on the genetic code. If I know that a certain polymorphism is tied to some trait I want, I might not have to even bother spending the time growing a plant: I know that it&#x27;s not what I want, and discard it as a seed.<p>With humans we are probably not going to see much modification soon, but just being able to detect genetic diseases, risk factors for other diseases that have genetic omponents, or allow for selection of embryos in cases of artificial insemination is already quite valuable.<p>It&#x27;s not source code that we are all that good at understanding just yet, but there&#x27;s already some applications, and we have good reason to think there&#x27;s a lot more to come</div><br/></div></div><div id="37257575" class="c"><input type="checkbox" id="c-37257575" checked=""/><div class="controls bullet"><span class="by">khazhoux</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37257544">prev</a><span>|</span><a href="#37257820">next</a><span>|</span><label class="collapse" for="c-37257575">[-]</label><label class="expand" for="c-37257575">[35 more]</label></div><br/><div class="children"><div class="content">It&#x27;s just about 3 gigabytes (each byte a letter).  Pretty mind-blowing, if you ask me.</div><br/><div id="37257719" class="c"><input type="checkbox" id="c-37257719" checked=""/><div class="controls bullet"><span class="by">cvoss</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257575">parent</a><span>|</span><a href="#37257935">next</a><span>|</span><label class="collapse" for="c-37257719">[-]</label><label class="expand" for="c-37257719">[32 more]</label></div><br/><div class="children"><div class="content">It&#x27;s a slight exaggeration of the information content to report the data size using an ASCII encoding. Since there are 4 bases, each can be encoded using 2 bits, rather than 8. So we&#x27;re really talking 750 megabytes. But still mind-blowing.</div><br/><div id="37257784" class="c"><input type="checkbox" id="c-37257784" checked=""/><div class="controls bullet"><span class="by">borissk</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257719">parent</a><span>|</span><a href="#37257867">next</a><span>|</span><label class="collapse" for="c-37257784">[-]</label><label class="expand" for="c-37257784">[24 more]</label></div><br/><div class="children"><div class="content">I&#x27;m curious if these 750MB + the DNA of mitochondria + the protein metagenomics contain all the information needed to build a human, or if there&#x27;s extra info stored in the machinery of the first cell.<p>That is if we transfer the DNA to an advanced alien civilization - would they be able to make a human.</div><br/><div id="37259343" class="c"><input type="checkbox" id="c-37259343" checked=""/><div class="controls bullet"><span class="by">mjan22640</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37257931">next</a><span>|</span><label class="collapse" for="c-37259343">[-]</label><label class="expand" for="c-37259343">[1 more]</label></div><br/><div class="children"><div class="content">Imagine a machine shop that has blueprints of components of the machines they use in the shop, and processes to assemble machines from the components. When a machine shop grows large and splits in two, each inherits a half of shop with the ongoing processes and a copy of the blueprints.
<a href="https:&#x2F;&#x2F;m.youtube.com&#x2F;watch?v=B7PMf7bBczQ&amp;pp=QAFIAQ%3D%3D">https:&#x2F;&#x2F;m.youtube.com&#x2F;watch?v=B7PMf7bBczQ&amp;pp=QAFIAQ%3D%3D</a><p>DNA is the blueprints.
There are infinite possibilities what to do with them. The advanced civilization would need additional information, like that they are supposed create a cell from the components to begin with, and a lot of detailed information how exactly to do that.<p>Edit: improved clarity</div><br/></div></div><div id="37257931" class="c"><input type="checkbox" id="c-37257931" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37259343">prev</a><span>|</span><a href="#37258880">next</a><span>|</span><label class="collapse" for="c-37257931">[-]</label><label class="expand" for="c-37257931">[11 more]</label></div><br/><div class="children"><div class="content">This is a complex question.  The cocktail soup in a gamete (sperm or egg) and the resulting zygote contains an awful lot of stuff that would be extremely hard to replace.  I could imagine that if the receiving civilization was sufficiently advanced <i>and</i> had a model of what those cells contained (beyond the genomic information) they could build some sort of artificial cell that could bootstrap the genome to the point of being able to start the development process.  it would be quite an accomplishment.<p>If they just received the DNA without some information about the zygote, I don&#x27;t think it would be practical for even advanced alien civilization (LR5 or LR6) but probably an LR7 and definitely an LR8 could.</div><br/><div id="37259457" class="c"><input type="checkbox" id="c-37259457" checked=""/><div class="controls bullet"><span class="by">Angostura</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257931">parent</a><span>|</span><a href="#37258347">next</a><span>|</span><label class="collapse" for="c-37259457">[-]</label><label class="expand" for="c-37259457">[1 more]</label></div><br/><div class="children"><div class="content">I’m just pondering this, and it’s not clear to me that there is anything intrinsic in the genome itself that explicitly’says’ “this sequence of DNA bases encodes a protein” or even “these three base-pairs equate to this amino acid”.<p>I wonder if that information could ever really be untangled  by a civilisation starting entirely from scratch without access to a cell</div><br/></div></div><div id="37258347" class="c"><input type="checkbox" id="c-37258347" checked=""/><div class="controls bullet"><span class="by">namanyayg</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257931">parent</a><span>|</span><a href="#37259457">prev</a><span>|</span><a href="#37257966">next</a><span>|</span><label class="collapse" for="c-37258347">[-]</label><label class="expand" for="c-37258347">[3 more]</label></div><br/><div class="children"><div class="content">What do you mean by &quot;LR&quot;? I queried an LLM but no results there either.</div><br/><div id="37258419" class="c"><input type="checkbox" id="c-37258419" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258347">parent</a><span>|</span><a href="#37258707">next</a><span>|</span><label class="collapse" for="c-37258419">[-]</label><label class="expand" for="c-37258419">[1 more]</label></div><br/><div class="children"><div class="content">oops i&#x27;ve said too much</div><br/></div></div></div></div><div id="37257966" class="c"><input type="checkbox" id="c-37257966" checked=""/><div class="controls bullet"><span class="by">borissk</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257931">parent</a><span>|</span><a href="#37258347">prev</a><span>|</span><a href="#37258880">next</a><span>|</span><label class="collapse" for="c-37257966">[-]</label><label class="expand" for="c-37257966">[6 more]</label></div><br/><div class="children"><div class="content">The code how to build a sperm and an egg is inside the human DNA, isn&#x27;t it?</div><br/><div id="37257996" class="c"><input type="checkbox" id="c-37257996" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257966">parent</a><span>|</span><a href="#37258548">next</a><span>|</span><label class="collapse" for="c-37257996">[-]</label><label class="expand" for="c-37257996">[1 more]</label></div><br/><div class="children"><div class="content">Yes, but it currently requires developmentally mature individuals to build the gametes, and the &quot;code&quot; is so complex you couldn&#x27;t really decipher it from first principles.</div><br/></div></div><div id="37258548" class="c"><input type="checkbox" id="c-37258548" checked=""/><div class="controls bullet"><span class="by">klyrs</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257966">parent</a><span>|</span><a href="#37257996">prev</a><span>|</span><a href="#37258167">next</a><span>|</span><label class="collapse" for="c-37258548">[-]</label><label class="expand" for="c-37258548">[3 more]</label></div><br/><div class="children"><div class="content">Given code written for unknown hardware... can you execute it?</div><br/><div id="37258620" class="c"><input type="checkbox" id="c-37258620" checked=""/><div class="controls bullet"><span class="by">borissk</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258548">parent</a><span>|</span><a href="#37258167">next</a><span>|</span><label class="collapse" for="c-37258620">[-]</label><label class="expand" for="c-37258620">[2 more]</label></div><br/><div class="children"><div class="content">Given that the code contains the instructions how to make the hardware - if one is very smart than yes.</div><br/><div id="37258832" class="c"><input type="checkbox" id="c-37258832" checked=""/><div class="controls bullet"><span class="by">bbrx</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258620">parent</a><span>|</span><a href="#37258167">next</a><span>|</span><label class="collapse" for="c-37258832">[-]</label><label class="expand" for="c-37258832">[1 more]</label></div><br/><div class="children"><div class="content">It would not necessary be possible, because it&#x27;s incremental instructions on how to make the hardware, but based on already existing, unspecified and very complex, hardware. So the first instruction would be something like &quot;take the stuff you have on your left and fuse it with the stuff you have on your right&quot;, both stuff being unspecified very complex protein assumed to be present.</div><br/></div></div></div></div></div></div><div id="37258167" class="c"><input type="checkbox" id="c-37258167" checked=""/><div class="controls bullet"><span class="by">tobinfricke</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257966">parent</a><span>|</span><a href="#37258548">prev</a><span>|</span><a href="#37258880">next</a><span>|</span><label class="collapse" for="c-37258167">[-]</label><label class="expand" for="c-37258167">[1 more]</label></div><br/><div class="children"><div class="content">The code to build mitochondria is not.</div><br/></div></div></div></div></div></div><div id="37258880" class="c"><input type="checkbox" id="c-37258880" checked=""/><div class="controls bullet"><span class="by">pmoriarty</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37257931">prev</a><span>|</span><a href="#37257908">next</a><span>|</span><label class="collapse" for="c-37258880">[-]</label><label class="expand" for="c-37258880">[3 more]</label></div><br/><div class="children"><div class="content"><i>&quot;if we transfer the DNA to an advanced alien civilization - would they be able to make a human.&quot;</i><p>I&#x27;m really surprised that in all these responses to your question no one&#x27;s mentioned the womb or the mother, who (at least with current technology) is still necessary for making a human.<p>That&#x27;s not to mention the necessity of the egg.<p>We&#x27;re not just DNA.</div><br/><div id="37259097" class="c"><input type="checkbox" id="c-37259097" checked=""/><div class="controls bullet"><span class="by">dchftcs</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258880">parent</a><span>|</span><a href="#37259047">next</a><span>|</span><label class="collapse" for="c-37259097">[-]</label><label class="expand" for="c-37259097">[1 more]</label></div><br/><div class="children"><div class="content">This is a question about theoretical possibilities and what you&#x27;re saying seems to be a rigid belief in an answer &quot;no&quot;. But you provided no evidence or justification, except for &quot;with current technology&quot;, which answers nothing about the theoretical question.</div><br/></div></div><div id="37259047" class="c"><input type="checkbox" id="c-37259047" checked=""/><div class="controls bullet"><span class="by">borissk</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258880">parent</a><span>|</span><a href="#37259097">prev</a><span>|</span><a href="#37257908">next</a><span>|</span><label class="collapse" for="c-37259047">[-]</label><label class="expand" for="c-37259047">[1 more]</label></div><br/><div class="children"><div class="content">Instructions on how to make a womb and an &quot;egg&quot; are contained within the human DNA.</div><br/></div></div></div></div><div id="37257908" class="c"><input type="checkbox" id="c-37257908" checked=""/><div class="controls bullet"><span class="by">senkora</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37258880">prev</a><span>|</span><a href="#37257851">next</a><span>|</span><label class="collapse" for="c-37257908">[-]</label><label class="expand" for="c-37257908">[1 more]</label></div><br/><div class="children"><div class="content">I can’t wait until we can bootstrap a human from a stage 3 tarball.</div><br/></div></div><div id="37257851" class="c"><input type="checkbox" id="c-37257851" checked=""/><div class="controls bullet"><span class="by">ejstronge</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37257908">prev</a><span>|</span><a href="#37258756">next</a><span>|</span><label class="collapse" for="c-37257851">[-]</label><label class="expand" for="c-37257851">[5 more]</label></div><br/><div class="children"><div class="content">&gt; That is if we transfer the DNA to an advanced alien civilization - would they be able to make a human.<p>You&#x27;d need a cell to start the process, with the various nucleic acids distributed correctly and proteins&#x2F;energy with which to create further proteins using the information encoded by the DNA. Thus the civilization would need information about cells and a set of building blocks before being able to use the DNA.</div><br/><div id="37257951" class="c"><input type="checkbox" id="c-37257951" checked=""/><div class="controls bullet"><span class="by">borissk</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257851">parent</a><span>|</span><a href="#37258756">next</a><span>|</span><label class="collapse" for="c-37257951">[-]</label><label class="expand" for="c-37257951">[4 more]</label></div><br/><div class="children"><div class="content">The DNA contains all the code that creates and regulates the proteins.</div><br/><div id="37258131" class="c"><input type="checkbox" id="c-37258131" checked=""/><div class="controls bullet"><span class="by">penteract</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257951">parent</a><span>|</span><a href="#37258218">next</a><span>|</span><label class="collapse" for="c-37258131">[-]</label><label class="expand" for="c-37258131">[2 more]</label></div><br/><div class="children"><div class="content">Including code for the proteins that read DNA to produce proteins. You might hit similar problems trying to understand C given the source code for a C compiler - a non-standard environment could reproduce itself given the source code, meaning the code alone doesn&#x27;t strictly determine the output.</div><br/><div id="37258588" class="c"><input type="checkbox" id="c-37258588" checked=""/><div class="controls bullet"><span class="by">kibibyte</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258131">parent</a><span>|</span><a href="#37258218">next</a><span>|</span><label class="collapse" for="c-37258588">[-]</label><label class="expand" for="c-37258588">[1 more]</label></div><br/><div class="children"><div class="content">I&#x27;ll torture this DNA and C source code analogy a bit.<p>Epigenetics is missing in this discussion about reproducing a human from just the DNA. These are superficial modifications (e.g. methylation, histone modification, repressor factors) to a strand of DNA that can drastically alter how specific regions get expressed into proteins. These mechanisms essentially work by either hiding or unhiding DNA from RNA polymerases and other parts of the transcription complex. These mechanisms can change throughout your lifetime because of environmental factors and can be inherited.<p>So it&#x27;s like reading C source code, except there so many of these inscrutable C preprocessor directives strewn all throughout. You won&#x27;t get a successful compilation by turning on or off all the directives. Instead, you need to get this similarly inscrutable configuration blob that tells you how to set each directive.<p>I guess in a way, it&#x27;s like the weights for an ML model. It just works, you can&#x27;t explain why it works, and changing this weight here produces a program that crashes prematurely, and changing a weight there produces a program with allergic reactions to everything.</div><br/></div></div></div></div><div id="37258218" class="c"><input type="checkbox" id="c-37258218" checked=""/><div class="controls bullet"><span class="by">khazhoux</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257951">parent</a><span>|</span><a href="#37258131">prev</a><span>|</span><a href="#37258756">next</a><span>|</span><label class="collapse" for="c-37258218">[-]</label><label class="expand" for="c-37258218">[1 more]</label></div><br/><div class="children"><div class="content">And how will you decode it?</div><br/></div></div></div></div></div></div><div id="37258756" class="c"><input type="checkbox" id="c-37258756" checked=""/><div class="controls bullet"><span class="by">mfld</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37257851">prev</a><span>|</span><a href="#37258534">next</a><span>|</span><label class="collapse" for="c-37258756">[-]</label><label class="expand" for="c-37258756">[1 more]</label></div><br/><div class="children"><div class="content">Yes, there is extra information in the first cells, in particular regulatory elements such as miRNAs. The headline here is epigenetics.</div><br/></div></div><div id="37258534" class="c"><input type="checkbox" id="c-37258534" checked=""/><div class="controls bullet"><span class="by">singularity2001</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257784">parent</a><span>|</span><a href="#37258756">prev</a><span>|</span><a href="#37257867">next</a><span>|</span><label class="collapse" for="c-37258534">[-]</label><label class="expand" for="c-37258534">[1 more]</label></div><br/><div class="children"><div class="content">Our DNA does not contain the mitochondria nor the gut bacteria so the raw data would most certainly not be enough to build a working copy</div><br/></div></div></div></div><div id="37257867" class="c"><input type="checkbox" id="c-37257867" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257719">parent</a><span>|</span><a href="#37257784">prev</a><span>|</span><a href="#37257905">next</a><span>|</span><label class="collapse" for="c-37257867">[-]</label><label class="expand" for="c-37257867">[2 more]</label></div><br/><div class="children"><div class="content">And since the data is highly redundant the 750MB can be compressed down even further using standard approaches (DEFLATE works well, it uses both huffman coding and dictionary backreferences).<p>Or, you could build an embedding with far fewer parameters that could explain the vast majority of phenotypic differences.  the genome is a hierarchical palimpsest of low entropy.<p>My standard interview question- because I hate leetcode- walks the interviewee through compressing DNA using bit encoding, then using that to implement a rolling hash to do fast frequency counting.  Some folks get stuck at &quot;how many bits in a byte&quot;, others at &quot;if you have 4 symbols, how many bits are required to encode a symbol?&quot;, and other candidates jump straight to bloom filters and other probabilistic approaches 
(<a href="https:&#x2F;&#x2F;github.com&#x2F;bcgsc&#x2F;ntHash">https:&#x2F;&#x2F;github.com&#x2F;bcgsc&#x2F;ntHash</a> and <a href="https:&#x2F;&#x2F;github.com&#x2F;dib-lab&#x2F;khmer">https:&#x2F;&#x2F;github.com&#x2F;dib-lab&#x2F;khmer</a> are good places to start if you are interested).</div><br/></div></div><div id="37257905" class="c"><input type="checkbox" id="c-37257905" checked=""/><div class="controls bullet"><span class="by">khazhoux</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257719">parent</a><span>|</span><a href="#37257867">prev</a><span>|</span><a href="#37257935">next</a><span>|</span><label class="collapse" for="c-37257905">[-]</label><label class="expand" for="c-37257905">[5 more]</label></div><br/><div class="children"><div class="content">Yes, and if you gzip it it&#x27;s even smaller.  But the big takeaway is that the amount of info that fully defines a human, is what we consider &quot;not much data,&quot; even in its plainest encoding.</div><br/><div id="37258100" class="c"><input type="checkbox" id="c-37258100" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257905">parent</a><span>|</span><a href="#37258020">next</a><span>|</span><label class="collapse" for="c-37258100">[-]</label><label class="expand" for="c-37258100">[1 more]</label></div><br/><div class="children"><div class="content">bzip2 is marginally better, and then genome-specific compressors were developed, and then finally, people started storing individual genomes as diffs from a single reference, <a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;CRAM_(file_format)" rel="nofollow noreferrer">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;CRAM_(file_format)</a><p>Since genome files contain more data than just ATGC (typically a comment line, then a DNA line, then a quality score line), and each of those draws from a different distribution, DEFLATE on a FASTA file doesn&#x27;t reach the full potential of the compressor because the huffman table ends up having to hold all three distributions, and the dictionary backlookups aren&#x27;t as efficient either.  
It turns out you <i>can</i> split the file into multiple streams, one per line type, and then compress those independently, with slightly better compression ratios, but it&#x27;s still not great.</div><br/></div></div><div id="37258020" class="c"><input type="checkbox" id="c-37258020" checked=""/><div class="controls bullet"><span class="by">astrange</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257905">parent</a><span>|</span><a href="#37258100">prev</a><span>|</span><a href="#37257935">next</a><span>|</span><label class="collapse" for="c-37258020">[-]</label><label class="expand" for="c-37258020">[3 more]</label></div><br/><div class="children"><div class="content">We don&#x27;t know that it fully defines a human until we can create one without the starting condition of being inside another human. It&#x27;s prototype-based inheritance.</div><br/><div id="37258403" class="c"><input type="checkbox" id="c-37258403" checked=""/><div class="controls bullet"><span class="by">hotnfresh</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258020">parent</a><span>|</span><a href="#37257935">next</a><span>|</span><label class="collapse" for="c-37258403">[-]</label><label class="expand" for="c-37258403">[2 more]</label></div><br/><div class="children"><div class="content">Some of the research about being able to make simple animals grow structures from <i>other animals in their evolutionary “tree”</i> by changing chemical signaling—among other wild things like finding that memories may be stored outside the brain, at least in some animals—makes me think you need more than just the “code” to get the animal that would have been produced if that “code” were in its full context (of a reproductive cell doing all sorts of other stuff). Even if the dna contains the instructions for that reproductive cell, too, in some sense… which instructions do you “run”? There might be multiple possible variants, some of which don’t actually reproduce the animal you took the dna from.</div><br/><div id="37259522" class="c"><input type="checkbox" id="c-37259522" checked=""/><div class="controls bullet"><span class="by">astrange</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258403">parent</a><span>|</span><a href="#37257935">next</a><span>|</span><label class="collapse" for="c-37259522">[-]</label><label class="expand" for="c-37259522">[1 more]</label></div><br/><div class="children"><div class="content">My favorite trivia here is that flamingos aren&#x27;t actually &quot;genetically&quot; pink but &quot;environmentally&quot; pink because they pick up the color from eating algae.<p>Except of course &quot;genetics&quot; and &quot;environment&quot; aren&#x27;t actually separate things; sure, people&#x27;s skin color isn&#x27;t usually affected by their food, but only because most people don&#x27;t eat colloidal silver.<p><a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Paul_Karason" rel="nofollow noreferrer">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Paul_Karason</a></div><br/></div></div></div></div></div></div></div></div></div></div></div></div><div id="37257820" class="c"><input type="checkbox" id="c-37257820" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37257575">prev</a><span>|</span><a href="#37258973">next</a><span>|</span><label class="collapse" for="c-37257820">[-]</label><label class="expand" for="c-37257820">[10 more]</label></div><br/><div class="children"><div class="content">If you think like an ML engineer, the genome is a feature vector 3B bases (or 6B binary bits) long that is highly redundant (many sections contain repeats and other regions that are correlated to other regions), and the mapping between that feature vector and an individual&#x27;s specific properties (their &quot;phenotype&quot;, which could be their height at full maturity, or their eye color, or hair properties, or propensity to diseases, etc) is highly nonlinear.<p>If you had a list of all the genomes of all the people in the world, and all their phenotypes (height, eye color, hair type, etc), you could take all their genomes as input variables and treat all their phenotypes as output variables, and make embeddings or other models that mapped from genomes to phenotypes.  The result would be a predictive model that could take a human genome, and spit out a prediction of what that person looks like and other details around them (up to the limits of heritability).<p>A good example is height.  If you take a very large diverse sample of people, and sequence them, you will find that about 50% of the variance in height can be traced to the genomic sequence of that individual (other things, such as socioeconomic status, access to health care, pollution, etc, which are non-genomic, contribute as well).  originally many geneticists believed that a small number of genes- tiny parts of the feature vector- would be the important features in the genome that explained height.<p>But it didn&#x27;t turn out that way.  Instead, height is a nonlinear function of thousands of different locations (either individual bases, entire genes, or other structures that vary between individuals) in the genome.  This was less surprising to folks who are molecular biologists (mainly based on the mental models geneticists and MBers use to think about the mapping of genotype to phenotype), and we still don&#x27;t have great mechanistic explanations of how each individual difference works in concert with all the others to lead to specific heights.<p>When I started out studying this some 35 years ago the problem sounded fairly simple, I assumed it would be easy to find the place in my genome that led to my funny shaped (inherited) nose, but the more I learn about genomics and phenotypes, the more I appreciate that the problem is unbelievably complex, and really well suited to large datasets and machine learning.  All the pharma have petabytes of genome sequences in the cloud that they try hard to analyze but the results are mixed.<p>I spent my entire thesis working on ATGCAAAT, by the way.  <a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Octamer_transcription_factor" rel="nofollow noreferrer">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Octamer_transcription_factor</a> is a family of proteins that are incredibly important during growth and development.  Your genome is sprinkled with locations that contain that sequence- or ones like it- that are used to regulate the expression of proteins to carry out the development plan.</div><br/><div id="37259011" class="c"><input type="checkbox" id="c-37259011" checked=""/><div class="controls bullet"><span class="by">aastronaut</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257820">parent</a><span>|</span><a href="#37257904">next</a><span>|</span><label class="collapse" for="c-37259011">[-]</label><label class="expand" for="c-37259011">[1 more]</label></div><br/><div class="children"><div class="content">&gt; If you had a list of all the genomes of all the people in the world, and all their phenotypes (height, eye color, hair type, etc), you could take all their genomes as input variables and treat all their phenotypes as output variables, and make embeddings or other models that mapped from genomes to phenotypes. The result would be a predictive model that could take a human genome, and spit out a prediction of what that person looks like and other details around them (up to the limits of heritability).<p>Would such a predictive model really be possible? As far as I&#x27;m aware there is contradicting research whether a specific phenotype distinctly originates from a SNP&#x2F;genotype.</div><br/></div></div><div id="37257904" class="c"><input type="checkbox" id="c-37257904" checked=""/><div class="controls bullet"><span class="by">alexchantavy</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257820">parent</a><span>|</span><a href="#37259011">prev</a><span>|</span><a href="#37258973">next</a><span>|</span><label class="collapse" for="c-37257904">[-]</label><label class="expand" for="c-37257904">[8 more]</label></div><br/><div class="children"><div class="content">Fascinating, are there lots of people looking at genetics with this ML kind of lens?</div><br/><div id="37257986" class="c"><input type="checkbox" id="c-37257986" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257904">parent</a><span>|</span><a href="#37258636">next</a><span>|</span><label class="collapse" for="c-37257986">[-]</label><label class="expand" for="c-37257986">[6 more]</label></div><br/><div class="children"><div class="content">Sure, although I&#x27;m not aware of anybody who is contemplating quite the level I believe is necessary to really nail the problem into the ground.  When I worked at Google, I proposed that Google build a datacenter-sized sequencing center in Iowa or Nebraska near its data centers, buy thousands of sequencers, and run industrial-scale sequencing, push the data straight to the cloud over fat fiber, followed by machine learning, for health research.  I don&#x27;t think Google wants to get involved in the physical sequencing part but they did listen to my ideas and they have several teams working on applying ML to genomics as well as other health research problems, and my part of my job today (working at a biotech) is to manage the flows of petabytes of genomic data into the cloud and make it accessible to our machine learning engineers.<p>The really interesting approaches these days, IMHO, combine genomics and microscopic imaging of organoids, and many folks are trying to set up a &quot;lab in the loop&quot;, in which large-scale experiments run autonomously by sophisticated ML systems could accelerate discovery.  It&#x27;s a fractally complex and challenging problem.<p>Statistics has been key to understanding genetics from the beginning (see Mendel, Fisher) and so at a big pharma you will see everything from Bayesian bootstrappers using R to deep learners using pytorch.</div><br/><div id="37259024" class="c"><input type="checkbox" id="c-37259024" checked=""/><div class="controls bullet"><span class="by">2dvisio</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257986">parent</a><span>|</span><a href="#37258654">next</a><span>|</span><label class="collapse" for="c-37259024">[-]</label><label class="expand" for="c-37259024">[1 more]</label></div><br/><div class="children"><div class="content">Guys at Verily are working on Terra.bio with the Broad institute and others. Genomics England in the UK is also experiencing with multimodal and machine learning applied to whole genome sequences [1].<p>[1] <a href="https:&#x2F;&#x2F;www.genomicsengland.co.uk&#x2F;blog&#x2F;data-representations-biomedical-research" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.genomicsengland.co.uk&#x2F;blog&#x2F;data-representations-...</a></div><br/></div></div><div id="37258654" class="c"><input type="checkbox" id="c-37258654" checked=""/><div class="controls bullet"><span class="by">krab</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257986">parent</a><span>|</span><a href="#37259024">prev</a><span>|</span><a href="#37258204">next</a><span>|</span><label class="collapse" for="c-37258654">[-]</label><label class="expand" for="c-37258654">[1 more]</label></div><br/><div class="children"><div class="content">But why Google? This is what big pharma are doing. Also you can outsource the data collection part. See for example UK Biobank. Their data are available to multiple companies after some period so it makes it more cost efficient.</div><br/></div></div><div id="37258204" class="c"><input type="checkbox" id="c-37258204" checked=""/><div class="controls bullet"><span class="by">panosfilianos</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257986">parent</a><span>|</span><a href="#37258654">prev</a><span>|</span><a href="#37258636">next</a><span>|</span><label class="collapse" for="c-37258204">[-]</label><label class="expand" for="c-37258204">[3 more]</label></div><br/><div class="children"><div class="content">I have spent the better part of the past year looking obsessively over genomics papers for cancer and I&#x27;ve grown very fond of the field.<p>Are there any positions at Google&#x2F; companies you wold suggest me to look into? I&#x27;m coming from algortrading&#x2F; ML research with ML MSc.</div><br/><div id="37258479" class="c"><input type="checkbox" id="c-37258479" checked=""/><div class="controls bullet"><span class="by">asielen</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258204">parent</a><span>|</span><a href="#37258636">next</a><span>|</span><label class="collapse" for="c-37258479">[-]</label><label class="expand" for="c-37258479">[2 more]</label></div><br/><div class="children"><div class="content">You could try Calico. They are an Alphabet company that specifically studies aging. There how a good amount of machine learning roles. However biotech typically pays less than finance or software.<p><a href="https:&#x2F;&#x2F;calicolabs.com&#x2F;careers&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;calicolabs.com&#x2F;careers&#x2F;</a></div><br/><div id="37258610" class="c"><input type="checkbox" id="c-37258610" checked=""/><div class="controls bullet"><span class="by">panosfilianos</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258479">parent</a><span>|</span><a href="#37258636">next</a><span>|</span><label class="collapse" for="c-37258610">[-]</label><label class="expand" for="c-37258610">[1 more]</label></div><br/><div class="children"><div class="content">Thanks!</div><br/></div></div></div></div></div></div></div></div><div id="37258636" class="c"><input type="checkbox" id="c-37258636" checked=""/><div class="controls bullet"><span class="by">krab</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257904">parent</a><span>|</span><a href="#37257986">prev</a><span>|</span><a href="#37258973">next</a><span>|</span><label class="collapse" for="c-37258636">[-]</label><label class="expand" for="c-37258636">[1 more]</label></div><br/><div class="children"><div class="content">Yes. For example when word2vec came out, immediately there were people trying similar approaches to protein sequences. Transformers work better.</div><br/></div></div></div></div></div></div><div id="37258973" class="c"><input type="checkbox" id="c-37258973" checked=""/><div class="controls bullet"><span class="by">globular-toast</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257439">parent</a><span>|</span><a href="#37257820">prev</a><span>|</span><a href="#37258863">next</a><span>|</span><label class="collapse" for="c-37258973">[-]</label><label class="expand" for="c-37258973">[1 more]</label></div><br/><div class="children"><div class="content">The genetic code maps nucleotide sequences (DNA) to amino acid sequences (proteins). Every three bases (say AGT) maps to one amino acid. So you can literally read a sequence of ACGTs and decode it into a protein. A sequence that encodes a protein is called a gene.<p>Almost all variations that humans have in their genomes (compared to each other or a reference genome) are tiny, mostly one base differences called single nucleotide polymorphisms (SNPs). These tiny changes encode who you are. The rest of it just makes you carbon-based organism, a eukaryote, an animal, a mammal etc, just like a whole load of other organisms.</div><br/></div></div></div></div><div id="37258863" class="c"><input type="checkbox" id="c-37258863" checked=""/><div class="controls bullet"><span class="by">lopis</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37257439">prev</a><span>|</span><a href="#37258306">next</a><span>|</span><label class="collapse" for="c-37258863">[-]</label><label class="expand" for="c-37258863">[2 more]</label></div><br/><div class="children"><div class="content">While you are correct, the differences between different people&#x27;s DNA is tiny, less 1% at best. So this information is still very valuable. This article is talking about the first time in finishing sequencing one person&#x27;s Y chromosome&#x27;s DNA.</div><br/><div id="37259490" class="c"><input type="checkbox" id="c-37259490" checked=""/><div class="controls bullet"><span class="by">hk__2</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37258863">parent</a><span>|</span><a href="#37258306">next</a><span>|</span><label class="collapse" for="c-37259490">[-]</label><label class="expand" for="c-37259490">[1 more]</label></div><br/><div class="children"><div class="content">&gt; While you are correct, the differences between different people&#x27;s DNA is tiny, less 1% at best<p>How do we know this, if we have only sequenced the chromosome of one individual?</div><br/></div></div></div></div><div id="37258306" class="c"><input type="checkbox" id="c-37258306" checked=""/><div class="controls bullet"><span class="by">jhbadger</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37258863">prev</a><span>|</span><a href="#37258346">next</a><span>|</span><label class="collapse" for="c-37258306">[-]</label><label class="expand" for="c-37258306">[1 more]</label></div><br/><div class="children"><div class="content">Traditional sequences of the Y chromosome (and other chromosomes) were missing parts, particularly the highly repetitive regions called &quot;telomeres&quot;. This is different from the issue of individual variation (although the authors do provide a map of known variations as well).</div><br/></div></div><div id="37258346" class="c"><input type="checkbox" id="c-37258346" checked=""/><div class="controls bullet"><span class="by">dclowd9901</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37258306">prev</a><span>|</span><a href="#37258068">next</a><span>|</span><label class="collapse" for="c-37258346">[-]</label><label class="expand" for="c-37258346">[1 more]</label></div><br/><div class="children"><div class="content">Thank you for putting into words exactly the thing I wanted to understand but couldn’t figure out how to ask.</div><br/></div></div><div id="37258068" class="c"><input type="checkbox" id="c-37258068" checked=""/><div class="controls bullet"><span class="by">photochemsyn</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37258346">prev</a><span>|</span><a href="#37257433">next</a><span>|</span><label class="collapse" for="c-37258068">[-]</label><label class="expand" for="c-37258068">[1 more]</label></div><br/><div class="children"><div class="content">Good question.  Practically they call their complete sequence a &#x27;reference sequence&#x27; which can be thought of as a baseline for comparison to the complete spectrum of human Y chromosome genetic variation, so at least people have <i>something</i> to use as a standard for comparison.  The line in the abstract &quot;mapped available population variation, clinical variants&quot; is about the only mention of the issue.<p>Ideally we&#x27;d have hundreds if not thousands of complete genomes which in total would reveal the population diversity of the human species as it currently exists, but this is a big ask.  &quot;Clincal variants&quot; are of particular interest as those are regions of the genome associated with certain inherited diseases, although the promises of individual genomic knowledge leading to a medical revolution have turned out to be wildly overblown.<p>Since the paper is paywalled, there&#x27;s not much else to say than that they have a (fairly arbitrary in origin, i.e. it could have been from any one individual or possibly even a chimera of several individuals) reference sequence to which other specific human Y chromosomes can be compared, eventually leading to a larger dataset from many individuals which will reveal the highly conserved and highly variable regions of the chromosome, population-wise.</div><br/></div></div><div id="37257433" class="c"><input type="checkbox" id="c-37257433" checked=""/><div class="controls bullet"><span class="by">gorjusborg</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37258068">prev</a><span>|</span><a href="#37258850">next</a><span>|</span><label class="collapse" for="c-37257433">[-]</label><label class="expand" for="c-37257433">[3 more]</label></div><br/><div class="children"><div class="content">I had the same question. Perhaps this will help you.<p><a href="https:&#x2F;&#x2F;en.m.wikipedia.org&#x2F;wiki&#x2F;DNA_sequencing" rel="nofollow noreferrer">https:&#x2F;&#x2F;en.m.wikipedia.org&#x2F;wiki&#x2F;DNA_sequencing</a></div><br/><div id="37257482" class="c"><input type="checkbox" id="c-37257482" checked=""/><div class="controls bullet"><span class="by">mikepurvis</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257433">parent</a><span>|</span><a href="#37258850">next</a><span>|</span><label class="collapse" for="c-37257482">[-]</label><label class="expand" for="c-37257482">[2 more]</label></div><br/><div class="children"><div class="content">That page describes the human genome as having been sequenced back in 2003. *confusion intensifies*</div><br/><div id="37258009" class="c"><input type="checkbox" id="c-37258009" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257407">root</a><span>|</span><a href="#37257482">parent</a><span>|</span><a href="#37258850">next</a><span>|</span><label class="collapse" for="c-37258009">[-]</label><label class="expand" for="c-37258009">[1 more]</label></div><br/><div class="children"><div class="content">the project started around 1990, they announced a draft completed in 2000, &quot;completion&quot; in 2003 (this was more a token announcement based on a threshold than a true milestone).  Even then the scientists knew that major parts of the centromere, telomere, and highly repetitive regions were not fully resolved, and that was fully admitted.  The work by Karen Miga at UCSC and others is more of a mop-up now that genome sequencing is a mature technology and we have much better ways at getting at those tricky regions.<p>another &quot;completion&quot; happened 3 years ago, before this announcement.  but this is the last one.  I promise.</div><br/></div></div></div></div></div></div><div id="37258850" class="c"><input type="checkbox" id="c-37258850" checked=""/><div class="controls bullet"><span class="by">wheelerof4te</span><span>|</span><a href="#37257407">parent</a><span>|</span><a href="#37257433">prev</a><span>|</span><a href="#37257555">next</a><span>|</span><label class="collapse" for="c-37258850">[-]</label><label class="expand" for="c-37258850">[1 more]</label></div><br/><div class="children"><div class="content">It means that they are trying to find a baseline from which they can eventually clone a human being.<p>Let&#x27;s not pretend that this is not an end goal. It always was.</div><br/></div></div></div></div><div id="37257555" class="c"><input type="checkbox" id="c-37257555" checked=""/><div class="controls bullet"><span class="by">space_fountain</span><span>|</span><a href="#37257407">prev</a><span>|</span><a href="#37258307">next</a><span>|</span><label class="collapse" for="c-37257555">[-]</label><label class="expand" for="c-37257555">[2 more]</label></div><br/><div class="children"><div class="content">As I understand it this was hard because most mechanisms we have for sequencing genes work by first splitting large sequences of genes into random chunks and sequencing each chunk individually. These chunks then need arranged properly to form the entire genome. This is possible at all because we can get overlap between the chunks, but when a sequence has repeating sections overlap still isn’t enough to stitch everything together correctly. There’s also a newer technique involving nano pores. The dna is literally pulled through a little hole and the electrical properties change depending on which base pair is inside the pore, sadly it’s liable to errors in a deterministic way depending on the sequence. I don’t have access to the full paper so I’m not quite sure what was done here, but I have heard people talk about combining the two approaches above</div><br/><div id="37258896" class="c"><input type="checkbox" id="c-37258896" checked=""/><div class="controls bullet"><span class="by">jakobnissen</span><span>|</span><a href="#37257555">parent</a><span>|</span><a href="#37258307">next</a><span>|</span><label class="collapse" for="c-37258896">[-]</label><label class="expand" for="c-37258896">[1 more]</label></div><br/><div class="children"><div class="content">They relied mainly on PacBio HiFi reads (which I think is a truly understated revolutionary technology in genomics), then used Nanopore sequencing to link together the HiFi contigs to cover the truly gnarly regions of the chromosome.<p>And yeah, you&#x27;re right: All modern large-scale sequencing is shotgun sequencing where the DNA is randomly broken up, each fragment is sequenced, and then the individual segments (reads) are assembled using a genome assembler.</div><br/></div></div></div></div><div id="37258307" class="c"><input type="checkbox" id="c-37258307" checked=""/><div class="controls bullet"><span class="by">msuvakov</span><span>|</span><a href="#37257555">prev</a><span>|</span><a href="#37258271">next</a><span>|</span><label class="collapse" for="c-37258307">[-]</label><label class="expand" for="c-37258307">[2 more]</label></div><br/><div class="children"><div class="content">This news should be dated back in December, not now.<p>The T2T team published a preprint [1] last December and released the data [2] in March. However, due to the peer review process, the findings have only just been formally published in Nature. The publication timeline can indeed be slow, and in cases like this one, the question is: what&#x27;s the point when all scientists interested in the topic already know about it and working with this assembly?<p>[1] <a href="https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2022.12.01.518724v1.full" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2022.12.01.518724v1....</a>
[2] <a href="https:&#x2F;&#x2F;github.com&#x2F;marbl&#x2F;CHM13">https:&#x2F;&#x2F;github.com&#x2F;marbl&#x2F;CHM13</a></div><br/><div id="37258989" class="c"><input type="checkbox" id="c-37258989" checked=""/><div class="controls bullet"><span class="by">goodcanadian</span><span>|</span><a href="#37258307">parent</a><span>|</span><a href="#37258271">next</a><span>|</span><label class="collapse" for="c-37258989">[-]</label><label class="expand" for="c-37258989">[1 more]</label></div><br/><div class="children"><div class="content"><i>what&#x27;s the point when all scientists interested in the topic already know about it and working with this assembly?</i><p>In this context, I would say the point of a press release is getting the news out generally to non-scientists.</div><br/></div></div></div></div><div id="37258271" class="c"><input type="checkbox" id="c-37258271" checked=""/><div class="controls bullet"><span class="by">coldtea</span><span>|</span><a href="#37258307">prev</a><span>|</span><a href="#37257348">next</a><span>|</span><label class="collapse" for="c-37258271">[-]</label><label class="expand" for="c-37258271">[4 more]</label></div><br/><div class="children"><div class="content">Is this like: &quot;we have a working keyboard driver&quot; or more like &quot;we identified all 104 keys on a standard 104-key US QWERTY layout&quot;?</div><br/><div id="37258287" class="c"><input type="checkbox" id="c-37258287" checked=""/><div class="controls bullet"><span class="by">gandalfgreybeer</span><span>|</span><a href="#37258271">parent</a><span>|</span><a href="#37257348">next</a><span>|</span><label class="collapse" for="c-37258287">[-]</label><label class="expand" for="c-37258287">[3 more]</label></div><br/><div class="children"><div class="content">Not answering your question, but I have to say, this is one of the best analogies I&#x27;ve encountered in a while.</div><br/><div id="37258587" class="c"><input type="checkbox" id="c-37258587" checked=""/><div class="controls bullet"><span class="by">quickthrower2</span><span>|</span><a href="#37258271">root</a><span>|</span><a href="#37258287">parent</a><span>|</span><a href="#37258397">next</a><span>|</span><label class="collapse" for="c-37258587">[-]</label><label class="expand" for="c-37258587">[1 more]</label></div><br/><div class="children"><div class="content">The keyboard driver means knowing what effect every key has?</div><br/></div></div><div id="37258397" class="c"><input type="checkbox" id="c-37258397" checked=""/><div class="controls bullet"><span class="by">coldtea</span><span>|</span><a href="#37258271">root</a><span>|</span><a href="#37258287">parent</a><span>|</span><a href="#37258587">prev</a><span>|</span><a href="#37257348">next</a><span>|</span><label class="collapse" for="c-37258397">[-]</label><label class="expand" for="c-37258397">[1 more]</label></div><br/><div class="children"><div class="content">Wow, thanks!</div><br/></div></div></div></div></div></div><div id="37257348" class="c"><input type="checkbox" id="c-37257348" checked=""/><div class="controls bullet"><span class="by">brayman30</span><span>|</span><a href="#37258271">prev</a><span>|</span><a href="#37257437">next</a><span>|</span><label class="collapse" for="c-37257348">[-]</label><label class="expand" for="c-37257348">[13 more]</label></div><br/><div class="children"><div class="content">This is actually kind of a huge deal since it means that all 24 chromosomes have now been fully sequenced. As it says in the abstract, up until now the Y chromosome proved difficult to sequence due to its nature.</div><br/><div id="37258168" class="c"><input type="checkbox" id="c-37258168" checked=""/><div class="controls bullet"><span class="by">AgentME</span><span>|</span><a href="#37257348">parent</a><span>|</span><a href="#37258196">next</a><span>|</span><label class="collapse" for="c-37258168">[-]</label><label class="expand" for="c-37258168">[2 more]</label></div><br/><div class="children"><div class="content">I thought the human genome project finished in 2003, but now that I look it up again, apparently &quot;finished&quot; meant 92%. And then it was finished in 2022. And now it&#x27;s finished again. Are there any remaining milestones left toward maybe finishing it again? Was the Y chromosome really just considered an exception until now?</div><br/><div id="37258443" class="c"><input type="checkbox" id="c-37258443" checked=""/><div class="controls bullet"><span class="by">dclowd9901</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37258168">parent</a><span>|</span><a href="#37258196">next</a><span>|</span><label class="collapse" for="c-37258443">[-]</label><label class="expand" for="c-37258443">[1 more]</label></div><br/><div class="children"><div class="content">human_genome_final_FINAL_3.txt</div><br/></div></div></div></div><div id="37258196" class="c"><input type="checkbox" id="c-37258196" checked=""/><div class="controls bullet"><span class="by">tobinfricke</span><span>|</span><a href="#37257348">parent</a><span>|</span><a href="#37258168">prev</a><span>|</span><a href="#37257452">next</a><span>|</span><label class="collapse" for="c-37258196">[-]</label><label class="expand" for="c-37258196">[5 more]</label></div><br/><div class="children"><div class="content">What makes the Y chromosome more difficult?</div><br/><div id="37258986" class="c"><input type="checkbox" id="c-37258986" checked=""/><div class="controls bullet"><span class="by">bux93</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37258196">parent</a><span>|</span><a href="#37259236">next</a><span>|</span><label class="collapse" for="c-37258986">[-]</label><label class="expand" for="c-37258986">[1 more]</label></div><br/><div class="children"><div class="content">It&#x27;s on line 1 of the linked article: &quot;The human Y chromosome has been notoriously difficult to sequence and assemble because of its complex repeat structure that includes long palindromes, tandem repeats and segmental duplications&quot; and comes with 3 citations.</div><br/></div></div><div id="37259236" class="c"><input type="checkbox" id="c-37259236" checked=""/><div class="controls bullet"><span class="by">BbzzbB</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37258196">parent</a><span>|</span><a href="#37258986">prev</a><span>|</span><a href="#37259070">next</a><span>|</span><label class="collapse" for="c-37259236">[-]</label><label class="expand" for="c-37259236">[2 more]</label></div><br/><div class="children"><div class="content">Maybe someone can correct the details since it&#x27;s been a few years I done this, but we sequence DNA by PCR. Roughly, (1) breaking it up in small pieces and split strands, (2) mixing it with an enzyme that completes each single strand, (3) repeat 1 and 2 a bunch to multiply the strands many times over to make the solution a sense DNA juice, (4) pass it through a machine that&#x27;ll sequence thousands of these small strands and (5) align these short DNA sequences with a software that matches unique sequences.<p>I did it with COI gene, which is just a short (1000ish base pairs with our snails IIRC) sequence of purely random ATGC base pairs. Lots of unique sequences make the short strands easy to match, just get a bunch of 10-15 BP bits and you can match the whole thing.<p>Now if your gene is 62M BP of repeating palindrome sequences, you can imagine how hard it would be to align random pieces sequenced as it will be very hard to find unique sequences to match.</div><br/><div id="37259269" class="c"><input type="checkbox" id="c-37259269" checked=""/><div class="controls bullet"><span class="by">inciampati</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37259236">parent</a><span>|</span><a href="#37259070">next</a><span>|</span><label class="collapse" for="c-37259269">[-]</label><label class="expand" for="c-37259269">[1 more]</label></div><br/><div class="children"><div class="content">We don&#x27;t use PCR anymore! It&#x27;s direct sequencing of the primary DNA. We can read single molecules. That&#x27;s the quiet revolution in nanotechnology that&#x27;s driving all these complete assemblies.</div><br/></div></div></div></div></div></div><div id="37257452" class="c"><input type="checkbox" id="c-37257452" checked=""/><div class="controls bullet"><span class="by">bonestamp2</span><span>|</span><a href="#37257348">parent</a><span>|</span><a href="#37258196">prev</a><span>|</span><a href="#37257562">next</a><span>|</span><label class="collapse" for="c-37257452">[-]</label><label class="expand" for="c-37257452">[4 more]</label></div><br/><div class="children"><div class="content">As others have asked in this thread, what does &quot;fully sequenced&quot; mean to the layman?</div><br/><div id="37257610" class="c"><input type="checkbox" id="c-37257610" checked=""/><div class="controls bullet"><span class="by">yellowcake0</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37257452">parent</a><span>|</span><a href="#37257628">next</a><span>|</span><label class="collapse" for="c-37257610">[-]</label><label class="expand" for="c-37257610">[2 more]</label></div><br/><div class="children"><div class="content">It looks like from the preprint that they sequenced the Y chromosome of HG002, which was one of the original 1000 genomes samples from way back in the day, still held in deep freeze at a number of biobanks.<p>Short-read sequencing data is a notoriously bad datatype for reconstructing the low-complexity &#x2F; repetitive regions of genomes, so up until recently the most commonly used reference genomes have left many of these regions &quot;dark&quot;. According to the preprint, the Y chromosome has the highest density of these low-complexity regions. It&#x27;s also something of a bioinformatic nuisance when constructing a generic human reference genome, as it&#x27;s only present in 50% of the population.</div><br/><div id="37259513" class="c"><input type="checkbox" id="c-37259513" checked=""/><div class="controls bullet"><span class="by">scandum</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37257610">parent</a><span>|</span><a href="#37257628">next</a><span>|</span><label class="collapse" for="c-37259513">[-]</label><label class="expand" for="c-37259513">[1 more]</label></div><br/><div class="children"><div class="content">Isn&#x27;t the problem the absence of random DNA?<p>I wouldn&#x27;t call random data &#x27;complex&#x27;, but it is easy to sequence when assembling short reads.</div><br/></div></div></div></div><div id="37257628" class="c"><input type="checkbox" id="c-37257628" checked=""/><div class="controls bullet"><span class="by">icholy</span><span>|</span><a href="#37257348">root</a><span>|</span><a href="#37257452">parent</a><span>|</span><a href="#37257610">prev</a><span>|</span><a href="#37257562">next</a><span>|</span><label class="collapse" for="c-37257628">[-]</label><label class="expand" for="c-37257628">[1 more]</label></div><br/><div class="children"><div class="content">It provides a complete baseline&#x2F;reference DNA &quot;map&quot;. Common &quot;problems&quot; show up on specific parts of this map. So you can sequence small subsets of a patient&#x27;s DNA and compare it to these &quot;problem areas&quot; to detect genetic diseases.</div><br/></div></div></div></div><div id="37257562" class="c"><input type="checkbox" id="c-37257562" checked=""/><div class="controls bullet"><span class="by">p1mrx</span><span>|</span><a href="#37257348">parent</a><span>|</span><a href="#37257452">prev</a><span>|</span><a href="#37257437">next</a><span>|</span><label class="collapse" for="c-37257562">[-]</label><label class="expand" for="c-37257562">[1 more]</label></div><br/><div class="children"><div class="content">You can tell it&#x27;s the Y chromosome because of the way it is.</div><br/></div></div></div></div><div id="37257437" class="c"><input type="checkbox" id="c-37257437" checked=""/><div class="controls bullet"><span class="by">UI_at_80x24</span><span>|</span><a href="#37257348">prev</a><span>|</span><a href="#37258598">next</a><span>|</span><label class="collapse" for="c-37257437">[-]</label><label class="expand" for="c-37257437">[4 more]</label></div><br/><div class="children"><div class="content">For a very entertaining and educational book that tangentially related; I can highly recommend:  Y - The Descent of Man by Steve Jones [0]<p>I am thrilled to see more chromosomes being mapped&#x2F;sequenced.  Please excuse my high-school level of biology knowledge here, but have we definitively progressed beyond correlation when it comes to genes, gene expression, and how they all interact?<p>Take &quot;Blue eyes&quot; as an example,  we know that [1] OCA2 is responsible for brown&#x2F;blue eye colour.  BUT are we sure that none of the 20,000 others are involved&#x2F;needed?<p>In laymans terms I would guess that to get &#x27;blue eyes&#x27; would require several genes to ALL be on&#x2F;active&#x2F;present as well.  More like a recipe, then a simple on&#x2F;off it&#x27;s located in 1 position.<p>[0] <a href="https:&#x2F;&#x2F;www.goodreads.com&#x2F;work&#x2F;editions&#x2F;436071-y-the-descent-of-men" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.goodreads.com&#x2F;work&#x2F;editions&#x2F;436071-y-the-descent...</a>
[1] <a href="https:&#x2F;&#x2F;www.nature.com&#x2F;articles&#x2F;s41433-021-01749-x" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nature.com&#x2F;articles&#x2F;s41433-021-01749-x</a></div><br/><div id="37257893" class="c"><input type="checkbox" id="c-37257893" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257437">parent</a><span>|</span><a href="#37257476">next</a><span>|</span><label class="collapse" for="c-37257893">[-]</label><label class="expand" for="c-37257893">[1 more]</label></div><br/><div class="children"><div class="content">Eye color is subtle.  How do you define blue eyes?  It&#x27;s not just one phenotype; there are a range of blue eyes, whose differences correspond to fairly subtle variations between people.  The article you linked [1], is quite good but requires quite some time to absorb completely.</div><br/></div></div><div id="37257476" class="c"><input type="checkbox" id="c-37257476" checked=""/><div class="controls bullet"><span class="by">largbae</span><span>|</span><a href="#37257437">parent</a><span>|</span><a href="#37257893">prev</a><span>|</span><a href="#37257712">next</a><span>|</span><label class="collapse" for="c-37257476">[-]</label><label class="expand" for="c-37257476">[1 more]</label></div><br/><div class="children"><div class="content">I would imagine all sorts of genes are necessary to have blue eyes depending on how you look at it. After all, you need eyes in order to have blue eyes, and a whole bunch of other machinery to open them to check the color...</div><br/></div></div><div id="37257712" class="c"><input type="checkbox" id="c-37257712" checked=""/><div class="controls bullet"><span class="by">brayman30</span><span>|</span><a href="#37257437">parent</a><span>|</span><a href="#37257476">prev</a><span>|</span><a href="#37258598">next</a><span>|</span><label class="collapse" for="c-37257712">[-]</label><label class="expand" for="c-37257712">[1 more]</label></div><br/><div class="children"><div class="content">First I would like to state that human genetics are very, very complex and we still do not have a complete understanding of them. For example, only relatively recently has the field of epigenetics made major advancementa, which (broadly speaking) studies how a cell&#x27;s behavior can be changed WITHOUT it&#x27;s DNA being changed.[1] This brings into question the age-old questions about environmental impact vs genetic impact in even greater detail. Anyways, sidebar over.<p>No, we are not completely sure how many chromosomes play a role in determining eye color. However, we do have a pretty good guess. Most recent estimates I found out the number at 16. OCA2 and HERC2 have the largest impact on eye color, but, there are many OTHER genes that also have smaller impacts on eye color. [2] That article I cited is actually amazing. But, it does touch on some more advanced subjects that are more introductory college-level biology or AP Biology than standard high school bio e.g. Gene Regulation, introns and extrons, etc.<p>To answer your more general question, in my (admittedly only mildly less basic, introductory college-level biology) opinion, it is unlikely that we will, anytime soon, reach a point where anything in genetics can be completly, 100% definitive.
This is not to say that we haven&#x27;t made amazing advancements in the field of genetics and biology more broadly. BUT, it would be a mistake to take for granted the complexity of the human genome. There is most definitely things we still do not understand about the genome, and will not for some time.<p>But, practically speaking, while other genes may have impact on some simple phenotypic traits, such as eye color, we can generally make an accurate guess based on only a few genes. In eye color, for example, one study was able to predict eye color with only 6 genes with about ~75% accuracy. [3]<p>The crazy part is, sometimes the genes that effect the phenotypic trait, don&#x27;t actually store genetic material that determines the trait. In other words, they don&#x27;t DIRECTLY determine the trait at all. Rather, the only effect ORHER genes which then effect the expression of the trait more directly. You can imagine that this could become very complex very very quickly when you have multiple genes effecting other genes which effect other genes, not to mention accoubting for environmental and demographic baises when doing these studies and you begin to see why this genetics is such a difficult field of study.<p>Apologies for the long post and rambling. Hopefully I was still able to provide you with some mediocre introductory-collage-level biology<p>[1] <a href="https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC2791696&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC2791696&#x2F;</a>
[2] <a href="https:&#x2F;&#x2F;www.nature.com&#x2F;articles&#x2F;jhg2010126" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.nature.com&#x2F;articles&#x2F;jhg2010126</a>
[3] <a href="https:&#x2F;&#x2F;doi.org&#x2F;10.1016&#x2F;j.cub.2009.01.027" rel="nofollow noreferrer">https:&#x2F;&#x2F;doi.org&#x2F;10.1016&#x2F;j.cub.2009.01.027</a></div><br/></div></div></div></div><div id="37258598" class="c"><input type="checkbox" id="c-37258598" checked=""/><div class="controls bullet"><span class="by">ManuelKiessling</span><span>|</span><a href="#37257437">prev</a><span>|</span><a href="#37257728">next</a><span>|</span><label class="collapse" for="c-37258598">[-]</label><label class="expand" for="c-37258598">[3 more]</label></div><br/><div class="children"><div class="content">So, did they find the reason why BMW drivers are always tailgating me on the Autobahn?</div><br/><div id="37258670" class="c"><input type="checkbox" id="c-37258670" checked=""/><div class="controls bullet"><span class="by">wallaBBB</span><span>|</span><a href="#37258598">parent</a><span>|</span><a href="#37258662">next</a><span>|</span><label class="collapse" for="c-37258670">[-]</label><label class="expand" for="c-37258670">[1 more]</label></div><br/><div class="children"><div class="content">Yes, move out of the left lane.</div><br/></div></div><div id="37258662" class="c"><input type="checkbox" id="c-37258662" checked=""/><div class="controls bullet"><span class="by">jacquesm</span><span>|</span><a href="#37258598">parent</a><span>|</span><a href="#37258670">prev</a><span>|</span><a href="#37257728">next</a><span>|</span><label class="collapse" for="c-37258662">[-]</label><label class="expand" for="c-37258662">[1 more]</label></div><br/><div class="children"><div class="content">Because you stay in the left lane when you shouldn&#x27;t?</div><br/></div></div></div></div><div id="37257728" class="c"><input type="checkbox" id="c-37257728" checked=""/><div class="controls bullet"><span class="by">mrazomor</span><span>|</span><a href="#37258598">prev</a><span>|</span><a href="#37259382">next</a><span>|</span><label class="collapse" for="c-37257728">[-]</label><label class="expand" for="c-37257728">[2 more]</label></div><br/><div class="children"><div class="content">This is exciting!<p>BTW. if you want to know the applications of this work, have a look at this ACM SIGPLAN Keynote: <a href="https:&#x2F;&#x2F;youtu.be&#x2F;JTU3JYp3JYc?si=jOZz611ATQar3Gec" rel="nofollow noreferrer">https:&#x2F;&#x2F;youtu.be&#x2F;JTU3JYp3JYc?si=jOZz611ATQar3Gec</a> (helped me understand DNA more than all biology classes at my high school)</div><br/><div id="37258517" class="c"><input type="checkbox" id="c-37258517" checked=""/><div class="controls bullet"><span class="by">dramebaaz</span><span>|</span><a href="#37257728">parent</a><span>|</span><a href="#37259382">next</a><span>|</span><label class="collapse" for="c-37258517">[-]</label><label class="expand" for="c-37258517">[1 more]</label></div><br/><div class="children"><div class="content">Thanks for that ACM SIGPLAN talk link, it was very fascinating and engaging.</div><br/></div></div></div></div><div id="37259382" class="c"><input type="checkbox" id="c-37259382" checked=""/><div class="controls bullet"><span class="by">catsarebetter</span><span>|</span><a href="#37257728">prev</a><span>|</span><a href="#37258298">next</a><span>|</span><label class="collapse" for="c-37259382">[-]</label><label class="expand" for="c-37259382">[1 more]</label></div><br/><div class="children"><div class="content">omg and with the new tech around generative AI...<p>This is shaping up to be a very interesting decade to be in tech.<p>We might actually be able to generate new sequences and test them against all the diseases to remove them.</div><br/></div></div><div id="37257651" class="c"><input type="checkbox" id="c-37257651" checked=""/><div class="controls bullet"><span class="by">soligern</span><span>|</span><a href="#37258298">prev</a><span>|</span><a href="#37257339">next</a><span>|</span><label class="collapse" for="c-37257651">[-]</label><label class="expand" for="c-37257651">[2 more]</label></div><br/><div class="children"><div class="content">Does this literally mean every single base pair with all the junk genome and everything else? Or is it some statistical model&#x2F;extrapolated etc.</div><br/><div id="37258038" class="c"><input type="checkbox" id="c-37258038" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#37257651">parent</a><span>|</span><a href="#37257339">next</a><span>|</span><label class="collapse" for="c-37258038">[-]</label><label class="expand" for="c-37258038">[1 more]</label></div><br/><div class="children"><div class="content">It is a completely gapless assembly from the beginning of the start of each chromosome all the way to it&#x27;s end (&quot;telomere to telomere&quot;).  I haven&#x27;t read in enough detail but I would imagine there are still some slightly fuzzy regions that are sort of a best-guess, although those would be quite small.</div><br/></div></div></div></div><div id="37257339" class="c"><input type="checkbox" id="c-37257339" checked=""/><div class="controls bullet"><span class="by">muizelaar</span><span>|</span><a href="#37257651">prev</a><span>|</span><a href="#37258482">next</a><span>|</span><label class="collapse" for="c-37257339">[-]</label><label class="expand" for="c-37257339">[1 more]</label></div><br/><div class="children"><div class="content">The preprint: <a href="https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2022.12.01.518724v2.full.pdf" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2022.12.01.518724v2....</a></div><br/></div></div><div id="37258482" class="c"><input type="checkbox" id="c-37258482" checked=""/><div class="controls bullet"><span class="by">farhanhubble</span><span>|</span><a href="#37257339">prev</a><span>|</span><a href="#37258679">next</a><span>|</span><label class="collapse" for="c-37258482">[-]</label><label class="expand" for="c-37258482">[1 more]</label></div><br/><div class="children"><div class="content">Does it make it any easier to snapshot Chromosomes and see them evolve with age and perhaps better correlate that with the onset of diseases? The correlation is still possible but will having multiple snapshots enable better association?</div><br/></div></div><div id="37258679" class="c"><input type="checkbox" id="c-37258679" checked=""/><div class="controls bullet"><span class="by">dools</span><span>|</span><a href="#37258482">prev</a><span>|</span><a href="#37257954">next</a><span>|</span><label class="collapse" for="c-37258679">[-]</label><label class="expand" for="c-37258679">[1 more]</label></div><br/><div class="children"><div class="content">Scary cowboy on Sesame Street: I WANNA KNOW Y!<p>Scientists: here you go</div><br/></div></div><div id="37257954" class="c"><input type="checkbox" id="c-37257954" checked=""/><div class="controls bullet"><span class="by">moelf</span><span>|</span><a href="#37258679">prev</a><span>|</span><a href="#37257589">next</a><span>|</span><label class="collapse" for="c-37257954">[-]</label><label class="expand" for="c-37257954">[1 more]</label></div><br/><div class="children"><div class="content">I guess nih will update this soon: <a href="https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;assembly&#x2F;GCF_009914755.1&#x2F;" rel="nofollow noreferrer">https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;assembly&#x2F;GCF_009914755.1&#x2F;</a> since it was gap-less except for Y chromosome</div><br/></div></div><div id="37257589" class="c"><input type="checkbox" id="c-37257589" checked=""/><div class="controls bullet"><span class="by">mikeyouse</span><span>|</span><a href="#37257954">prev</a><span>|</span><a href="#37257947">next</a><span>|</span><label class="collapse" for="c-37257589">[-]</label><label class="expand" for="c-37257589">[1 more]</label></div><br/><div class="children"><div class="content">A good article about the achievement from UC Santa Cruz, where one of the lead scientists of the Telomere-to-Telomere project works:<p><a href="https:&#x2F;&#x2F;news.ucsc.edu&#x2F;2023&#x2F;08&#x2F;t2t-y-chromosome.html" rel="nofollow noreferrer">https:&#x2F;&#x2F;news.ucsc.edu&#x2F;2023&#x2F;08&#x2F;t2t-y-chromosome.html</a></div><br/></div></div><div id="37257947" class="c"><input type="checkbox" id="c-37257947" checked=""/><div class="controls bullet"><span class="by">xwdv</span><span>|</span><a href="#37257589">prev</a><span>|</span><a href="#37258664">next</a><span>|</span><label class="collapse" for="c-37257947">[-]</label><label class="expand" for="c-37257947">[3 more]</label></div><br/><div class="children"><div class="content">Could we in theory just build up any kind of creature from scratch if we write out its genome? How would we compile it into organic matter?</div><br/><div id="37258930" class="c"><input type="checkbox" id="c-37258930" checked=""/><div class="controls bullet"><span class="by">jakobnissen</span><span>|</span><a href="#37257947">parent</a><span>|</span><a href="#37259066">next</a><span>|</span><label class="collapse" for="c-37258930">[-]</label><label class="expand" for="c-37258930">[1 more]</label></div><br/><div class="children"><div class="content">No, this is not technically possible. Cells are simply to complex to just build using humanity&#x27;s current level of technology.
The closest thing we have done is to fully synthesize a bacterium&#x27;s genome, remove the genome from a bacterial cell, then transplant the synthetic genome into the cell and have the cell survive: <a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Mycoplasma_laboratorium" rel="nofollow noreferrer">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Mycoplasma_laboratorium</a><p>That&#x27;s insanely cool, but very far from what you are asking. An analogy: We have re-installed the operating system of an existing computer, when you are asking whether we can manufacture a computer.</div><br/></div></div><div id="37259066" class="c"><input type="checkbox" id="c-37259066" checked=""/><div class="controls bullet"><span class="by">globular-toast</span><span>|</span><a href="#37257947">parent</a><span>|</span><a href="#37258930">prev</a><span>|</span><a href="#37258664">next</a><span>|</span><label class="collapse" for="c-37259066">[-]</label><label class="expand" for="c-37259066">[1 more]</label></div><br/><div class="children"><div class="content">There&#x27;s a bootstrapping problem. You need something to run the code. An egg cell and sperm cell contain loads of other stuff in addition to the genome and the only way to get that stuff is from the genome...<p>It&#x27;s like if you had a program to 3D print a 3D printer but you don&#x27;t have a 3D printer that can read that program.<p>It&#x27;s all ridiculously complicated. To get an idea about it you could learn about how simple RNA viruses like influenza or HIV replicate. It&#x27;s easy enough for anyone to understand. But no single person can fully understand humans (or other mammals, plants etc, we&#x27;re not special in that respect).</div><br/></div></div></div></div><div id="37258664" class="c"><input type="checkbox" id="c-37258664" checked=""/><div class="controls bullet"><span class="by">renewiltord</span><span>|</span><a href="#37257947">prev</a><span>|</span><a href="#37258264">next</a><span>|</span><label class="collapse" for="c-37258664">[-]</label><label class="expand" for="c-37258664">[2 more]</label></div><br/><div class="children"><div class="content">So I have my whole genome sequenced by Nebula. What do I have to do to match it up?</div><br/><div id="37258729" class="c"><input type="checkbox" id="c-37258729" checked=""/><div class="controls bullet"><span class="by">mfld</span><span>|</span><a href="#37258664">parent</a><span>|</span><a href="#37258264">next</a><span>|</span><label class="collapse" for="c-37258729">[-]</label><label class="expand" for="c-37258729">[1 more]</label></div><br/><div class="children"><div class="content">You could download the raw sequence reads (FASTQ files) and map them to the new T2T-Y. Probably it makes more sense to wait for a new release of the T2T human genome reference, as mapping to a subset of the genome increases false positives. It helps to have some genome bioinformatics knowledge to correctly handle those T2T references.</div><br/></div></div></div></div><div id="37258264" class="c"><input type="checkbox" id="c-37258264" checked=""/><div class="controls bullet"><span class="by">hanniabu</span><span>|</span><a href="#37258664">prev</a><span>|</span><a href="#37257354">next</a><span>|</span><label class="collapse" for="c-37258264">[-]</label><label class="expand" for="c-37258264">[1 more]</label></div><br/><div class="children"><div class="content">What are the implications of this? What&#x27;s now possible with this feat completed?</div><br/></div></div><div id="37257354" class="c"><input type="checkbox" id="c-37257354" checked=""/><div class="controls bullet"><span class="by">Legend2440</span><span>|</span><a href="#37258264">prev</a><span>|</span><a href="#37257499">next</a><span>|</span><label class="collapse" for="c-37257354">[-]</label><label class="expand" for="c-37257354">[1 more]</label></div><br/><div class="children"><div class="content">TIL the human Y chromosome hadn&#x27;t been completely sequenced until now.</div><br/></div></div></div></div></div></div></div></body></html>