<!DOCTYPE html><html lang="en"><head><title>Static News</title><meta charSet="utf-8"/><meta name="description" content="Static delayed Hacker News."/><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"/><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1d1f21"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><meta name="application-name" content="Static News"/><meta name="apple-mobile-web-app-title" content="Static News"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="#1d1f21"/><link rel="preload" href="styles.css?v=1707210060341" as="style"/><link rel="stylesheet" href="styles.css?v=1707210060341"/></head><body><div id="container"><div id="inner"><header><a href="/">Static News</a><a href="/about">about</a></header><div id="content"><div><div id="title"><a href="http://omicsomics.blogspot.com/2024/02/want-to-build-sequencer-454bio-opens-up.html">Want to build a sequencer? 454.bio opens up their plans</a> <span class="domain">(<a href="http://omicsomics.blogspot.com">omicsomics.blogspot.com</a>)</span></div><div class="subtext"><span>mfld</span> | <span>52 comments</span></div><br/><div><div id="39267402" class="c"><input type="checkbox" id="c-39267402" checked=""/><div class="controls bullet"><span class="by">epistasis</span><span>|</span><a href="#39267853">next</a><span>|</span><label class="collapse" for="c-39267402">[-]</label><label class="expand" for="c-39267402">[9 more]</label></div><br/><div class="children"><div class="content">The most interesting thing here is the open-source bio startup aspect. This is a bold step, and I hope it prompts more startups to go this direction.<p>They are clearly at the early stage of tech dev. And the reward of $200k for the polymerase to allow longer reads seems pretty low to me... unless you&#x27;re a grad student and the university doenst take your IP...<p>Honestly it&#x27;s pretty hard to get lab access without somebody also getting rights to your IP, and labs are a bit more expensive than setting up an H100, so it might be tough going.<p>Even if no external party solves their polymerase problem I have a good feeling about this direction though. Godspeed.</div><br/><div id="39268023" class="c"><input type="checkbox" id="c-39268023" checked=""/><div class="controls bullet"><span class="by">ak217</span><span>|</span><a href="#39267402">parent</a><span>|</span><a href="#39267853">next</a><span>|</span><label class="collapse" for="c-39268023">[-]</label><label class="expand" for="c-39268023">[8 more]</label></div><br/><div class="children"><div class="content">$200k for a better polymerase was definitely way too low a decade ago, when PacBio had an enzymology group with a multi-million $ budget running crystallography at SLAC to build a better polymerase.<p>On the other hand, $200k might be enough today if someone finds a way to apply AlphaFold and friends to this problem.<p>On the other other hand, I&#x27;m not convinced a better polymerase will solve their problems. Photobleaching might be their biggest problem, and I don&#x27;t know that the polymerase itself can do much to protect against that.</div><br/><div id="39268202" class="c"><input type="checkbox" id="c-39268202" checked=""/><div class="controls bullet"><span class="by">koeng</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39268023">parent</a><span>|</span><a href="#39269162">next</a><span>|</span><label class="collapse" for="c-39268202">[-]</label><label class="expand" for="c-39268202">[6 more]</label></div><br/><div class="children"><div class="content">The most annoying part with the 200k polymerase prize is that they don&#x27;t actually provide protocols for validating the polymerase works well. Me and some friends can definitely do the AI-protein design + DNA synthesis + protein synthesis (and would want to), but no way am I gonna figure out idiosyncratic sequencer protocols.</div><br/><div id="39268553" class="c"><input type="checkbox" id="c-39268553" checked=""/><div class="controls bullet"><span class="by">mbreese</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39268202">parent</a><span>|</span><a href="#39269068">next</a><span>|</span><label class="collapse" for="c-39268553">[-]</label><label class="expand" for="c-39268553">[3 more]</label></div><br/><div class="children"><div class="content">The lack of a validation protocol is also an issue for the another way for doing this — generationally mutate the polymerase and let evolution do the work for you. But, this only really works when you have an automated test to know which variations are working better than others. You really want to be able to automate the process to get many (many) generations to effectively explore the search space.<p>But, the contest isn’t really about length — it’s about polymerase preference for fluorescently labeled dNTPs vs unlabeled.<p>From: <a href="https:&#x2F;&#x2F;454.bio&#x2F;blog&#x2F;2024&#x2F;01&#x2F;23&#x2F;polymerase-competition&#x2F;" rel="nofollow">https:&#x2F;&#x2F;454.bio&#x2F;blog&#x2F;2024&#x2F;01&#x2F;23&#x2F;polymerase-competition&#x2F;</a><p><i>&gt; 454 Bio have opened a competition with a prize of $200,000.00 to evolve a DNA polymerase optimized to use Lightning Terminators™ while not incorporating their hydroxymethyl cleavage products with a minimum 1000:1 ratio. </i><p>This seems to be an issue with their chosen “One Pot” reaction mixture. When you can’t wash away byproducts after each cycle, with this method, you get a build up of non-labeled molecules. And these are the ones that the polymerase prefers. So, of course they end up with single-digit read lengths… the polymerase is doing what it does best. This seems like figuring out this chemistry should have been fleshed out before trying to build the sequencer.<p>So, I guess it turns out that there is a validation endpoint, but still not a very convenient way to measure it…</div><br/><div id="39269855" class="c"><input type="checkbox" id="c-39269855" checked=""/><div class="controls bullet"><span class="by">ak217</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39268553">parent</a><span>|</span><a href="#39269068">next</a><span>|</span><label class="collapse" for="c-39269855">[-]</label><label class="expand" for="c-39269855">[2 more]</label></div><br/><div class="children"><div class="content">Yeah, now that you mention it, that seems like a major issue. Of course unlabeled nucleotides will outcompete labeled ones, they&#x27;re smaller so they get less steric hindrance. And I don&#x27;t understand what can be done about it at the polymerase binding site. Maybe some induced proximity chemistry?</div><br/><div id="39270460" class="c"><input type="checkbox" id="c-39270460" checked=""/><div class="controls bullet"><span class="by">mbreese</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39269855">parent</a><span>|</span><a href="#39269068">next</a><span>|</span><label class="collapse" for="c-39270460">[-]</label><label class="expand" for="c-39270460">[1 more]</label></div><br/><div class="children"><div class="content">Right? The unlabeled nucleotide will still fit in a binding pocket for the labeled version. And it should always have better binding.<p>But this is an obvious issue. With this type of photo-reaction, you’ll get a build up of unlabeled nucleotides. There is still some potential for the technology, but it’s really hard for me to see how this works in one tube without some kind of washing (which also increases costs by a lot).<p>Maybe there is a way to add multiple moieties to the NTPs, such that removal of the fluorophore would cause a bigger hindrance? (Fluorophore + extra-moiety would bind, but extra-moiety alone would have more trouble binding?)</div><br/></div></div></div></div></div></div><div id="39269068" class="c"><input type="checkbox" id="c-39269068" checked=""/><div class="controls bullet"><span class="by">anonymouskimmer</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39268202">parent</a><span>|</span><a href="#39268553">prev</a><span>|</span><a href="#39269162">next</a><span>|</span><label class="collapse" for="c-39269068">[-]</label><label class="expand" for="c-39269068">[2 more]</label></div><br/><div class="children"><div class="content">&gt; AI-protein design<p>Any recommendations on learning how to do this? As well as the limitations to your approach? I&#x27;ve been interested in engineering site-specific DNA binding proteins for over a decade now, but the tools I&#x27;ve looked at are specific for protein to small-molecule or protein to protein interfaces.</div><br/></div></div></div></div><div id="39269162" class="c"><input type="checkbox" id="c-39269162" checked=""/><div class="controls bullet"><span class="by">mbreese</span><span>|</span><a href="#39267402">root</a><span>|</span><a href="#39268023">parent</a><span>|</span><a href="#39268202">prev</a><span>|</span><a href="#39267853">next</a><span>|</span><label class="collapse" for="c-39269162">[-]</label><label class="expand" for="c-39269162">[1 more]</label></div><br/><div class="children"><div class="content">$200k might be enough to generate a few candidate designs (in silico), but still probably not enough to actually test&#x2F;validate them.</div><br/></div></div></div></div></div></div><div id="39267853" class="c"><input type="checkbox" id="c-39267853" checked=""/><div class="controls bullet"><span class="by">ak217</span><span>|</span><a href="#39267402">prev</a><span>|</span><a href="#39267710">next</a><span>|</span><label class="collapse" for="c-39267853">[-]</label><label class="expand" for="c-39267853">[1 more]</label></div><br/><div class="children"><div class="content">This is super interesting and a promising development for those of us dreaming of running a lab out of a garage. I&#x27;ve never worked with Rothberg directly, but I&#x27;ve read great stories of him cultivating entrepreneural talent in molecular biology.<p>When PacBio was scaling their technology, they had lots of problems with photobleaching. It turns out shining intense laser light at a polymerase-nucleotide-fluorophore complex and inducing lots of electron flux in the fluorophore eventually causes an electron to get overexcited, jump around, and damage the machinery. PacBio eventually invested a lot in builidng elaborate shields into the nucleotide-fluorophore link, able to absorb the stray electrons and save the polymerase.<p>Given that experience, I&#x27;d expect shining intense UV light into a TIRF volume to break the reversible terminator link will cause lots of photobleaching. It will be interesting to see if 454.bio can overcome this.</div><br/></div></div><div id="39267710" class="c"><input type="checkbox" id="c-39267710" checked=""/><div class="controls bullet"><span class="by">codetrotter</span><span>|</span><a href="#39267853">prev</a><span>|</span><a href="#39270971">next</a><span>|</span><label class="collapse" for="c-39267710">[-]</label><label class="expand" for="c-39267710">[7 more]</label></div><br/><div class="children"><div class="content">Not the kind of sequencer I was first expecting[0], but I&#x27;ll take it.<p>[0]: <a href="https:&#x2F;&#x2F;www.sweetwater.com&#x2F;insync&#x2F;step-sequencer&#x2F;" rel="nofollow">https:&#x2F;&#x2F;www.sweetwater.com&#x2F;insync&#x2F;step-sequencer&#x2F;</a></div><br/><div id="39268153" class="c"><input type="checkbox" id="c-39268153" checked=""/><div class="controls bullet"><span class="by">mfld</span><span>|</span><a href="#39267710">parent</a><span>|</span><a href="#39267827">next</a><span>|</span><label class="collapse" for="c-39268153">[-]</label><label class="expand" for="c-39268153">[1 more]</label></div><br/><div class="children"><div class="content">Yes, I remember putting &quot;DNA sequencer&quot; in the initial title for that reason</div><br/></div></div><div id="39267827" class="c"><input type="checkbox" id="c-39267827" checked=""/><div class="controls bullet"><span class="by">bonestamp2</span><span>|</span><a href="#39267710">parent</a><span>|</span><a href="#39268153">prev</a><span>|</span><a href="#39269899">next</a><span>|</span><label class="collapse" for="c-39267827">[-]</label><label class="expand" for="c-39267827">[3 more]</label></div><br/><div class="children"><div class="content">Whew, I wasn&#x27;t the only one.</div><br/><div id="39270734" class="c"><input type="checkbox" id="c-39270734" checked=""/><div class="controls bullet"><span class="by">dhosek</span><span>|</span><a href="#39267710">root</a><span>|</span><a href="#39267827">parent</a><span>|</span><a href="#39268630">next</a><span>|</span><label class="collapse" for="c-39270734">[-]</label><label class="expand" for="c-39270734">[1 more]</label></div><br/><div class="children"><div class="content">Yeah, I was reading it and I got really confused when they talked about $20 sequencing runs.</div><br/></div></div><div id="39268630" class="c"><input type="checkbox" id="c-39268630" checked=""/><div class="controls bullet"><span class="by">aldanor</span><span>|</span><a href="#39267710">root</a><span>|</span><a href="#39267827">parent</a><span>|</span><a href="#39270734">prev</a><span>|</span><a href="#39269899">next</a><span>|</span><label class="collapse" for="c-39268630">[-]</label><label class="expand" for="c-39268630">[1 more]</label></div><br/><div class="children"><div class="content">Came here expecting something like Orca. Sigh.</div><br/></div></div></div></div><div id="39269899" class="c"><input type="checkbox" id="c-39269899" checked=""/><div class="controls bullet"><span class="by">Hbruz0</span><span>|</span><a href="#39267710">parent</a><span>|</span><a href="#39267827">prev</a><span>|</span><a href="#39270971">next</a><span>|</span><label class="collapse" for="c-39269899">[-]</label><label class="expand" for="c-39269899">[2 more]</label></div><br/><div class="children"><div class="content">And here I was, thinking of yet another type of sequencer :<p><a href="https:&#x2F;&#x2F;book.starknet.io&#x2F;ch03-02-sequencers.html" rel="nofollow">https:&#x2F;&#x2F;book.starknet.io&#x2F;ch03-02-sequencers.html</a></div><br/><div id="39271589" class="c"><input type="checkbox" id="c-39271589" checked=""/><div class="controls bullet"><span class="by">kabes</span><span>|</span><a href="#39267710">root</a><span>|</span><a href="#39269899">parent</a><span>|</span><a href="#39270971">next</a><span>|</span><label class="collapse" for="c-39271589">[-]</label><label class="expand" for="c-39271589">[1 more]</label></div><br/><div class="children"><div class="content">I was thinking a sequencer for modular synths. Took me embarrassingly long to realize what it actually was about.</div><br/></div></div></div></div></div></div><div id="39270971" class="c"><input type="checkbox" id="c-39270971" checked=""/><div class="controls bullet"><span class="by">daemonk</span><span>|</span><a href="#39267710">prev</a><span>|</span><a href="#39267341">next</a><span>|</span><label class="collapse" for="c-39270971">[-]</label><label class="expand" for="c-39270971">[1 more]</label></div><br/><div class="children"><div class="content">The problem of non-selective cleavage of all terminators, whether they have been incorporated or not seems to be the most significant one. Is engineering a selective polymerase really the best solution here? If you don&#x27;t top-up the reaction with un-cleaved terminators, won&#x27;t the reaction become slower and slower as you go? The ideal solution is somehow have terminators that will only get cleaved after incorporation (maybe due to conformational changes); but I can see how that&#x27;s probably even harder to engineer than a selective polymerase.<p>I can see that the complex microfluidics and reagent cycling can be a big added complexity&#x2F;cost. The continuous process is definitely more elegant, but also seems really difficult to get right for longer read lengths and I wonder if it is really worth it.<p>What would be the potential benefits of the continuous process? I guess cheaper reagents since it is a one-pot process and faster sequencing speed since you don&#x27;t need to cycle reagents?</div><br/></div></div><div id="39267341" class="c"><input type="checkbox" id="c-39267341" checked=""/><div class="controls bullet"><span class="by">lbotos</span><span>|</span><a href="#39270971">prev</a><span>|</span><a href="#39270889">next</a><span>|</span><label class="collapse" for="c-39267341">[-]</label><label class="expand" for="c-39267341">[3 more]</label></div><br/><div class="children"><div class="content">Straight to the &quot;Build&quot; page: <a href="http:&#x2F;&#x2F;454.bio&#x2F;docs&#x2F;build&#x2F;" rel="nofollow">http:&#x2F;&#x2F;454.bio&#x2F;docs&#x2F;build&#x2F;</a></div><br/><div id="39269033" class="c"><input type="checkbox" id="c-39269033" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39267341">parent</a><span>|</span><a href="#39270889">next</a><span>|</span><label class="collapse" for="c-39269033">[-]</label><label class="expand" for="c-39269033">[2 more]</label></div><br/><div class="children"><div class="content">I was surprised by their choice of current-limiting resistors for the LEDs (instead of constant current driver) but I guess their goal was to keep the hardware As Simple As Possible</div><br/><div id="39269984" class="c"><input type="checkbox" id="c-39269984" checked=""/><div class="controls bullet"><span class="by">userbinator</span><span>|</span><a href="#39267341">root</a><span>|</span><a href="#39269033">parent</a><span>|</span><a href="#39270889">next</a><span>|</span><label class="collapse" for="c-39269984">[-]</label><label class="expand" for="c-39269984">[1 more]</label></div><br/><div class="children"><div class="content">Even commercial designs will cost-optimise. There&#x27;s no need for extra complexity if it costs more and has lower reliability, unless they&#x27;re going for planned obsolescence.</div><br/></div></div></div></div></div></div><div id="39270889" class="c"><input type="checkbox" id="c-39270889" checked=""/><div class="controls bullet"><span class="by">Jun8</span><span>|</span><a href="#39267341">prev</a><span>|</span><a href="#39270411">next</a><span>|</span><label class="collapse" for="c-39270889">[-]</label><label class="expand" for="c-39270889">[2 more]</label></div><br/><div class="children"><div class="content">This really gets my blood flowing, it really is the future! The question is: can we get experimentation in biology, say, for high schoolers and undergrads, as easy&#x2F;cheap&#x2F;interesting as computing? On the order of buying a good laptop, i.e. - $1,500. What can be done on the order of buying a rapid and a few sensors and a breadboard, $50-$100?<p>Unfortunately pretty much biological equipment is so expensive and relatively hard to use. Case in point: my son is looking to measure amylase activity in the presence of various inhibitors for diabetes research. The cheapest spectrophotometer devices are $1k-$2k. Surely there must be a way to lose some accuracy but bring the price to $100-$200.</div><br/><div id="39270926" class="c"><input type="checkbox" id="c-39270926" checked=""/><div class="controls bullet"><span class="by">fragmede</span><span>|</span><a href="#39270889">parent</a><span>|</span><a href="#39270411">next</a><span>|</span><label class="collapse" for="c-39270926">[-]</label><label class="expand" for="c-39270926">[1 more]</label></div><br/><div class="children"><div class="content"><a href="https:&#x2F;&#x2F;a.co&#x2F;d&#x2F;1peZmjE" rel="nofollow">https:&#x2F;&#x2F;a.co&#x2F;d&#x2F;1peZmjE</a></div><br/></div></div></div></div><div id="39270411" class="c"><input type="checkbox" id="c-39270411" checked=""/><div class="controls bullet"><span class="by">AtomicOrbital</span><span>|</span><a href="#39270889">prev</a><span>|</span><a href="#39270111">next</a><span>|</span><label class="collapse" for="c-39270411">[-]</label><label class="expand" for="c-39270411">[1 more]</label></div><br/><div class="children"><div class="content">A real catalyst would be to couple a sequencer to digitize the DNA together with a matter compiler to output real DNA from the digital DNA representation<p>Why?  craft software to grow the organism in a biologically plausible manner in simulation from the digital DNA... then expose evolutionary pressure to this growth to evolve the DNA in simulation into a more value added organism<p>real organism --&gt; sequence it&#x27;s DNA into digital DNA --&gt; evolve the DNA in simulation software --&gt; output real DNA from digital DNA --&gt; insert this evolved real DNA into a real cell to clone an organism now with enhanced attributes</div><br/></div></div><div id="39270111" class="c"><input type="checkbox" id="c-39270111" checked=""/><div class="controls bullet"><span class="by">inciampati</span><span>|</span><a href="#39270411">prev</a><span>|</span><a href="#39270052">next</a><span>|</span><label class="collapse" for="c-39270111">[-]</label><label class="expand" for="c-39270111">[3 more]</label></div><br/><div class="children"><div class="content">I got my start in genomics working on an open source DNA sequencer. In the end, we built a powerful scanning fluorescent microscopy platform that saw a few beautiful experiments done on it. I always wondered when this might happen in a truly open way. With the right tech and a focused corporate backing it really could happen. Maybe this is the beginning of something very interesting.</div><br/><div id="39270176" class="c"><input type="checkbox" id="c-39270176" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39270111">parent</a><span>|</span><a href="#39270193">next</a><span>|</span><label class="collapse" for="c-39270176">[-]</label><label class="expand" for="c-39270176">[1 more]</label></div><br/><div class="children"><div class="content">Rather than sequencing, I think it would be better to develop a standardized platform for a wide range of microscopy, but nothing really ambitious.  Each new feature you add increases the build challenge.  For example, I&#x27;ve built a straightforward transmitted light microscope with automated XY stage and Z focus, with cheap electronics; changing from transmitted to fluorescence would require a great deal of additional components, complexity, and need for precision and accuracy in the design and the build.  Adding a &quot;growth chamber&quot; around the scope would as well.  Any sort of super-resolution, autofocus, extremely fine movements, etc, are pretty much off the table for a simple build.<p>Ideally you could draft off of the work done for openbuilds, which is mainly focused on printers, lasers, and CNC machines.<p>(there are several projects that attempt this; openflexure is an example, but every time I work with their design, I go back to leadscrews with NEMA steppers and linear rail.)<p>There&#x27;s a lot more biology than just sequencing DNA.  transmitted light microscopy of cells is remarkably useful if you can do it cheap.</div><br/></div></div><div id="39270193" class="c"><input type="checkbox" id="c-39270193" checked=""/><div class="controls bullet"><span class="by">tito</span><span>|</span><a href="#39270111">parent</a><span>|</span><a href="#39270176">prev</a><span>|</span><a href="#39270052">next</a><span>|</span><label class="collapse" for="c-39270193">[-]</label><label class="expand" for="c-39270193">[1 more]</label></div><br/><div class="children"><div class="content">cool! i worked on open hardware for PCR (<a href="http:&#x2F;&#x2F;openpcr.org" rel="nofollow">http:&#x2F;&#x2F;openpcr.org</a>). what are you working on now?<p>i&#x27;m working on reversing climate change at <a href="http:&#x2F;&#x2F;airminers.com" rel="nofollow">http:&#x2F;&#x2F;airminers.com</a></div><br/></div></div></div></div><div id="39270052" class="c"><input type="checkbox" id="c-39270052" checked=""/><div class="controls bullet"><span class="by">userbinator</span><span>|</span><a href="#39270111">prev</a><span>|</span><a href="#39267580">next</a><span>|</span><label class="collapse" for="c-39270052">[-]</label><label class="expand" for="c-39270052">[1 more]</label></div><br/><div class="children"><div class="content">I was curious about the reappearance of that number in a biology context (to me, it evokes a famous car engine), and remembered this interesting video on the old and once-very-expensive Roche 454 GS (also a very automotive-sounding name) sequencer:<p><a href="https:&#x2F;&#x2F;www.youtube.com&#x2F;watch?v=XaumUp4GpCw" rel="nofollow">https:&#x2F;&#x2F;www.youtube.com&#x2F;watch?v=XaumUp4GpCw</a></div><br/></div></div><div id="39267580" class="c"><input type="checkbox" id="c-39267580" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39270052">prev</a><span>|</span><a href="#39270432">next</a><span>|</span><label class="collapse" for="c-39267580">[-]</label><label class="expand" for="c-39267580">[7 more]</label></div><br/><div class="children"><div class="content">It&#x27;s basically a little custom microscope with a few illuminators and a Z stage for focusing (not even an XY stage).  Not surprising, as sequencing has been microscopy for a while now.  It uses TIRF optics.</div><br/><div id="39267737" class="c"><input type="checkbox" id="c-39267737" checked=""/><div class="controls bullet"><span class="by">epistasis</span><span>|</span><a href="#39267580">parent</a><span>|</span><a href="#39270432">next</a><span>|</span><label class="collapse" for="c-39267737">[-]</label><label class="expand" for="c-39267737">[6 more]</label></div><br/><div class="children"><div class="content">I&#x27;m sure you know this already, but for the benefit of the HN crowd...<p>Yeah it&#x27;s been unfortunate that the electronic sensors have not been able to keep up with the optical readouts. Ion Torrent and Genapsys are barely used due to accuracy problems, but Oxford Nanopore&#x27;s considerable logistical benefits of tolerating a wide range of input DNA concentration, quick time to data, low capital cost, and long reads has kept them alive. Maybe they will grow more with higher accuracy.<p>Illumina&#x27;s patents ending recently has prompted a few new startups beyond 454.bio, including Element and Singluar, but with the stiff headwinds of FDA regulatory uncertainty and the long capital life of existing sequencers, it&#x27;s hard for them to make progress even when they are far cheaper than Illumina. And then we have Ultima and MGI. The me of 10 years ago would not believe both 1) how many strong entrants we have to DNA sequencers these days, and 2) how much Illumina continues to dominate in sequencing. At some point us biologists only have ourselves to blame for our purchasing decisions and their consequences.</div><br/><div id="39268183" class="c"><input type="checkbox" id="c-39268183" checked=""/><div class="controls bullet"><span class="by">koeng</span><span>|</span><a href="#39267580">root</a><span>|</span><a href="#39267737">parent</a><span>|</span><a href="#39269313">next</a><span>|</span><label class="collapse" for="c-39268183">[-]</label><label class="expand" for="c-39268183">[2 more]</label></div><br/><div class="children"><div class="content">Oxford Nanopore with the new R10 cells is good enough for most synthetic biology applications (ie, plasmid sequencing and strain validation), which I&#x27;m super excited about. Combine low-capital and long-reads and you can do some really interesting things in the DNA assembly realm. It&#x27;s pretty recent they got there though<p>&gt; At some point us biologists only have ourselves to blame for our purchasing decisions and their consequences<p>Yep...</div><br/><div id="39271836" class="c"><input type="checkbox" id="c-39271836" checked=""/><div class="controls bullet"><span class="by">greazy</span><span>|</span><a href="#39267580">root</a><span>|</span><a href="#39268183">parent</a><span>|</span><a href="#39269313">next</a><span>|</span><label class="collapse" for="c-39271836">[-]</label><label class="expand" for="c-39271836">[1 more]</label></div><br/><div class="children"><div class="content">At the consensus level the accuracy is high. At the read level there&#x27;s a lot of room foe improvement. Currently running r10 routinely for surveillance purposes. Seeing per read quality score average of 15.</div><br/></div></div></div></div><div id="39269313" class="c"><input type="checkbox" id="c-39269313" checked=""/><div class="controls bullet"><span class="by">ejstronge</span><span>|</span><a href="#39267580">root</a><span>|</span><a href="#39267737">parent</a><span>|</span><a href="#39268183">prev</a><span>|</span><a href="#39267970">next</a><span>|</span><label class="collapse" for="c-39269313">[-]</label><label class="expand" for="c-39269313">[1 more]</label></div><br/><div class="children"><div class="content">&gt; Ion Torrent and Genapsys are barely used due to accuracy problems,<p>Accurancy didn&#x27;t have very much to do with Ion Torrent&#x27;s demise. As we&#x27;re reminded in the OP, Illumina pushed the MiSeq platform, and there wasn&#x27;t much reason to transition to Ion Torrent. Now, with 250 bp+ PE reads on Illumina, there really isn&#x27;t much special about Ion Torrent besides its somewhat superior homopolymer performance.<p>&gt; but Oxford Nanopore&#x27;s considerable logistical benefits of tolerating a wide range of input DNA concentration, quick time to data, low capital cost, and long reads has kept them alive. Maybe they will grow more with higher accuracy.<p>This statement is reasonable in today&#x27;s era, but ONT was nowhere close to being competitive with Ion Torrent and other platforms during the time where labs were considering the Ion Torrent devices vs continued spend on Illumina platforms.<p>The accuracy, tolerance to unusual DNA samples, and read length with clinically-relevant base qualities were unusable until 2018&#x2F;2019.</div><br/></div></div><div id="39267970" class="c"><input type="checkbox" id="c-39267970" checked=""/><div class="controls bullet"><span class="by">ak217</span><span>|</span><a href="#39267580">root</a><span>|</span><a href="#39267737">parent</a><span>|</span><a href="#39269313">prev</a><span>|</span><a href="#39270432">next</a><span>|</span><label class="collapse" for="c-39267970">[-]</label><label class="expand" for="c-39267970">[2 more]</label></div><br/><div class="children"><div class="content">It&#x27;s been a while since I caught up with the field, and this sounds exciting. I&#x27;d like to learn more about what exactly has expired (or will expire?). Is it just the reversible terminator patents, or is there more? Is this the main patent in question? <a href="https:&#x2F;&#x2F;patents.google.com&#x2F;patent&#x2F;US7541444B2&#x2F;en" rel="nofollow">https:&#x2F;&#x2F;patents.google.com&#x2F;patent&#x2F;US7541444B2&#x2F;en</a> or are there others?</div><br/><div id="39268363" class="c"><input type="checkbox" id="c-39268363" checked=""/><div class="controls bullet"><span class="by">epistasis</span><span>|</span><a href="#39267580">root</a><span>|</span><a href="#39267970">parent</a><span>|</span><a href="#39270432">next</a><span>|</span><label class="collapse" for="c-39268363">[-]</label><label class="expand" for="c-39268363">[1 more]</label></div><br/><div class="children"><div class="content">I&#x27;m not an expert in the IP, but this 2021 summary is better than my best understanding<p><a href="https:&#x2F;&#x2F;41j.com&#x2F;blog&#x2F;2021&#x2F;05&#x2F;the-next-few-years-in-dna-sequencing&#x2F;" rel="nofollow">https:&#x2F;&#x2F;41j.com&#x2F;blog&#x2F;2021&#x2F;05&#x2F;the-next-few-years-in-dna-seque...</a></div><br/></div></div></div></div></div></div></div></div><div id="39270432" class="c"><input type="checkbox" id="c-39270432" checked=""/><div class="controls bullet"><span class="by">w10-1</span><span>|</span><a href="#39267580">prev</a><span>|</span><a href="#39270308">next</a><span>|</span><label class="collapse" for="c-39270432">[-]</label><label class="expand" for="c-39270432">[1 more]</label></div><br/><div class="children"><div class="content">Sorry, I just don&#x27;t see how to extend the limited read length of ~5 without more microfluidics, and the reagent story is unclear.</div><br/></div></div><div id="39270308" class="c"><input type="checkbox" id="c-39270308" checked=""/><div class="controls bullet"><span class="by">wslh</span><span>|</span><a href="#39270432">prev</a><span>|</span><a href="#39269931">next</a><span>|</span><label class="collapse" for="c-39270308">[-]</label><label class="expand" for="c-39270308">[2 more]</label></div><br/><div class="children"><div class="content">ELI5: could you have your full DNA? I thought that a device retrieving your full DNA is in the USD 100k range.</div><br/><div id="39271689" class="c"><input type="checkbox" id="c-39271689" checked=""/><div class="controls bullet"><span class="by">greazy</span><span>|</span><a href="#39270308">parent</a><span>|</span><a href="#39269931">next</a><span>|</span><label class="collapse" for="c-39271689">[-]</label><label class="expand" for="c-39271689">[1 more]</label></div><br/><div class="children"><div class="content">Maybe 10 years ago. You can buy a mondern sequencing device for 1k usd. Look at ONT minION. Reagents and flow cost of course not included.</div><br/></div></div></div></div><div id="39269931" class="c"><input type="checkbox" id="c-39269931" checked=""/><div class="controls bullet"><span class="by">potsandpans</span><span>|</span><a href="#39270308">prev</a><span>|</span><a href="#39271152">next</a><span>|</span><label class="collapse" for="c-39269931">[-]</label><label class="expand" for="c-39269931">[5 more]</label></div><br/><div class="children"><div class="content">Kinda cool, I&#x27;m out of my depth here. What would be some interesting things to do with this once it was built?</div><br/><div id="39269990" class="c"><input type="checkbox" id="c-39269990" checked=""/><div class="controls bullet"><span class="by">Thebroser</span><span>|</span><a href="#39269931">parent</a><span>|</span><a href="#39271152">next</a><span>|</span><label class="collapse" for="c-39269990">[-]</label><label class="expand" for="c-39269990">[4 more]</label></div><br/><div class="children"><div class="content">Home lab, or even be able to sequence your genome at home, privately and securely without risking giving your genetic information to companies like 23andme</div><br/><div id="39270036" class="c"><input type="checkbox" id="c-39270036" checked=""/><div class="controls bullet"><span class="by">potsandpans</span><span>|</span><a href="#39269931">root</a><span>|</span><a href="#39269990">parent</a><span>|</span><a href="#39271152">next</a><span>|</span><label class="collapse" for="c-39270036">[-]</label><label class="expand" for="c-39270036">[3 more]</label></div><br/><div class="children"><div class="content">Forgive me, I&#x27;m just totally ignorant here. In order to sequence your own genome and do something like 23andme, I kinda just guessed that you&#x27;d need some kind of existing corpus and database to derive meaning from the results.<p>Is that true? And if so, is it all open source or free?<p>It&#x27;s something that&#x27;s kind of interesting to me. I really enjoyed biotech and chemistry in HS but ended up going in a different direction. If this is something that you can just get into now as a hobby maybe I&#x27;ll jump down the rabbithole</div><br/><div id="39271867" class="c"><input type="checkbox" id="c-39271867" checked=""/><div class="controls bullet"><span class="by">greazy</span><span>|</span><a href="#39269931">root</a><span>|</span><a href="#39270036">parent</a><span>|</span><a href="#39271152">next</a><span>|</span><label class="collapse" for="c-39271867">[-]</label><label class="expand" for="c-39271867">[2 more]</label></div><br/><div class="children"><div class="content">Nearly everything in this space is OSS. From the aligners to the assembly and the databases.<p>Last I checked 23andMe didn&#x27;t perform whole genome sequencing but looked at some number of point mutations. It depends on what information you want. Eg for the sake of interest of actional genomics data.</div><br/><div id="39272081" class="c"><input type="checkbox" id="c-39272081" checked=""/><div class="controls bullet"><span class="by">mfld</span><span>|</span><a href="#39269931">root</a><span>|</span><a href="#39271867">parent</a><span>|</span><a href="#39271152">next</a><span>|</span><label class="collapse" for="c-39272081">[-]</label><label class="expand" for="c-39272081">[1 more]</label></div><br/><div class="children"><div class="content">While it&#x27;s true that most of the tools for genomics are OSS, there is some good software and in particular some databases that are commercial and quite pricey.<p>And the sequencing machines and chemicals also don&#x27;t come cheap, as the market has long been dominated by one vendor.</div><br/></div></div></div></div></div></div></div></div></div></div><div id="39271152" class="c"><input type="checkbox" id="c-39271152" checked=""/><div class="controls bullet"><span class="by">CyberDildonics</span><span>|</span><a href="#39269931">prev</a><span>|</span><a href="#39267905">next</a><span>|</span><label class="collapse" for="c-39271152">[-]</label><label class="expand" for="c-39271152">[1 more]</label></div><br/><div class="children"><div class="content">Someone really should have put &quot;DNA sequencer&quot; in the title.</div><br/></div></div><div id="39267905" class="c"><input type="checkbox" id="c-39267905" checked=""/><div class="controls bullet"><span class="by">Palomides</span><span>|</span><a href="#39271152">prev</a><span>|</span><a href="#39268245">next</a><span>|</span><label class="collapse" for="c-39267905">[-]</label><label class="expand" for="c-39267905">[1 more]</label></div><br/><div class="children"><div class="content">single digit read lengths is pretty rough, I don&#x27;t have the chemistry knowledge to tell if this approach is scalable</div><br/></div></div><div id="39267992" class="c"><input type="checkbox" id="c-39267992" checked=""/><div class="controls bullet"><span class="by">pixl97</span><span>|</span><a href="#39268245">prev</a><span>|</span><label class="collapse" for="c-39267992">[-]</label><label class="expand" for="c-39267992">[1 more]</label></div><br/><div class="children"><div class="content">How long before I can build my own, hook it to ChatGPT,  make the national news  and all the AI safety people go ballistic?</div><br/></div></div></div></div></div></div></div></body></html>