<!DOCTYPE html><html lang="en"><head><title>Static News</title><meta charSet="utf-8"/><meta name="description" content="Static delayed Hacker News."/><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"/><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1d1f21"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><meta name="application-name" content="Static News"/><meta name="apple-mobile-web-app-title" content="Static News"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="#1d1f21"/><link rel="preload" href="styles.css?v=1710493274638" as="style"/><link rel="stylesheet" href="styles.css?v=1710493274638"/></head><body><div id="container"><div id="inner"><header><a href="/">Static News</a><a href="/about">about</a></header><div id="content"><div><div id="title"><a href="https://hazyresearch.stanford.edu/blog/2024-03-14-evo">Learning From DNA: a grand challenge in biology</a> <span class="domain">(<a href="https://hazyresearch.stanford.edu">hazyresearch.stanford.edu</a>)</span></div><div class="subtext"><span>ninjha01</span> | <span>29 comments</span></div><br/><div><div id="39709002" class="c"><input type="checkbox" id="c-39709002" checked=""/><div class="controls bullet"><span class="by">ninjha01</span><span>|</span><a href="#39709614">next</a><span>|</span><label class="collapse" for="c-39709002">[-]</label><label class="expand" for="c-39709002">[3 more]</label></div><br/><div class="children"><div class="content">I built the wrapper&#x2F;playground [0] linked in the article. Feel free to give feedback here or by the email in my bio<p>[0] <a href="https:&#x2F;&#x2F;evo.nitro.bio&#x2F;" rel="nofollow">https:&#x2F;&#x2F;evo.nitro.bio&#x2F;</a></div><br/><div id="39711505" class="c"><input type="checkbox" id="c-39711505" checked=""/><div class="controls bullet"><span class="by">timy2shoes</span><span>|</span><a href="#39709002">parent</a><span>|</span><a href="#39709614">next</a><span>|</span><label class="collapse" for="c-39711505">[-]</label><label class="expand" for="c-39711505">[2 more]</label></div><br/><div class="children"><div class="content">Hi Nishant.  Great work, as always.</div><br/><div id="39711626" class="c"><input type="checkbox" id="c-39711626" checked=""/><div class="controls bullet"><span class="by">ninjha01</span><span>|</span><a href="#39709002">root</a><span>|</span><a href="#39711505">parent</a><span>|</span><a href="#39709614">next</a><span>|</span><label class="collapse" for="c-39711626">[-]</label><label class="expand" for="c-39711626">[1 more]</label></div><br/><div class="children"><div class="content">Thanks for the kind words :)</div><br/></div></div></div></div></div></div><div id="39709614" class="c"><input type="checkbox" id="c-39709614" checked=""/><div class="controls bullet"><span class="by">jashephe</span><span>|</span><a href="#39709002">prev</a><span>|</span><a href="#39708892">next</a><span>|</span><label class="collapse" for="c-39709614">[-]</label><label class="expand" for="c-39709614">[4 more]</label></div><br/><div class="children"><div class="content">I&#x27;m a little disappointed that their linked preprint doesn&#x27;t appear to include any molecular biology; i.e. they don&#x27;t actually try to synthesize any of their predicted sequences and test function. It wouldn&#x27;t be an outrageous synthesis task to make some of the CRISPR-Cas sequences they generated.<p>Also interesting that AlphaMisense is omitted from Figure 2B; it substantially outperforms the ESM-based ESM1b in our hands. But I guess the idea is that this is a general-purpose DNA language model whereas AlphaMissense is domain-specific for variant effect prediction?</div><br/><div id="39710127" class="c"><input type="checkbox" id="c-39710127" checked=""/><div class="controls bullet"><span class="by">bnprks</span><span>|</span><a href="#39709614">parent</a><span>|</span><a href="#39711867">next</a><span>|</span><label class="collapse" for="c-39710127">[-]</label><label class="expand" for="c-39710127">[1 more]</label></div><br/><div class="children"><div class="content">Strong second for wishing they tried physically testing some model output. The importance of &quot;model that makes outputs AlphaFold thinks look like Cas&quot; is very different from &quot;model that makes functional Cas variants&quot;.<p>For design tasks like in this paper, I think computational models have a big hill to climb in order to compete with physical high-throughput screening. Most of the time the goal is to get a small number of hits (&lt;10) out of a pool of millions of candidates. At those levels, you need to work in the &gt;99.9% precision regime to have any hope of finding significant hits after multiple-hypothesis correction. I don&#x27;t think they showed anything near that accurate in the paper.<p>Maybe we&#x27;ll get there eventually, but the high-throughput techniques in molecular biology are also getting better at the same time.</div><br/></div></div><div id="39711867" class="c"><input type="checkbox" id="c-39711867" checked=""/><div class="controls bullet"><span class="by">ackbar03</span><span>|</span><a href="#39709614">parent</a><span>|</span><a href="#39710127">prev</a><span>|</span><a href="#39711747">next</a><span>|</span><label class="collapse" for="c-39711867">[-]</label><label class="expand" for="c-39711867">[1 more]</label></div><br/><div class="children"><div class="content">This should really be a requirement for bio type related generative methods rather than a nice-to-have. A very high percentage of compounds generated by genai type methods have been shown not to work as intended. Anything without wetlab validation should really be taken with a large grain of salt</div><br/></div></div><div id="39711747" class="c"><input type="checkbox" id="c-39711747" checked=""/><div class="controls bullet"><span class="by">rdmirza</span><span>|</span><a href="#39709614">parent</a><span>|</span><a href="#39711867">prev</a><span>|</span><a href="#39708892">next</a><span>|</span><label class="collapse" for="c-39711747">[-]</label><label class="expand" for="c-39711747">[1 more]</label></div><br/><div class="children"><div class="content">My immediate thought. Big Claims without backing.<p>Your model makes predictions. Prove they’re worth salt.</div><br/></div></div></div></div><div id="39708892" class="c"><input type="checkbox" id="c-39708892" checked=""/><div class="controls bullet"><span class="by">d_silin</span><span>|</span><a href="#39709614">prev</a><span>|</span><a href="#39708711">next</a><span>|</span><label class="collapse" for="c-39708892">[-]</label><label class="expand" for="c-39708892">[1 more]</label></div><br/><div class="children"><div class="content">Would be interesting to see what comes of it.<p>As you progress along the following chain: genomics--&gt;proteomics-&gt;interactomics-&gt;metabolomics, our understanding becomes blurrier and challenges harder.</div><br/></div></div><div id="39708745" class="c"><input type="checkbox" id="c-39708745" checked=""/><div class="controls bullet"><span class="by">visarga</span><span>|</span><a href="#39708711">prev</a><span>|</span><a href="#39708311">next</a><span>|</span><label class="collapse" for="c-39708745">[-]</label><label class="expand" for="c-39708745">[2 more]</label></div><br/><div class="children"><div class="content">DNA is all you need? In the future generative AI will generate You!</div><br/></div></div><div id="39708311" class="c"><input type="checkbox" id="c-39708311" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708745">prev</a><span>|</span><label class="collapse" for="c-39708311">[-]</label><label class="expand" for="c-39708311">[17 more]</label></div><br/><div class="children"><div class="content">Just gonna leave this here.<p><a href="https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2024.02.29.582810v1" rel="nofollow">https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2024.02.29.582810v1</a><p>Tl;dr: DNA is NOT all you need.</div><br/><div id="39709038" class="c"><input type="checkbox" id="c-39709038" checked=""/><div class="controls bullet"><span class="by">jhbadger</span><span>|</span><a href="#39708311">parent</a><span>|</span><a href="#39708367">next</a><span>|</span><label class="collapse" for="c-39709038">[-]</label><label class="expand" for="c-39709038">[4 more]</label></div><br/><div class="children"><div class="content">I think you are missing what the Evo project is trying to do -- create a new prokaryotic genome through a generative model. How this would work would be like the earlier hand-made synthetic genomes like Synthia (Gibson et al, 2010).<p>In such a system you would take an existing bacterial cell and replace its genome with the newly synthesized version. The proteins and other molecules from the existing cell would remain (before eventually being replaced) and serve to &quot;boot&quot; the new genome.</div><br/><div id="39711333" class="c"><input type="checkbox" id="c-39711333" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709038">parent</a><span>|</span><a href="#39709430">next</a><span>|</span><label class="collapse" for="c-39711333">[-]</label><label class="expand" for="c-39711333">[1 more]</label></div><br/><div class="children"><div class="content">Sounds cool, but how you define success for something like that?  I can copy a prokaryotic genome mutated based as some non-zero rate and it would probably be viable.  Is that synthetic enough to count?  Are they going for a minimal genome?<p>How about something more useful, lucrative, and easy to define success for like engineering a morphine synthesis pathway into E. coli or something.<p>Imo, if you are talking about synthetic biology, then their training data is insufficient.  Synthetic bio explores a lot of design space that is far outside of anything you would see in nature.  There the secret sauce would not be in the generative pretraining, but in the RL.  Unfortunately bio experiments are noisy, slow, and expensive so good luck getting enough data before the heat death of the universe.</div><br/></div></div><div id="39709430" class="c"><input type="checkbox" id="c-39709430" checked=""/><div class="controls bullet"><span class="by">nextos</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709038">parent</a><span>|</span><a href="#39711333">prev</a><span>|</span><a href="#39708367">next</a><span>|</span><label class="collapse" for="c-39709430">[-]</label><label class="expand" for="c-39709430">[2 more]</label></div><br/><div class="children"><div class="content">It&#x27;s an interesting endeavor, but there are some obvious safety concerns.<p>Within Prokaryotes, there is a lot of horizontal gene transfer. What if some of the synthetic sequences get into other organisms and spread out?</div><br/><div id="39709656" class="c"><input type="checkbox" id="c-39709656" checked=""/><div class="controls bullet"><span class="by">UniverseHacker</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709430">parent</a><span>|</span><a href="#39708367">next</a><span>|</span><label class="collapse" for="c-39709656">[-]</label><label class="expand" for="c-39709656">[1 more]</label></div><br/><div class="children"><div class="content">Those genes would have to confer an evolutionary advantage or they would immediately be discarded&#x2F;selected against. The chances of that happening are nil… we’re not going to come up with something more useful to bacteria than billions of years of natural selection. Synthetic biology with organisms produced to generate small molecules useful for humans is widely practiced but has the opposite problem- all of the engineering changes to the cells are constantly being selected against, and revert on their own.</div><br/></div></div></div></div></div></div><div id="39708367" class="c"><input type="checkbox" id="c-39708367" checked=""/><div class="controls bullet"><span class="by">samuell</span><span>|</span><a href="#39708311">parent</a><span>|</span><a href="#39709038">prev</a><span>|</span><a href="#39710148">next</a><span>|</span><label class="collapse" for="c-39708367">[-]</label><label class="expand" for="c-39708367">[3 more]</label></div><br/><div class="children"><div class="content">I tend to agree (the cell being in control and all the 4D interactions and epigenetics mechanisms etc), but out of curiosity, what would you say we also need?</div><br/><div id="39708405" class="c"><input type="checkbox" id="c-39708405" checked=""/><div class="controls bullet"><span class="by">COGlory</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39708367">parent</a><span>|</span><a href="#39708469">next</a><span>|</span><label class="collapse" for="c-39708405">[-]</label><label class="expand" for="c-39708405">[1 more]</label></div><br/><div class="children"><div class="content">For starters, chemical environment modeling. But also cells differentiate, so in any system you need to understand the differentiation, and how those differentiated cells will change the environment of other cells, based on the environment they encounter.<p>That&#x27;s not to say you can&#x27;t glean a ton from DNA, but there are some external inputs we may simply never know enough about to incorporate into the model. Ultimately DNA IS all you need...if you have perfect environmental information.</div><br/></div></div><div id="39708469" class="c"><input type="checkbox" id="c-39708469" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39708367">parent</a><span>|</span><a href="#39708405">prev</a><span>|</span><a href="#39710148">next</a><span>|</span><label class="collapse" for="c-39708469">[-]</label><label class="expand" for="c-39708469">[1 more]</label></div><br/><div class="children"><div class="content">The article I posted shows what is working better -  the Olga Troyanskaya &#x2F; David Kelley style models. There was another one (Kundaje group?) recently that used Hi-C data.</div><br/></div></div></div></div><div id="39710148" class="c"><input type="checkbox" id="c-39710148" checked=""/><div class="controls bullet"><span class="by">t_serpico</span><span>|</span><a href="#39708311">parent</a><span>|</span><a href="#39708367">prev</a><span>|</span><a href="#39708585">next</a><span>|</span><label class="collapse" for="c-39710148">[-]</label><label class="expand" for="c-39710148">[1 more]</label></div><br/><div class="children"><div class="content"><a href="https:&#x2F;&#x2F;onlinelibrary.wiley.com&#x2F;doi&#x2F;10.1002&#x2F;bies.201300153" rel="nofollow">https:&#x2F;&#x2F;onlinelibrary.wiley.com&#x2F;doi&#x2F;10.1002&#x2F;bies.201300153</a> 
tl;dr: metabolism is all you need.<p>while potentially interesting work, very shortsighted and premature to say this is a &quot;GPT&quot; moment in biology. ML people in bio need to think hard not only about what they are doing, but why are they are doing it (other than this is cool and will lead to a nice Nature publication). Their basic premise (learning from DNA is the next grand challenge in biology) is shaky. Imo, the grand challenge in biology is determining what the grand challenge is, and that is a deep scientific&#x2F;philosophical question.</div><br/></div></div><div id="39708585" class="c"><input type="checkbox" id="c-39708585" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39708311">parent</a><span>|</span><a href="#39710148">prev</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39708585">[-]</label><label class="expand" for="c-39708585">[7 more]</label></div><br/><div class="children"><div class="content">most of the examples in that paper (a single paper) show that DNA is nearly all you need, with the rest being RNA.</div><br/><div id="39709012" class="c"><input type="checkbox" id="c-39709012" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39708585">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39709012">[-]</label><label class="expand" for="c-39709012">[6 more]</label></div><br/><div class="children"><div class="content">RNA is an obvious example.  The examples and benchmarks they give in the paper are not the straw men the DNA LLMs are beating the stuffing out.<p>Also CRE activity is highly cell type specific.  This article is a pretty awesome demonstration of model guided design of cell type specific cis regulatory elements.<p><a href="https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2023.08.08.552077v1" rel="nofollow">https:&#x2F;&#x2F;www.biorxiv.org&#x2F;content&#x2F;10.1101&#x2F;2023.08.08.552077v1</a><p>An LLM would not be able to do this because DNA itself contains no contextual information about cell type - every cell has a copy of the full genome.  Epigenetic tracks however contain a lot of information germane to the cellular context - ex which parts of the genome are being transcribed.</div><br/><div id="39709158" class="c"><input type="checkbox" id="c-39709158" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709012">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39709158">[-]</label><label class="expand" for="c-39709158">[5 more]</label></div><br/><div class="children"><div class="content">but epigenetics is just DNA.  it&#x27;s state information stored directly in the DNA, or in directly attached machinery.  from the perspective of learned models, those are just other features.<p>But realistically, the right source for transcription is the RNA in the cell, <i>not</i> the epigenetics.  Nearly all cell type profiling is based on RNA.  It&#x27;s far easier and more reliable to interrogate the transcriptome than to try to gain info from epigenetic states.</div><br/><div id="39709382" class="c"><input type="checkbox" id="c-39709382" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709158">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39709382">[-]</label><label class="expand" for="c-39709382">[4 more]</label></div><br/><div class="children"><div class="content">Epigenetics is not just DNA, think of it more like the (hidden) state of DNA.  Histone modifications and open chromatin and other epigenetic readouts are like emissions &#x2F; indicators of the hidden state.<p>The relationship is like that between the words in a book and the page that is actively being read.  I know that is a hackneyed analogy; but coffee is wearing off :)</div><br/><div id="39709820" class="c"><input type="checkbox" id="c-39709820" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709382">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39709820">[-]</label><label class="expand" for="c-39709820">[3 more]</label></div><br/><div class="children"><div class="content">Those are all readable using standard DNA sequencing techniques, so again, it&#x27;s just state attributes of the DNA.<p>(I&#x27;ve worked in genomics for 30+ years.  I&#x27;m not just spitballing here).</div><br/><div id="39711169" class="c"><input type="checkbox" id="c-39711169" checked=""/><div class="controls bullet"><span class="by">pfisherman</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39709820">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39711169">[-]</label><label class="expand" for="c-39711169">[2 more]</label></div><br/><div class="children"><div class="content">DNA libraries are read out using sequencing techniques.  But I doubt anyone would say they are just measuring DNA.  It’s kind of like saying a luciferase based assay is just measuring the intensity of light.</div><br/><div id="39711485" class="c"><input type="checkbox" id="c-39711485" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#39708311">root</a><span>|</span><a href="#39711169">parent</a><span>|</span><a href="#39708366">next</a><span>|</span><label class="collapse" for="c-39711485">[-]</label><label class="expand" for="c-39711485">[1 more]</label></div><br/><div class="children"><div class="content">yes, that is the terminology we use.</div><br/></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></body></html>