<!DOCTYPE html><html lang="en"><head><title>Static News</title><meta charSet="utf-8"/><meta name="description" content="Static delayed Hacker News."/><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"/><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1d1f21"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><meta name="application-name" content="Static News"/><meta name="apple-mobile-web-app-title" content="Static News"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="#1d1f21"/><link rel="preload" href="styles.css?v=1715245271374" as="style"/><link rel="stylesheet" href="styles.css?v=1715245271374"/></head><body><div id="container"><div id="inner"><header><a href="/">Static News</a><a href="/about">about</a></header><div id="content"><div><div id="title"><a href="https://blog.google/technology/ai/google-deepmind-isomorphic-alphafold-3-ai-model/">AlphaFold 3 predicts the structure and interactions of life&#x27;s molecules</a> <span class="domain">(<a href="https://blog.google">blog.google</a>)</span></div><div class="subtext"><span>zerojames</span> | <span>110 comments</span></div><br/><div><div id="40301755" class="c"><input type="checkbox" id="c-40301755" checked=""/><div class="controls bullet"><span class="by">lysozyme</span><span>|</span><a href="#40299838">next</a><span>|</span><label class="collapse" for="c-40301755">[-]</label><label class="expand" for="c-40301755">[4 more]</label></div><br/><div class="children"><div class="content">Probably worth mentioning that David Baker’s lab released a similar model (predicts protein structure along with bound DNA and ligands), just a couple of months ago, and it is open source [1].<p>It’s also worth remembering that it was David Baker who originally came up with the idea of extending AlphaFold from predicting just proteins to predicting ligands as well [2].<p>1. <a href="https:&#x2F;&#x2F;github.com&#x2F;baker-laboratory&#x2F;RoseTTAFold-All-Atom">https:&#x2F;&#x2F;github.com&#x2F;baker-laboratory&#x2F;RoseTTAFold-All-Atom</a><p>2. <a href="https:&#x2F;&#x2F;alexcarlin.bearblog.dev&#x2F;generalized&#x2F;" rel="nofollow">https:&#x2F;&#x2F;alexcarlin.bearblog.dev&#x2F;generalized&#x2F;</a><p>Unlike AlphaFold 3, which predicts only a small, preselected subset of ligands, RosettaFold All Atom predicts a much wider range of small molecules. While I am certain that neither network is up to the task of designing an enzyme, these are exciting steps.<p>One of the more exciting aspects of the RosettaFold paper is that they train the model for predicting structures, but then also use the structure predicting model as the denoising model in a diffusion process, enabling them to actually design new functional proteins. Presumably, DeepMind is working on this problem as well.</div><br/><div id="40303478" class="c"><input type="checkbox" id="c-40303478" checked=""/><div class="controls bullet"><span class="by">refulgentis</span><span>|</span><a href="#40301755">parent</a><span>|</span><a href="#40302377">next</a><span>|</span><label class="collapse" for="c-40303478">[-]</label><label class="expand" for="c-40303478">[1 more]</label></div><br/><div class="children"><div class="content">I appreciated this, but it&#x27;s probably worth mentioning: when you say AlphaFold 3, you&#x27;re talking about AlphaFold 2.<p>TFA announces AlphaFold 3.<p>Post: &quot;Unlike AlphaFold 3, which predicts only a small, preselected subset of ligands, RosettaFold All Atom predicts a much wider range of small molecules&quot;<p>TFA: &quot;AlphaFold 3...*models large biomolecules such as proteins, DNA and RNA*, as well as small molecules, also known as ligands&quot;<p>Post: &quot;they also use the structure predicting model as the denoising model in a diffusion process...Presumably, DeepMind is working on this problem as well.&quot;<p>TFA: &quot;AlphaFold 3 assembles its predictions using a diffusion network, akin to those found in AI image generators.&quot;</div><br/></div></div><div id="40302377" class="c"><input type="checkbox" id="c-40302377" checked=""/><div class="controls bullet"><span class="by">theGnuMe</span><span>|</span><a href="#40301755">parent</a><span>|</span><a href="#40303478">prev</a><span>|</span><a href="#40305835">next</a><span>|</span><label class="collapse" for="c-40302377">[-]</label><label class="expand" for="c-40302377">[1 more]</label></div><br/><div class="children"><div class="content">And that tech just got $1b in funding.</div><br/></div></div><div id="40305835" class="c"><input type="checkbox" id="c-40305835" checked=""/><div class="controls bullet"><span class="by">hackernewds</span><span>|</span><a href="#40301755">parent</a><span>|</span><a href="#40302377">prev</a><span>|</span><a href="#40299838">next</a><span>|</span><label class="collapse" for="c-40305835">[-]</label><label class="expand" for="c-40305835">[1 more]</label></div><br/><div class="children"><div class="content">Coming up with ideas is cheaper than executing the ideas. Predicting a wide range of molecules okay-ish is cheaper than predicting a small range of molecules very well.</div><br/></div></div></div></div><div id="40299838" class="c"><input type="checkbox" id="c-40299838" checked=""/><div class="controls bullet"><span class="by">nybsjytm</span><span>|</span><a href="#40301755">prev</a><span>|</span><a href="#40299289">next</a><span>|</span><label class="collapse" for="c-40299838">[-]</label><label class="expand" for="c-40299838">[18 more]</label></div><br/><div class="children"><div class="content">Important caveat: it&#x27;s only about 70% accurate. Why doesn&#x27;t the press release say this explicitly?  It seems intentionally misleading to only report accuracy relative to existing methods, which apparently are just not so good (30%, 50% in various settings).
<a href="https:&#x2F;&#x2F;www.fastcompany.com&#x2F;91120456&#x2F;deepmind-alphafold-3-dna-rna-modeling" rel="nofollow">https:&#x2F;&#x2F;www.fastcompany.com&#x2F;91120456&#x2F;deepmind-alphafold-3-dn...</a></div><br/><div id="40305528" class="c"><input type="checkbox" id="c-40305528" checked=""/><div class="controls bullet"><span class="by">porphyra</span><span>|</span><a href="#40299838">parent</a><span>|</span><a href="#40304490">next</a><span>|</span><label class="collapse" for="c-40305528">[-]</label><label class="expand" for="c-40305528">[4 more]</label></div><br/><div class="children"><div class="content">They also had a headline for Alphazero that convinced everyone that they crushed Stockfish and that classical chess engines were stuff of the past, when in fact it was about 50 elo better than the Stockfish version they were testing against, or roughly the same as how much Stockfish improves each year.</div><br/><div id="40306362" class="c"><input type="checkbox" id="c-40306362" checked=""/><div class="controls bullet"><span class="by">kriro</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40305528">parent</a><span>|</span><a href="#40306028">next</a><span>|</span><label class="collapse" for="c-40306362">[-]</label><label class="expand" for="c-40306362">[2 more]</label></div><br/><div class="children"><div class="content">I think Alphazero is a lot more interesting than Stockfish though. Most notably it lead me to reevaluate positional play. Iirc A0 at around 2-3 ply is still above SuperGM Level which is pretty mind-blowing. Based on this I have increased my strategy to tactics ratio quite a bit. FWIW Stockfish is always evolving and adapting and has incorporated ideas from A0.</div><br/><div id="40306413" class="c"><input type="checkbox" id="c-40306413" checked=""/><div class="controls bullet"><span class="by">RUnconcerned</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40306362">parent</a><span>|</span><a href="#40306028">next</a><span>|</span><label class="collapse" for="c-40306413">[-]</label><label class="expand" for="c-40306413">[1 more]</label></div><br/><div class="children"><div class="content">Stockfish has not incorporated ideas from AlphaZero. Stockfish&#x27;s NN eval technology, NNUE, comes from Shogi and it predates Alphazero there.<p>The 2nd strongest engine, Leela Chess Zero, is indeed directly inspired by AlphaZero, though, and did surpass Stockfish until NNUE was introduced.</div><br/></div></div></div></div><div id="40306028" class="c"><input type="checkbox" id="c-40306028" checked=""/><div class="controls bullet"><span class="by">thom</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40305528">parent</a><span>|</span><a href="#40306362">prev</a><span>|</span><a href="#40304490">next</a><span>|</span><label class="collapse" for="c-40306028">[-]</label><label class="expand" for="c-40306028">[1 more]</label></div><br/><div class="children"><div class="content">And then what happened is AlphaZero changed the professional game in various interesting ways, and all its ideas were absorbed into Stockfish. A little bombast is forgivable for technology that goes on to have a big impact, and I don’t doubt it’s the same story here.</div><br/></div></div></div></div><div id="40304490" class="c"><input type="checkbox" id="c-40304490" checked=""/><div class="controls bullet"><span class="by">hereme888</span><span>|</span><a href="#40299838">parent</a><span>|</span><a href="#40305528">prev</a><span>|</span><a href="#40301404">next</a><span>|</span><label class="collapse" for="c-40304490">[-]</label><label class="expand" for="c-40304490">[1 more]</label></div><br/><div class="children"><div class="content">That&#x27;s what I thought. They go from &quot;predicting all of life&#x27;s molecules&quot; to &quot;it&#x27;s a 50% improvement...and we HOPE to...transform drug discovery...&quot;<p>Seems unfortunately typical of Google these days: &quot;Gemini will destroy GPT-4...&quot;</div><br/></div></div><div id="40301404" class="c"><input type="checkbox" id="c-40301404" checked=""/><div class="controls bullet"><span class="by">Aunche</span><span>|</span><a href="#40299838">parent</a><span>|</span><a href="#40304490">prev</a><span>|</span><a href="#40301020">next</a><span>|</span><label class="collapse" for="c-40301404">[-]</label><label class="expand" for="c-40301404">[2 more]</label></div><br/><div class="children"><div class="content">IIRC the next best models all have all been using AlphaFold 2&#x27;s methodology, so that&#x27;s still a massive improvement.<p>Edit: I see now that you&#x27;re probably objecting to the headline that got edited on HN.</div><br/><div id="40301563" class="c"><input type="checkbox" id="c-40301563" checked=""/><div class="controls bullet"><span class="by">nybsjytm</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301404">parent</a><span>|</span><a href="#40301020">next</a><span>|</span><label class="collapse" for="c-40301563">[-]</label><label class="expand" for="c-40301563">[1 more]</label></div><br/><div class="children"><div class="content">Not just the headline, the whole press release. And not questioning that it&#x27;s a big improvement.</div><br/></div></div></div></div><div id="40301020" class="c"><input type="checkbox" id="c-40301020" checked=""/><div class="controls bullet"><span class="by">bluerooibos</span><span>|</span><a href="#40299838">parent</a><span>|</span><a href="#40301404">prev</a><span>|</span><a href="#40299289">next</a><span>|</span><label class="collapse" for="c-40301020">[-]</label><label class="expand" for="c-40301020">[10 more]</label></div><br/><div class="children"><div class="content">That&#x27;s pretty good.  Based on the previous performance improvements of Alpha-- models, it&#x27;ll be nearing 100% in the next couple of years.</div><br/><div id="40301534" class="c"><input type="checkbox" id="c-40301534" checked=""/><div class="controls bullet"><span class="by">akira2501</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301020">parent</a><span>|</span><a href="#40301184">next</a><span>|</span><label class="collapse" for="c-40301534">[-]</label><label class="expand" for="c-40301534">[2 more]</label></div><br/><div class="children"><div class="content">&gt; it&#x27;ll be nearing 100% in the next couple of years.<p>What are you basing this on?  There is no established &quot;moores law&quot; for computational models.</div><br/><div id="40303849" class="c"><input type="checkbox" id="c-40303849" checked=""/><div class="controls bullet"><span class="by">OrigamiPastrami</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301534">parent</a><span>|</span><a href="#40301184">next</a><span>|</span><label class="collapse" for="c-40303849">[-]</label><label class="expand" for="c-40303849">[1 more]</label></div><br/><div class="children"><div class="content">It&#x27;s the internet.  There is no source more cited than &quot;trust me bro&quot;</div><br/></div></div></div></div><div id="40301184" class="c"><input type="checkbox" id="c-40301184" checked=""/><div class="controls bullet"><span class="by">nybsjytm</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301020">parent</a><span>|</span><a href="#40301534">prev</a><span>|</span><a href="#40305771">next</a><span>|</span><label class="collapse" for="c-40301184">[-]</label><label class="expand" for="c-40301184">[5 more]</label></div><br/><div class="children"><div class="content">Just &quot;Alpha-- models&quot; in general?? That&#x27;s not a remotely reasonable way to reason about it. Even if it were, why should it stop DeepMind from clearly communicating accuracy?</div><br/><div id="40301352" class="c"><input type="checkbox" id="c-40301352" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301184">parent</a><span>|</span><a href="#40301959">next</a><span>|</span><label class="collapse" for="c-40301352">[-]</label><label class="expand" for="c-40301352">[3 more]</label></div><br/><div class="children"><div class="content">The way I think about this (specifically, deepmind not publishing their code or sharing their exact experimental results): advanced science is a game played by the most sophisticated actors in the world.  Demis is one of those actors, and he plays the games those actors play better than anybody else I&#x27;ve ever seen. Those actors don&#x27;t care much about the details of any specific system&#x27;s accuracy: they care to know that it&#x27;s possible to do this, and some general numbers about how well it works, and some hints what approaches they should take.  And Nature, like other top journals, is more than willing to publish articles like this because they know it stimulates the most competitive players to bring their best games.<p>(I&#x27;m not defending this approach, just making an observation)</div><br/><div id="40301546" class="c"><input type="checkbox" id="c-40301546" checked=""/><div class="controls bullet"><span class="by">nybsjytm</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301352">parent</a><span>|</span><a href="#40306067">next</a><span>|</span><label class="collapse" for="c-40301546">[-]</label><label class="expand" for="c-40301546">[1 more]</label></div><br/><div class="children"><div class="content">I think it&#x27;s important to qualify that the relevant &quot;game&quot; is not advanced science per se; the game is business whose product is science. The aim isn&#x27;t to do novel science; it&#x27;s to do something which can be advertised as novel science. That isn&#x27;t to cast aspersions on the personal motivations of Hassabis or any other individual researcher working there (which itself isn&#x27;t to remove their responsibilities to public understanding); it&#x27;s to cast aspersions on the structure that they&#x27;re part of. And it&#x27;s not to say that they can&#x27;t produce novel or important science as part of their work there. And it&#x27;s also not to say that the same tension isn&#x27;t often present in the science world - but I think it&#x27;s present to an extreme degree at DeepMind.<p>(Sometimes the distinction between novel science and advertisably novel science is very important, as seems to be the case in the &quot;new materials&quot; research dopylitty linked to in these comments: here <a href="https:&#x2F;&#x2F;www.404media.co&#x2F;google-says-it-discovered-millions-of-new-materials-with-ai-human-researchers&#x2F;" rel="nofollow">https:&#x2F;&#x2F;www.404media.co&#x2F;google-says-it-discovered-millions-o...</a>)</div><br/></div></div><div id="40306067" class="c"><input type="checkbox" id="c-40306067" checked=""/><div class="controls bullet"><span class="by">moomin</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301352">parent</a><span>|</span><a href="#40301546">prev</a><span>|</span><a href="#40301959">next</a><span>|</span><label class="collapse" for="c-40306067">[-]</label><label class="expand" for="c-40306067">[1 more]</label></div><br/><div class="children"><div class="content">Anyone remember how he marketed his computer games?</div><br/></div></div></div></div><div id="40301959" class="c"><input type="checkbox" id="c-40301959" checked=""/><div class="controls bullet"><span class="by">7734128</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301184">parent</a><span>|</span><a href="#40301352">prev</a><span>|</span><a href="#40305771">next</a><span>|</span><label class="collapse" for="c-40301959">[-]</label><label class="expand" for="c-40301959">[1 more]</label></div><br/><div class="children"><div class="content">I&#x27;m quite hyped for the upcoming BetaFold, or even ReleaseCandidateFold models. They just have to be great.</div><br/></div></div></div></div><div id="40305771" class="c"><input type="checkbox" id="c-40305771" checked=""/><div class="controls bullet"><span class="by">tsimionescu</span><span>|</span><a href="#40299838">root</a><span>|</span><a href="#40301020">parent</a><span>|</span><a href="#40301184">prev</a><span>|</span><a href="#40301681">next</a><span>|</span><label class="collapse" for="c-40305771">[-]</label><label class="expand" for="c-40305771">[1 more]</label></div><br/><div class="children"><div class="content">Which specific AlphaX model evolved like that? Most of the ones that were in the press had essentially a single showing, typically very good, but didn&#x27;t really improve after that.</div><br/></div></div></div></div></div></div><div id="40299289" class="c"><input type="checkbox" id="c-40299289" checked=""/><div class="controls bullet"><span class="by">renonce</span><span>|</span><a href="#40299838">prev</a><span>|</span><a href="#40300996">next</a><span>|</span><label class="collapse" for="c-40299289">[-]</label><label class="expand" for="c-40299289">[7 more]</label></div><br/><div class="children"><div class="content">&gt; What is different about the new AlphaFold3 model compared to AlphaFold2?<p>&gt; AlphaFold3 can predict many biomolecules in addition to proteins. AlphaFold2 predicts structures of proteins and protein-protein complexes. AlphaFold3 can generate predictions containing proteins, DNA, RNA, ions,ligands, and chemical modifications. The new model also improves the protein complex modelling accuracy. Please refer to our paper for more information on performance improvements.<p>AlphaFold 2 generally produces looping “ribbon-like” predictions for disordered regions. AlphaFold3 also does this, but will occasionally output segments with secondary structure within disordered regions instead, mostly spurious alpha helices with very low confidence (pLDDT) and inconsistent position across predictions.<p>So the criticism towards AlphaFold 2 will likely still apply? For example, it’s more accurate for predicting structures similar to existing ones, and fails at novel patterns?</div><br/><div id="40299476" class="c"><input type="checkbox" id="c-40299476" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299289">parent</a><span>|</span><a href="#40299581">next</a><span>|</span><label class="collapse" for="c-40299476">[-]</label><label class="expand" for="c-40299476">[1 more]</label></div><br/><div class="children"><div class="content">I am not aware of anybody currently criticiszing AF2&#x27;s abilities outside of its training set.  In fact the most recent papers (written by crystallographers) they are mostly arguing about atomic-level details of side chains at this point.</div><br/></div></div><div id="40299581" class="c"><input type="checkbox" id="c-40299581" checked=""/><div class="controls bullet"><span class="by">COGlory</span><span>|</span><a href="#40299289">parent</a><span>|</span><a href="#40299476">prev</a><span>|</span><a href="#40299674">next</a><span>|</span><label class="collapse" for="c-40299581">[-]</label><label class="expand" for="c-40299581">[2 more]</label></div><br/><div class="children"><div class="content">&gt;So the criticism towards AlphaFold 2 will likely still apply? For example, it’s more accurate for predicting structures similar to existing ones, and fails at novel patterns?<p>Yes, and there is simply no way to bridge that gap with this technique. We can make it better and better at pattern matching, but it is not going to predict novel folds.</div><br/><div id="40299663" class="c"><input type="checkbox" id="c-40299663" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299289">root</a><span>|</span><a href="#40299581">parent</a><span>|</span><a href="#40299674">next</a><span>|</span><label class="collapse" for="c-40299663">[-]</label><label class="expand" for="c-40299663">[1 more]</label></div><br/><div class="children"><div class="content">alphafold has been shown to accurately predict some novel folds.  The technique doesn&#x27;t entirely depend on whole-domain homology.</div><br/></div></div></div></div><div id="40299674" class="c"><input type="checkbox" id="c-40299674" checked=""/><div class="controls bullet"><span class="by">rolph</span><span>|</span><a href="#40299289">parent</a><span>|</span><a href="#40299581">prev</a><span>|</span><a href="#40299387">next</a><span>|</span><label class="collapse" for="c-40299674">[-]</label><label class="expand" for="c-40299674">[2 more]</label></div><br/><div class="children"><div class="content">problem is biomolecules, are &quot;chaperoned&quot; to fold properly, only specific regions such as, alpha helix, or beta pleatedsheet will fold de novo.<p>Chaperone (protein)<p><a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Chaperone_(protein)" rel="nofollow">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;Chaperone_(protein)</a></div><br/><div id="40305506" class="c"><input type="checkbox" id="c-40305506" checked=""/><div class="controls bullet"><span class="by">staticautomatic</span><span>|</span><a href="#40299289">root</a><span>|</span><a href="#40299674">parent</a><span>|</span><a href="#40299387">next</a><span>|</span><label class="collapse" for="c-40305506">[-]</label><label class="expand" for="c-40305506">[1 more]</label></div><br/><div class="children"><div class="content">In principle couldn’t we just incorporate knowledge about chaperones into the model?</div><br/></div></div></div></div></div></div><div id="40300996" class="c"><input type="checkbox" id="c-40300996" checked=""/><div class="controls bullet"><span class="by">uptownfunk</span><span>|</span><a href="#40299289">prev</a><span>|</span><a href="#40301141">next</a><span>|</span><label class="collapse" for="c-40300996">[-]</label><label class="expand" for="c-40300996">[18 more]</label></div><br/><div class="children"><div class="content">Very sad to see they did not make it open source. When you have a technology that has the potential to be a gateway for drug development, to the cures of new diseases, and instead you choose to make it closed, it is a very huge disservice to the community at large. Sure, release your own product alongside it, but making it closed source does not help the scientific community upon which all these innovations were built. Especially if you have lost a loved one to a disease which this technology will one day be able to create cures for, it is very disappointing.</div><br/><div id="40306273" class="c"><input type="checkbox" id="c-40306273" checked=""/><div class="controls bullet"><span class="by">robertlagrant</span><span>|</span><a href="#40300996">parent</a><span>|</span><a href="#40302319">next</a><span>|</span><label class="collapse" for="c-40306273">[-]</label><label class="expand" for="c-40306273">[1 more]</label></div><br/><div class="children"><div class="content">&gt; the community at large<p>Which community? Not any I&#x27;m part of.</div><br/></div></div><div id="40302319" class="c"><input type="checkbox" id="c-40302319" checked=""/><div class="controls bullet"><span class="by">falcor84</span><span>|</span><a href="#40300996">parent</a><span>|</span><a href="#40306273">prev</a><span>|</span><a href="#40301141">next</a><span>|</span><label class="collapse" for="c-40302319">[-]</label><label class="expand" for="c-40302319">[16 more]</label></div><br/><div class="children"><div class="content">The closer it gets to enabling full drug discovery, the closer it also gets to enabling bioterrorism. Taking it to the extreme, if they had the theory of everything, I don&#x27;t think I&#x27;d want it to be made available to the whole world as it is today.<p>On a related note, I highly recommend The Talos Principle 2, which really made me think about these questions.</div><br/><div id="40302744" class="c"><input type="checkbox" id="c-40302744" checked=""/><div class="controls bullet"><span class="by">pythonguython</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40302319">parent</a><span>|</span><a href="#40306299">next</a><span>|</span><label class="collapse" for="c-40302744">[-]</label><label class="expand" for="c-40302744">[12 more]</label></div><br/><div class="children"><div class="content">Any organization&#x2F;country that has the ability to use a tool like this to create a bio weapon is already sophisticated enough to do bioterrorism today.</div><br/><div id="40303358" class="c"><input type="checkbox" id="c-40303358" checked=""/><div class="controls bullet"><span class="by">ramon156</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40302744">parent</a><span>|</span><a href="#40306299">next</a><span>|</span><label class="collapse" for="c-40303358">[-]</label><label class="expand" for="c-40303358">[11 more]</label></div><br/><div class="children"><div class="content">Alright, but now picture this: it&#x27;s now open to the masses, meaning an individual could probably even do it.</div><br/><div id="40305817" class="c"><input type="checkbox" id="c-40305817" checked=""/><div class="controls bullet"><span class="by">tsimionescu</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303358">parent</a><span>|</span><a href="#40303617">next</a><span>|</span><label class="collapse" for="c-40305817">[-]</label><label class="expand" for="c-40305817">[4 more]</label></div><br/><div class="children"><div class="content">The problem of producing bio weapons is not computational, it&#x27;s physical in nature. Even if predictions from these tools become 100% accurate and encompassed 100% of the chemistry, you still need to actually do the manual steps to breed the bioagents. And, very importantly, you need to do so without getting you and your co-workers deadly ill long before finishing the thing. Which requires extremely sophisticated machinery.<p>Alternatively, you can go today in some of the poorer corners of the world, find some people with drug resistant tuberculosis, pay them a pittance to give you bodily fluids, and disperse those in a large crowd, say at a concert or similar. You&#x27;ll get a good chunk of the effects of the worse possible bioterrorism.</div><br/><div id="40305845" class="c"><input type="checkbox" id="c-40305845" checked=""/><div class="controls bullet"><span class="by">hackernewds</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40305817">parent</a><span>|</span><a href="#40303617">next</a><span>|</span><label class="collapse" for="c-40305845">[-]</label><label class="expand" for="c-40305845">[3 more]</label></div><br/><div class="children"><div class="content">These are very specific ideas you have..</div><br/><div id="40305927" class="c"><input type="checkbox" id="c-40305927" checked=""/><div class="controls bullet"><span class="by">baq</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40305845">parent</a><span>|</span><a href="#40306093">next</a><span>|</span><label class="collapse" for="c-40305927">[-]</label><label class="expand" for="c-40305927">[1 more]</label></div><br/><div class="children"><div class="content">These are ideas you should read and think about because intelligence agencies all over the world have been thinking about them for the past hundred years.</div><br/></div></div><div id="40306093" class="c"><input type="checkbox" id="c-40306093" checked=""/><div class="controls bullet"><span class="by">BriggyDwiggs42</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40305845">parent</a><span>|</span><a href="#40305927">prev</a><span>|</span><a href="#40303617">next</a><span>|</span><label class="collapse" for="c-40306093">[-]</label><label class="expand" for="c-40306093">[1 more]</label></div><br/><div class="children"><div class="content">It’s very frustrating when people consider the possession of information equivalent to malice. It suggests that the right way to run society is to keep people stupid and harmless.</div><br/></div></div></div></div></div></div><div id="40303617" class="c"><input type="checkbox" id="c-40303617" checked=""/><div class="controls bullet"><span class="by">pythonguython</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303358">parent</a><span>|</span><a href="#40305817">prev</a><span>|</span><a href="#40303503">next</a><span>|</span><label class="collapse" for="c-40303617">[-]</label><label class="expand" for="c-40303617">[2 more]</label></div><br/><div class="children"><div class="content">I hear you, but I don&#x27;t think an individual can. If I gave you $20,000 and an open source 70% accurate protein folding model and told you to develop, mass produce, and develop a deployment mechanism for a highly infectious and deadly pathogen, I don&#x27;t think you could make that happen. Nor do I think you could do it if you had a PhD in microbiology.</div><br/><div id="40304608" class="c"><input type="checkbox" id="c-40304608" checked=""/><div class="controls bullet"><span class="by">bongodongobob</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303617">parent</a><span>|</span><a href="#40303503">next</a><span>|</span><label class="collapse" for="c-40304608">[-]</label><label class="expand" for="c-40304608">[1 more]</label></div><br/><div class="children"><div class="content"><i>right now</i></div><br/></div></div></div></div><div id="40303503" class="c"><input type="checkbox" id="c-40303503" checked=""/><div class="controls bullet"><span class="by">uptownfunk</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303358">parent</a><span>|</span><a href="#40303617">prev</a><span>|</span><a href="#40304214">next</a><span>|</span><label class="collapse" for="c-40303503">[-]</label><label class="expand" for="c-40303503">[2 more]</label></div><br/><div class="children"><div class="content">The risks don&#x27;t exceed what is already out there. If someone wants to do damage, especially in America, there are more than enough ways they can do it already. The technology should be made free. I also wonder how much the claims are being exaggerated and are marketing-speak vs. real results. Is there any benchmark for this that they have published?</div><br/><div id="40304603" class="c"><input type="checkbox" id="c-40304603" checked=""/><div class="controls bullet"><span class="by">bongodongobob</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303503">parent</a><span>|</span><a href="#40304214">next</a><span>|</span><label class="collapse" for="c-40304603">[-]</label><label class="expand" for="c-40304603">[1 more]</label></div><br/><div class="children"><div class="content">No. I love to be egalitarian as well but this AI thing <i>really</i> feels different. We didn&#x27;t just invent a better plow or a more durable sword. We&#x27;re working on making a better brain. I think social media shows us a pretty good slice of the average person and it&#x27;s not great. Now imagine they can manipulate the smartest person in the world to do dangerous, dumb shit.</div><br/></div></div></div></div><div id="40304214" class="c"><input type="checkbox" id="c-40304214" checked=""/><div class="controls bullet"><span class="by">consumer451</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40303358">parent</a><span>|</span><a href="#40303503">prev</a><span>|</span><a href="#40306299">next</a><span>|</span><label class="collapse" for="c-40304214">[-]</label><label class="expand" for="c-40304214">[2 more]</label></div><br/><div class="children"><div class="content">You raise an extremely important point. It appears to me that most people do not understand the implications of your point.<p>Organized terrorism by groups is actually extremely rare. What is much less rare are mass shootings in the USA, by deranged individuals.<p>What would a psychopathic mass shooter type choose as a weapon if he not only had access to semi-automatic weapons, but now we added bio-weapons to the menu?<p>It seems very clear to me that when creating custom viruses becomes high school level knowledge, and the tools can be charged on a credit card, nuclear weapons will be relegated to the second most likely way that our human civilization will end.<p>I believe the two concepts being brought together here are the Law of Large Numbers, and the sudden ability for one single human to kill at least millions.</div><br/><div id="40305843" class="c"><input type="checkbox" id="c-40305843" checked=""/><div class="controls bullet"><span class="by">tsimionescu</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40304214">parent</a><span>|</span><a href="#40306299">next</a><span>|</span><label class="collapse" for="c-40305843">[-]</label><label class="expand" for="c-40305843">[1 more]</label></div><br/><div class="children"><div class="content">&gt; It seems very clear to me that when creating custom viruses becomes high school level knowledge<p>That would be very bad indeed, but there is no path from AI to that. Making custom viruses is never going to be an easy task even if you had a magic machine that could explain the effects of adding any chemical to the mix. You still need to procure the chemicals and work with them in very careful ways, often for a long time, in a highly controlled environment. It&#x27;s still biology lab work, even if you know exactly what you have to do.<p>Also, bioweapons already exist and have been used in a few conflicts, even as recently as WWII. They&#x27;re terrifying in many ways, but are not really comparable to the horror of nuclear weapons hitting major cities.</div><br/></div></div></div></div></div></div></div></div><div id="40306299" class="c"><input type="checkbox" id="c-40306299" checked=""/><div class="controls bullet"><span class="by">datadeft</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40302319">parent</a><span>|</span><a href="#40302744">prev</a><span>|</span><a href="#40303187">next</a><span>|</span><label class="collapse" for="c-40306299">[-]</label><label class="expand" for="c-40306299">[1 more]</label></div><br/><div class="children"><div class="content">Why on Earth would any terrorist want to invest into this when you can just purchase guns and explosives much easier?</div><br/></div></div><div id="40303187" class="c"><input type="checkbox" id="c-40303187" checked=""/><div class="controls bullet"><span class="by">LouisSayers</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40302319">parent</a><span>|</span><a href="#40306299">prev</a><span>|</span><a href="#40303925">next</a><span>|</span><label class="collapse" for="c-40303187">[-]</label><label class="expand" for="c-40303187">[1 more]</label></div><br/><div class="children"><div class="content">Why do you need AI for bioterrorism? There are plenty of well known biological organisms that can kill us today...</div><br/></div></div><div id="40303925" class="c"><input type="checkbox" id="c-40303925" checked=""/><div class="controls bullet"><span class="by">BriggyDwiggs42</span><span>|</span><a href="#40300996">root</a><span>|</span><a href="#40302319">parent</a><span>|</span><a href="#40303187">prev</a><span>|</span><a href="#40301141">next</a><span>|</span><label class="collapse" for="c-40303925">[-]</label><label class="expand" for="c-40303925">[1 more]</label></div><br/><div class="children"><div class="content">Oh please, like a terrorist cant fork over a couple bucks to do the bioterrorism. This excuse is utter bs, whether its applied to LLMs or to alphafold. The motivator is profit, not safety.</div><br/></div></div></div></div></div></div><div id="40301141" class="c"><input type="checkbox" id="c-40301141" checked=""/><div class="controls bullet"><span class="by">LarsDu88</span><span>|</span><a href="#40300996">prev</a><span>|</span><a href="#40300061">next</a><span>|</span><label class="collapse" for="c-40301141">[-]</label><label class="expand" for="c-40301141">[7 more]</label></div><br/><div class="children"><div class="content">As a software engineer, I kind of feel uncomfortable about this new model. It outperforms Alphafold 2 at ligand binding, but Alphafold 2 also had some more hardcoded and interpretable structural reasoning baked into the model architecture.<p>There&#x27;s so many things you can incorporate into a protein folding model such as structural constraints, rotational equivariance, etc, etc<p>This new model simple does away with some of that, achieving greater results. And the authors simply use distillation from data outputted from Alphafold2 and Alphafold2-multimer to get those better results for those cases where you wind up with implausible results.<p>You have to run all those previous models, and output their predictions to do the distillation to achieve a real end-to-end training from scratch for this new model! Makes me feel a bit uncomfortable.</div><br/><div id="40301416" class="c"><input type="checkbox" id="c-40301416" checked=""/><div class="controls bullet"><span class="by">sangnoir</span><span>|</span><a href="#40301141">parent</a><span>|</span><a href="#40301399">next</a><span>|</span><label class="collapse" for="c-40301416">[-]</label><label class="expand" for="c-40301416">[2 more]</label></div><br/><div class="children"><div class="content">&gt; Makes me feel a bit uncomfortable.<p>Why? Do compilers which can&#x27;t bootstrap themselves also make you uncomfortable due to dependencies on pre-built artifacts? I&#x27;m not saying you&#x27;re unjustified to feel that way, but sometimes more abstracted systems are quicker to build and may have better performance than those built from the ground up. Selecting which one is better depends on your constraints and taste</div><br/><div id="40305597" class="c"><input type="checkbox" id="c-40305597" checked=""/><div class="controls bullet"><span class="by">arjvik</span><span>|</span><a href="#40301141">root</a><span>|</span><a href="#40301416">parent</a><span>|</span><a href="#40301399">next</a><span>|</span><label class="collapse" for="c-40305597">[-]</label><label class="expand" for="c-40305597">[1 more]</label></div><br/><div class="children"><div class="content">Compilers are deterministic (for the most part, and it&#x27;s incredibly rare to introduce a compiler bug that self-replicates in future compilers (unless you&#x27;re Ken Thompson and are reflecting upon trust itself)).<p>Alternatively, AlphaFold 2&#x27;s output is noisy, and using that to train AlphaFold 3, which presumably may be used to train what becomes AlphaFold 4, results in a cascade of errors.</div><br/></div></div></div></div><div id="40301399" class="c"><input type="checkbox" id="c-40301399" checked=""/><div class="controls bullet"><span class="by">amitport</span><span>|</span><a href="#40301141">parent</a><span>|</span><a href="#40301416">prev</a><span>|</span><a href="#40300061">next</a><span>|</span><label class="collapse" for="c-40301399">[-]</label><label class="expand" for="c-40301399">[4 more]</label></div><br/><div class="children"><div class="content">Consider that humans also learn from other humans, and sometimes surpass their teachers.<p>A bit more comfortable?</div><br/><div id="40301572" class="c"><input type="checkbox" id="c-40301572" checked=""/><div class="controls bullet"><span class="by">Balgair</span><span>|</span><a href="#40301141">root</a><span>|</span><a href="#40301399">parent</a><span>|</span><a href="#40300061">next</a><span>|</span><label class="collapse" for="c-40301572">[-]</label><label class="expand" for="c-40301572">[3 more]</label></div><br/><div class="children"><div class="content">Ahh, but the new young master is able to explain their work and processes to the satisfaction of the old masters. In the &#x27;Science&#x27; of our modern times it&#x27;s a requirement to show your work (yes, yes, I know about the replication crisis and all that terrible jazz).<p>Not being able to ascertain how and why the ML&#x2F;AI is achieving results is not quite the same and more akin to the alchemists and sorcerers with their cyphers and hidden laboratories.</div><br/><div id="40302349" class="c"><input type="checkbox" id="c-40302349" checked=""/><div class="controls bullet"><span class="by">falcor84</span><span>|</span><a href="#40301141">root</a><span>|</span><a href="#40301572">parent</a><span>|</span><a href="#40304097">next</a><span>|</span><label class="collapse" for="c-40302349">[-]</label><label class="expand" for="c-40302349">[1 more]</label></div><br/><div class="children"><div class="content">&gt; the new young master is able to explain their work and processes to the satisfaction of the old masters<p>Yes, but it&#x27;s one level deep - 
in general they wouldn&#x27;t be able to explain their work to their master&#x27;s master (note &quot;science advances one funeral at a time&quot;).</div><br/></div></div><div id="40304097" class="c"><input type="checkbox" id="c-40304097" checked=""/><div class="controls bullet"><span class="by">hackerdood</span><span>|</span><a href="#40301141">root</a><span>|</span><a href="#40301572">parent</a><span>|</span><a href="#40302349">prev</a><span>|</span><a href="#40300061">next</a><span>|</span><label class="collapse" for="c-40304097">[-]</label><label class="expand" for="c-40304097">[1 more]</label></div><br/><div class="children"><div class="content">I’ll add that specially when it comes to playing go, professionals who are at the peak of their ability can often find the best move at a given point but be unable to explain why beyond “it feels right” or “it looks right”.</div><br/></div></div></div></div></div></div></div></div><div id="40300061" class="c"><input type="checkbox" id="c-40300061" checked=""/><div class="controls bullet"><span class="by">mchinen</span><span>|</span><a href="#40301141">prev</a><span>|</span><a href="#40306052">next</a><span>|</span><label class="collapse" for="c-40300061">[-]</label><label class="expand" for="c-40300061">[1 more]</label></div><br/><div class="children"><div class="content">I am trying to understand how accurate the docking predictions are.<p>Looking at the PoseBusters paper [1] they mention, they say they are 50% more accurate than traditional methods.<p>DiffDock, which is the best DL based systems gets 30-70% depending on the dataset, and traditional gets 50-70%. The paper highlighted some issues with the DL-based methods and given that DeepMind would have had time to incorporate this into their work and develop with the PoseBusters paper in mind, I&#x27;d hope it&#x27;s significantly better than 50-70%. They say 50% better than traditional so I expected something like 70-85% across all datasets.<p>I hope a paper will appear soon to illuminate these and other details.<p>[1] <a href="https:&#x2F;&#x2F;pubs.rsc.org&#x2F;en&#x2F;content&#x2F;articlehtml&#x2F;2024&#x2F;sc&#x2F;d3sc04185a" rel="nofollow">https:&#x2F;&#x2F;pubs.rsc.org&#x2F;en&#x2F;content&#x2F;articlehtml&#x2F;2024&#x2F;sc&#x2F;d3sc0418...</a></div><br/></div></div><div id="40306052" class="c"><input type="checkbox" id="c-40306052" checked=""/><div class="controls bullet"><span class="by">gajnadsgjoas</span><span>|</span><a href="#40300061">prev</a><span>|</span><a href="#40299697">next</a><span>|</span><label class="collapse" for="c-40306052">[-]</label><label class="expand" for="c-40306052">[1 more]</label></div><br/><div class="children"><div class="content">Can someone tell me what are the direct implication of this? I often see &quot;helps with a drug design&quot; but I&#x27;m too far from this industry and have never seen an example of such drugs</div><br/></div></div><div id="40299697" class="c"><input type="checkbox" id="c-40299697" checked=""/><div class="controls bullet"><span class="by">qwertox</span><span>|</span><a href="#40306052">prev</a><span>|</span><a href="#40301639">next</a><span>|</span><label class="collapse" for="c-40299697">[-]</label><label class="expand" for="c-40299697">[4 more]</label></div><br/><div class="children"><div class="content">&gt; Thrilled to announce AlphaFold 3 which can predict the structures and interactions of nearly all of life’s molecules with state-of-the-art accuracy including proteins, DNA and RNA. [1]<p>There&#x27;s a slight mismatch between the blog&#x27;s title and Demis Hassabis&#x27; tweet, where he uses &quot;nearly all&quot;.<p>The blog&#x27;s title suggests that it&#x27;s a 100% solved problem.<p>[1] <a href="https:&#x2F;&#x2F;twitter.com&#x2F;demishassabis&#x2F;status&#x2F;1788229162563420560" rel="nofollow">https:&#x2F;&#x2F;twitter.com&#x2F;demishassabis&#x2F;status&#x2F;1788229162563420560</a></div><br/><div id="40303302" class="c"><input type="checkbox" id="c-40303302" checked=""/><div class="controls bullet"><span class="by">TaupeRanger</span><span>|</span><a href="#40299697">parent</a><span>|</span><a href="#40304104">next</a><span>|</span><label class="collapse" for="c-40303302">[-]</label><label class="expand" for="c-40303302">[1 more]</label></div><br/><div class="children"><div class="content">First time reading a Deep Mind PR? This is literally their modus operandi.</div><br/></div></div><div id="40304104" class="c"><input type="checkbox" id="c-40304104" checked=""/><div class="controls bullet"><span class="by">bamboozled</span><span>|</span><a href="#40299697">parent</a><span>|</span><a href="#40303302">prev</a><span>|</span><a href="#40299906">next</a><span>|</span><label class="collapse" for="c-40304104">[-]</label><label class="expand" for="c-40304104">[1 more]</label></div><br/><div class="children"><div class="content">How to make the share price go up…surprised?</div><br/></div></div><div id="40299906" class="c"><input type="checkbox" id="c-40299906" checked=""/><div class="controls bullet"><span class="by">bmau5</span><span>|</span><a href="#40299697">parent</a><span>|</span><a href="#40304104">prev</a><span>|</span><a href="#40301639">next</a><span>|</span><label class="collapse" for="c-40299906">[-]</label><label class="expand" for="c-40299906">[1 more]</label></div><br/><div class="children"><div class="content">Marketing vs. Reality :)</div><br/></div></div></div></div><div id="40301639" class="c"><input type="checkbox" id="c-40301639" checked=""/><div class="controls bullet"><span class="by">wuj</span><span>|</span><a href="#40299697">prev</a><span>|</span><a href="#40303439">next</a><span>|</span><label class="collapse" for="c-40301639">[-]</label><label class="expand" for="c-40301639">[3 more]</label></div><br/><div class="children"><div class="content">This tool reminds me that the human body functions much like a black box. While physics can be modeled with equations and constraints, biology is inherently probabilistic and unpredictable. We verify the efficacy of a medicine by observing its outcomes: the medicine is the input, and the changes in symptoms are the output. However, we cannot model what happens in between, as we cannot definitively prove that the medicine affects only its intended targets. In many ways, much of what we understand about medicine is based on observing these black-box processes, and this tool helps to model that complexity.</div><br/><div id="40305283" class="c"><input type="checkbox" id="c-40305283" checked=""/><div class="controls bullet"><span class="by">a_bonobo</span><span>|</span><a href="#40301639">parent</a><span>|</span><a href="#40304131">next</a><span>|</span><label class="collapse" for="c-40305283">[-]</label><label class="expand" for="c-40305283">[1 more]</label></div><br/><div class="children"><div class="content">Classic essay in this vein:<p>&gt;Can a biologist fix a radio? — Or, what I learned while studying apoptosis<p><a href="https:&#x2F;&#x2F;www.cell.com&#x2F;cancer-cell&#x2F;pdf&#x2F;S1535-6108(02)00133-2.pdf" rel="nofollow">https:&#x2F;&#x2F;www.cell.com&#x2F;cancer-cell&#x2F;pdf&#x2F;S1535-6108(02)00133-2.p...</a><p>&gt;However, if the radio has tunable components, such as those
found in my old radio (indicated by yellow arrows in Figure 2,
inset) and in all live cells and organisms, the outcome will not be
so promising. Indeed, the radio may not work because several
components are not tuned properly, which is not reflected in their
appearance or their connections. What is the probability that this
radio will be fixed by our biologists? I might be overly pessimistic,
but a textbook example of the monkey that can, in principle, type
a Burns poem comes to mind. In other words, the radio will not
play music unless that lucky chance meets a prepared mind.</div><br/></div></div><div id="40304131" class="c"><input type="checkbox" id="c-40304131" checked=""/><div class="controls bullet"><span class="by">bamboozled</span><span>|</span><a href="#40301639">parent</a><span>|</span><a href="#40305283">prev</a><span>|</span><a href="#40303439">next</a><span>|</span><label class="collapse" for="c-40304131">[-]</label><label class="expand" for="c-40304131">[1 more]</label></div><br/><div class="children"><div class="content">I’d say it’s always been the case for medicine, when people first used medicines, the intention was never to fully understand what happens, just save a life, eliminate or reduce symptoms.<p>Now we’ve built explainable systems like computers and software, we try to overlay that onto everything and it might not work.<p>To quote Alan Watts, humans like to try square out wiggly systems because we’re not great and understanding wiggles.</div><br/></div></div></div></div><div id="40303439" class="c"><input type="checkbox" id="c-40303439" checked=""/><div class="controls bullet"><span class="by">zmmmmm</span><span>|</span><a href="#40301639">prev</a><span>|</span><a href="#40299360">next</a><span>|</span><label class="collapse" for="c-40303439">[-]</label><label class="expand" for="c-40303439">[1 more]</label></div><br/><div class="children"><div class="content">So much of the talk about their &quot;free server&quot; seems to be trying to distract from the fact that they are not releasing the model.<p>I feel like it&#x27;s an important threshold moment if this gets accepted into scientific use without the model being available - reproducibility of results becomes dependent on the good graces of a single commercial entity. I kind of hope that like OpenAI it just spurs creation of equivalent open models that then actually get used.</div><br/></div></div><div id="40299360" class="c"><input type="checkbox" id="c-40299360" checked=""/><div class="controls bullet"><span class="by">tea-coffee</span><span>|</span><a href="#40303439">prev</a><span>|</span><a href="#40302304">next</a><span>|</span><label class="collapse" for="c-40299360">[-]</label><label class="expand" for="c-40299360">[15 more]</label></div><br/><div class="children"><div class="content">This is a basic question, but how is the accuracy of the predicted biomolecular interactions measured? Are the predicted interactions compared to known interactions? How would the accuracy of predicting unknown interactions be assessed?</div><br/><div id="40299419" class="c"><input type="checkbox" id="c-40299419" checked=""/><div class="controls bullet"><span class="by">joshuamcginnis</span><span>|</span><a href="#40299360">parent</a><span>|</span><a href="#40302304">next</a><span>|</span><label class="collapse" for="c-40299419">[-]</label><label class="expand" for="c-40299419">[14 more]</label></div><br/><div class="children"><div class="content">Accuracy can be assessed two main ways: computationally and experimentally. Computationally, they would compare the predicted structures and interactions with known data from databases like PDB (Protein Database). Experimentally, they can use tools like x-ray crystallography and NMR (nuclear magnetic resonance) to obtain the actual molecule structure and compare it to the predicted result. The outcomes of each approach would be fed back into the model for refining future predictions.<p><a href="https:&#x2F;&#x2F;www.rcsb.org&#x2F;" rel="nofollow">https:&#x2F;&#x2F;www.rcsb.org&#x2F;</a></div><br/><div id="40299495" class="c"><input type="checkbox" id="c-40299495" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299419">parent</a><span>|</span><a href="#40302304">next</a><span>|</span><label class="collapse" for="c-40299495">[-]</label><label class="expand" for="c-40299495">[13 more]</label></div><br/><div class="children"><div class="content">AlphaFold very explicitly (unless something has changed) removes NMR structures as references because they are not accurate enough.  I have a PhD in NMR biomolecular structure and I wouldn&#x27;t trust. the structures for anything.</div><br/><div id="40299614" class="c"><input type="checkbox" id="c-40299614" checked=""/><div class="controls bullet"><span class="by">JackFr</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299495">parent</a><span>|</span><a href="#40299636">next</a><span>|</span><label class="collapse" for="c-40299614">[-]</label><label class="expand" for="c-40299614">[4 more]</label></div><br/><div class="children"><div class="content">Sorry, I don’t mean to be dense - do you mean you don’t trust AlphaFolds structures or NMRs?</div><br/><div id="40299872" class="c"><input type="checkbox" id="c-40299872" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299614">parent</a><span>|</span><a href="#40299636">next</a><span>|</span><label class="collapse" for="c-40299872">[-]</label><label class="expand" for="c-40299872">[3 more]</label></div><br/><div class="children"><div class="content">I don&#x27;t trust NMR structures in nearly all cases.  The reasons are complex enough that I don&#x27;t think it&#x27;s worthwhile to discuss on Hacker News.</div><br/><div id="40300241" class="c"><input type="checkbox" id="c-40300241" checked=""/><div class="controls bullet"><span class="by">fikama</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299872">parent</a><span>|</span><a href="#40299636">next</a><span>|</span><label class="collapse" for="c-40300241">[-]</label><label class="expand" for="c-40300241">[2 more]</label></div><br/><div class="children"><div class="content">Hmm, I would say its always worth to share knowledge. Could you paste some links or maybe type a few key-words for anyone willing to reasearch the topic further on his own.</div><br/><div id="40300670" class="c"><input type="checkbox" id="c-40300670" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40300241">parent</a><span>|</span><a href="#40299636">next</a><span>|</span><label class="collapse" for="c-40300670">[-]</label><label class="expand" for="c-40300670">[1 more]</label></div><br/><div class="children"><div class="content">Read this, and recursively (breadth-first) read all its transitive references:
<a href="https:&#x2F;&#x2F;www.sciencedirect.com&#x2F;science&#x2F;article&#x2F;pii&#x2F;S0969212621002513" rel="nofollow">https:&#x2F;&#x2F;www.sciencedirect.com&#x2F;science&#x2F;article&#x2F;pii&#x2F;S096921262...</a></div><br/></div></div></div></div></div></div></div></div><div id="40299636" class="c"><input type="checkbox" id="c-40299636" checked=""/><div class="controls bullet"><span class="by">fabian2k</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299495">parent</a><span>|</span><a href="#40299614">prev</a><span>|</span><a href="#40302418">next</a><span>|</span><label class="collapse" for="c-40299636">[-]</label><label class="expand" for="c-40299636">[5 more]</label></div><br/><div class="children"><div class="content">Looking at the supplementary material (section 2.5.4) for the AlphaFold 3 paper it reads to me like they still use NMR structures for training, but not for evaluating performance of the model.</div><br/><div id="40299758" class="c"><input type="checkbox" id="c-40299758" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299636">parent</a><span>|</span><a href="#40302418">next</a><span>|</span><label class="collapse" for="c-40299758">[-]</label><label class="expand" for="c-40299758">[4 more]</label></div><br/><div class="children"><div class="content">I think it&#x27;s implicit in their description of filtering the training set, where they say they only include structures with resolution of 9A or less.  NMR structures don&#x27;t really have a resolution, that&#x27;s more specific to crystallography.  However, I can&#x27;t actually verify that no NMR structures were included without directly inspecting their list of selected structures.</div><br/><div id="40299790" class="c"><input type="checkbox" id="c-40299790" checked=""/><div class="controls bullet"><span class="by">fabian2k</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299758">parent</a><span>|</span><a href="#40302418">next</a><span>|</span><label class="collapse" for="c-40299790">[-]</label><label class="expand" for="c-40299790">[3 more]</label></div><br/><div class="children"><div class="content">I think it is very plausible that they don&#x27;t use NMR structures here, but I was looking for a specific statement on it in the paper. I think your guess is plausible, but I don&#x27;t think the paper is clear enough here to be sure about this interpretation.</div><br/><div id="40299820" class="c"><input type="checkbox" id="c-40299820" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299790">parent</a><span>|</span><a href="#40302418">next</a><span>|</span><label class="collapse" for="c-40299820">[-]</label><label class="expand" for="c-40299820">[2 more]</label></div><br/><div class="children"><div class="content">Yes, thanks for calling that out.  In verifying my statement I actually was confused because you can see they filter NMR out of the eval set (saying so explicitly) but don&#x27;t say that in the test set section (IMHO they should be required to publish the actual selection script so we can inspect the results).</div><br/><div id="40299919" class="c"><input type="checkbox" id="c-40299919" checked=""/><div class="controls bullet"><span class="by">fabian2k</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299820">parent</a><span>|</span><a href="#40302418">next</a><span>|</span><label class="collapse" for="c-40299919">[-]</label><label class="expand" for="c-40299919">[1 more]</label></div><br/><div class="children"><div class="content">Hmm, in the earlier AlphaFold 2 paper they state:<p>&gt; Input mmCIFs are restricted to have resolution less than 9 Å. This is not a very restrictive filter and only removes around 0.2% of structures<p>NMR structures are more than 0.2% so that doesn&#x27;t fit to the assumption that they implicitly remove NMR structures here. But if I filter by resolution on the PDB homepage it does remove essentially all NMR structures. I&#x27;m really not sure what to think here, the description seems too soft to know what they did exactly.</div><br/></div></div></div></div></div></div></div></div></div></div><div id="40302418" class="c"><input type="checkbox" id="c-40302418" checked=""/><div class="controls bullet"><span class="by">panabee</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299495">parent</a><span>|</span><a href="#40299636">prev</a><span>|</span><a href="#40303209">next</a><span>|</span><label class="collapse" for="c-40302418">[-]</label><label class="expand" for="c-40302418">[2 more]</label></div><br/><div class="children"><div class="content">interesting observation and experience. must have made thesis development complex, assuming the realization dawned on you during the phd.<p>what do you trust more than NMR?<p>AF&#x27;s dependence on MSAs also seems sub-optimal; curious to hear your thoughts?<p>that said, it&#x27;s understandable why they used MSAs, even if it seems to hint at winning CASP more than developing a generalizable model.<p>arguably, MSA-dependence is the wise choice for early prediction models as demonstrated by widespread accolades and adoption, i.e., it&#x27;s an MVP with known limitations as they build toward sophisticated approaches.</div><br/><div id="40302767" class="c"><input type="checkbox" id="c-40302767" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40302418">parent</a><span>|</span><a href="#40303209">next</a><span>|</span><label class="collapse" for="c-40302767">[-]</label><label class="expand" for="c-40302767">[1 more]</label></div><br/><div class="children"><div class="content">My realizations happened after my PhD.  When I was writing my PhD I still believed we would solve the protein folding and structure prediction problems using classical empirical force fields.<p>It wasn&#x27;t until I started my postdocs, where I started learning about protein evolutionary relationships (and competing in CASP), that I changed my mind.
I wouldn&#x27;t say it so much as &quot;multiple sequence alignments&quot;; those are just tools to express protein relationships in a structured way.<p>If Alphafold now, or in the future, requires no evolutionary relationships based on sequence (uniprot) and can work entirely by training on just the proteins in PDB (many of which <i>are</i> evoutionarily related) and still be able to predict novel folds, it will be very interesting times.  The one thing I have learned is that evolutionary knowledge makes many hard problems really easy, because you&#x27;re taking advantage of billions of years of nature and an easy readout.</div><br/></div></div></div></div><div id="40303209" class="c"><input type="checkbox" id="c-40303209" checked=""/><div class="controls bullet"><span class="by">heyoni</span><span>|</span><a href="#40299360">root</a><span>|</span><a href="#40299495">parent</a><span>|</span><a href="#40302418">prev</a><span>|</span><a href="#40302304">next</a><span>|</span><label class="collapse" for="c-40303209">[-]</label><label class="expand" for="c-40303209">[1 more]</label></div><br/><div class="children"><div class="content">Nice to see you on this thread as well! :)</div><br/></div></div></div></div></div></div></div></div><div id="40302304" class="c"><input type="checkbox" id="c-40302304" checked=""/><div class="controls bullet"><span class="by">itissid</span><span>|</span><a href="#40299360">prev</a><span>|</span><a href="#40303901">next</a><span>|</span><label class="collapse" for="c-40302304">[-]</label><label class="expand" for="c-40302304">[1 more]</label></div><br/><div class="children"><div class="content">Noob here. Can one make the following deduction:<p>In transformer based architectures, where one typically uses variation of attention mechanism to model interactions, even if one does not consider the autoregressive assumption of the domain&#x27;s &quot;nodes&quot;(amino acids, words, image patches), if the number of final states that nodes take eventually can be permuted only in a finite way(i.e. they have sparse interactions between them), then these architectures are efficient way of modeling such domains.<p>In plain english the final state of words in a sentence and amino acids in a protein have only so many ways they can be arranged and transformers do a good job of modeling it.<p>Also can one assume this won&#x27;t do well for domains where there is, say, sensitivity to initial conditions, like chaotic systems like wheather where the # final states just explodes?</div><br/></div></div><div id="40303901" class="c"><input type="checkbox" id="c-40303901" checked=""/><div class="controls bullet"><span class="by">nsoonhui</span><span>|</span><a href="#40302304">prev</a><span>|</span><a href="#40299579">next</a><span>|</span><label class="collapse" for="c-40303901">[-]</label><label class="expand" for="c-40303901">[4 more]</label></div><br/><div class="children"><div class="content">Here&#x27;s something that bugs me about ML: all we have is prediction and no explanation how we come to that prediction, ie: no deeper understanding on the underlying principles.<p>So despite that we got a good match this time, how can we be sure that the match will be equally good next time? And how to use ML to predict structure that we have no baseline to start with or experimental result to benchmark ? In the absence of physics-like principles, How can we ever be sure that ML results next time is correct ?</div><br/><div id="40306207" class="c"><input type="checkbox" id="c-40306207" checked=""/><div class="controls bullet"><span class="by">coriny</span><span>|</span><a href="#40303901">parent</a><span>|</span><a href="#40303941">next</a><span>|</span><label class="collapse" for="c-40306207">[-]</label><label class="expand" for="c-40306207">[1 more]</label></div><br/><div class="children"><div class="content">There is a biannual structural prediction contest called CASP [1], in which a set of newly determined structures is used to benchmark the prediction methods. Some of these structures will be &quot;novel&quot;, and so can be used to estimate the performance of current methods on predicting &quot;structure that we have no baseline to start with&quot;.<p>CASP-style assessments are something that should done for more research fields, but it&#x27;s really hard to persuade funders and researchers to put up the money and embargo the data as required.<p>[1] <a href="https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;CASP" rel="nofollow">https:&#x2F;&#x2F;en.wikipedia.org&#x2F;wiki&#x2F;CASP</a></div><br/></div></div><div id="40303941" class="c"><input type="checkbox" id="c-40303941" checked=""/><div class="controls bullet"><span class="by">throwaway4aday</span><span>|</span><a href="#40303901">parent</a><span>|</span><a href="#40306207">prev</a><span>|</span><a href="#40299579">next</a><span>|</span><label class="collapse" for="c-40303941">[-]</label><label class="expand" for="c-40303941">[2 more]</label></div><br/><div class="children"><div class="content">Speaking of physics, we should borrow the quote &quot;Shut up and calculate&quot; to describe the situation: it works so use it now and worry about the explanations later.</div><br/><div id="40305763" class="c"><input type="checkbox" id="c-40305763" checked=""/><div class="controls bullet"><span class="by">d0mine</span><span>|</span><a href="#40303901">root</a><span>|</span><a href="#40303941">parent</a><span>|</span><a href="#40299579">next</a><span>|</span><label class="collapse" for="c-40305763">[-]</label><label class="expand" for="c-40305763">[1 more]</label></div><br/><div class="children"><div class="content">Except the model is not open-source. You can&#x27;t calculate anything.</div><br/></div></div></div></div></div></div><div id="40299579" class="c"><input type="checkbox" id="c-40299579" checked=""/><div class="controls bullet"><span class="by">_xerces_</span><span>|</span><a href="#40303901">prev</a><span>|</span><a href="#40301880">next</a><span>|</span><label class="collapse" for="c-40299579">[-]</label><label class="expand" for="c-40299579">[2 more]</label></div><br/><div class="children"><div class="content">A video summary of why this research is important:
<a href="https:&#x2F;&#x2F;youtu.be&#x2F;Mz7Qp73lj9o?si=29vjdQtTtIOk_0CV" rel="nofollow">https:&#x2F;&#x2F;youtu.be&#x2F;Mz7Qp73lj9o?si=29vjdQtTtIOk_0CV</a></div><br/><div id="40300170" class="c"><input type="checkbox" id="c-40300170" checked=""/><div class="controls bullet"><span class="by">ProllyInfamous</span><span>|</span><a href="#40299579">parent</a><span>|</span><a href="#40301880">next</a><span>|</span><label class="collapse" for="c-40300170">[-]</label><label class="expand" for="c-40300170">[1 more]</label></div><br/><div class="children"><div class="content">Thanks for this informative video summary. As a layperson, with a BS in Chemistry, it was quite helpful in understanding main bulletpoints of this accomplishment.</div><br/></div></div></div></div><div id="40301880" class="c"><input type="checkbox" id="c-40301880" checked=""/><div class="controls bullet"><span class="by">ak_111</span><span>|</span><a href="#40299579">prev</a><span>|</span><a href="#40300322">next</a><span>|</span><label class="collapse" for="c-40301880">[-]</label><label class="expand" for="c-40301880">[1 more]</label></div><br/><div class="children"><div class="content">If you work in this space would be interested to know what <i>material</i> impact has alphafold caused in your workflow since its release 4 years ago?</div><br/></div></div><div id="40300322" class="c"><input type="checkbox" id="c-40300322" checked=""/><div class="controls bullet"><span class="by">dsign</span><span>|</span><a href="#40301880">prev</a><span>|</span><a href="#40299217">next</a><span>|</span><label class="collapse" for="c-40300322">[-]</label><label class="expand" for="c-40300322">[2 more]</label></div><br/><div class="children"><div class="content">For a couple of years I&#x27;ve been expecting that ML models would be able to &#x27;accelerate&#x27; bio-molecular simulations, using physics-based simulations as ground  truth. But this seems to be a step beyond that.</div><br/><div id="40301531" class="c"><input type="checkbox" id="c-40301531" checked=""/><div class="controls bullet"><span class="by">dekhn</span><span>|</span><a href="#40300322">parent</a><span>|</span><a href="#40299217">next</a><span>|</span><label class="collapse" for="c-40301531">[-]</label><label class="expand" for="c-40301531">[1 more]</label></div><br/><div class="children"><div class="content">When I competed in CASP 20 years ago (and lost terribly) I predicted that the next step to improve predictions would be to develop empirically fitted force fields to make MD produce accurate structure predictions (MD already uses empirically fitted force fields, but they are not great).  This area was explored, there are now better force fields, but that didn&#x27;t really push protein structure prediction forward.<p>Another approach is fully differentiable force fields- the idea that the force field function itself is a trainable structure (rather than just the parameters&#x2F;weights&#x2F;constants) that can be optimized directly towards a goal.  Also explored, produced some interesting results, but nothing that woudl be considered transformative.<p>The field still generally believes that if you had a perfect force field and infinite computing time, you could directly recapitulate the trajectories of proteins folding (from fully unfolded to final state along with all the intermediates), but that doesn&#x27;t address any practical problems, and is massively wasteful of resources compared to using ML models that exploit evolutionary information encoded in sequence and structures.<p>In retrospect I&#x27;m pretty relieved I was wrong, as the new methods are more effective with far fewer resources.</div><br/></div></div></div></div><div id="40299217" class="c"><input type="checkbox" id="c-40299217" checked=""/><div class="controls bullet"><span class="by">s1artibartfast</span><span>|</span><a href="#40300322">prev</a><span>|</span><a href="#40302340">next</a><span>|</span><label class="collapse" for="c-40299217">[-]</label><label class="expand" for="c-40299217">[6 more]</label></div><br/><div class="children"><div class="content">The article was heavy on the free research aspect, but light on the commercial application.<p>I&#x27;m curious about the business strategy. Does Google intend to license out tools, partner, or consult for  commercial partners?</div><br/><div id="40304678" class="c"><input type="checkbox" id="c-40304678" checked=""/><div class="controls bullet"><span class="by">a_bonobo</span><span>|</span><a href="#40299217">parent</a><span>|</span><a href="#40299628">next</a><span>|</span><label class="collapse" for="c-40304678">[-]</label><label class="expand" for="c-40304678">[1 more]</label></div><br/><div class="children"><div class="content">This version has Isomorphic Labs far more in the focus of the press release, which seems to be now the commercial arm more or less licensing access out.<p>The new AlphaFold server does not do everything the paper says AlphaFold 3 says it does. You cannot predict docking with the server! That is the main interest of pharma companies, &#x27;does our medication bind to the target protein?&#x27;. From the FAQ: &#x27;AlphaFold Server is a web-service that offers customized biomolecular structure prediction. It makes several newer AlphaFold3 capabilities available, including support for a wider range of molecule type&#x27; - that&#x27;s not ALL AlphaFold3 capabilities. Isomorphic prints the money with those additional capabilities.<p>It&#x27;s hilarious that Google says they don&#x27;t allow this for safety reasons, pure OpenAI fluff. It&#x27;s just money.</div><br/></div></div><div id="40299628" class="c"><input type="checkbox" id="c-40299628" checked=""/><div class="controls bullet"><span class="by">candiodari</span><span>|</span><a href="#40299217">parent</a><span>|</span><a href="#40304678">prev</a><span>|</span><a href="#40299485">next</a><span>|</span><label class="collapse" for="c-40299628">[-]</label><label class="expand" for="c-40299628">[1 more]</label></div><br/><div class="children"><div class="content">I wonder what the license for RoseTTAFold  is. On github you have:<p><a href="https:&#x2F;&#x2F;github.com&#x2F;RosettaCommons&#x2F;RoseTTAFold&#x2F;blob&#x2F;main&#x2F;LICENSE">https:&#x2F;&#x2F;github.com&#x2F;RosettaCommons&#x2F;RoseTTAFold&#x2F;blob&#x2F;main&#x2F;LICE...</a><p>But there&#x27;s also:<p><a href="https:&#x2F;&#x2F;files.ipd.uw.edu&#x2F;pub&#x2F;RoseTTAFold&#x2F;Rosetta-DL_LICENSE.txt" rel="nofollow">https:&#x2F;&#x2F;files.ipd.uw.edu&#x2F;pub&#x2F;RoseTTAFold&#x2F;Rosetta-DL_LICENSE....</a><p>Which is it?</div><br/></div></div><div id="40299303" class="c"><input type="checkbox" id="c-40299303" checked=""/><div class="controls bullet"><span class="by">ilrwbwrkhv</span><span>|</span><a href="#40299217">parent</a><span>|</span><a href="#40299485">prev</a><span>|</span><a href="#40302340">next</a><span>|</span><label class="collapse" for="c-40299303">[-]</label><label class="expand" for="c-40299303">[2 more]</label></div><br/><div class="children"><div class="content">as soon as google tries to think commercially this will shut down so the longer it stays pure research the better. google is bad with productization.</div><br/><div id="40299685" class="c"><input type="checkbox" id="c-40299685" checked=""/><div class="controls bullet"><span class="by">s1artibartfast</span><span>|</span><a href="#40299217">root</a><span>|</span><a href="#40299303">parent</a><span>|</span><a href="#40302340">next</a><span>|</span><label class="collapse" for="c-40299685">[-]</label><label class="expand" for="c-40299685">[1 more]</label></div><br/><div class="children"><div class="content">I don&#x27;t think it was ever pure research. The article talks about infinity labs, which is the co. Mercial branch for drug discovery.<p>I do agree that Google seems bad at commercialization, which is why I&#x27;m curious on what the strategy is.<p>It is hard to see them being paid consultants or effective partners for pharma companies, let alone developing drugs themselves.</div><br/></div></div></div></div></div></div><div id="40302340" class="c"><input type="checkbox" id="c-40302340" checked=""/><div class="controls bullet"><span class="by">ricksunny</span><span>|</span><a href="#40299217">prev</a><span>|</span><a href="#40302051">next</a><span>|</span><label class="collapse" for="c-40302340">[-]</label><label class="expand" for="c-40302340">[2 more]</label></div><br/><div class="children"><div class="content">I&#x27;m interested in how they measure accuracy of binding site identification and binding pose prediction.  This was missing for the hitherto widely-used binding pose prediction tool Autodock Vina (and in silico binding pose tools in general). Despite the time I invested in learning &amp; exercising that tool, I avoided using it for published research because I could not credibly cite its general-use accuracy.  Is &#x2F; will Alphafold 3 be citeable in the sense of &quot;I have run Alphafold on this particular target of interest and this array of ligands, and have found these poses of X kJ&#x2F;mol binding energy, and this is known to an accuracy of Y% because of Alphafold 3&#x27;s training set results cited below&#x27;</div><br/><div id="40302407" class="c"><input type="checkbox" id="c-40302407" checked=""/><div class="controls bullet"><span class="by">l33tman</span><span>|</span><a href="#40302340">parent</a><span>|</span><a href="#40302051">next</a><span>|</span><label class="collapse" for="c-40302407">[-]</label><label class="expand" for="c-40302407">[1 more]</label></div><br/><div class="children"><div class="content">I&#x27;ve never trusted those predicted binding energies. If you have predicted a ligand&#x2F;protein complex and have high confidence in it and want to study the binding energy I really think you should do a full MD simulation, you can pull the ligand-protein complex apart and measure the change in free energy explicitly.<p>Also, and this is an unfounded guess only, the problem of protein &#x2F; ligand docking is quite a bit more complex than protein folding - there seems to be a finite set of overall folds used in nature, while docking a small ligand to a big protein with flexible sidechains and even flexible large-scale structures can have induced fits that are really important to know and estimate, and I&#x27;m just very sceptical that it&#x27;s going to be possible to in a general fashion ever predict these accurately by the AI model with the limited training data.<p>Though you just need some hints, then you can run MD sims on them to see what happens for real.</div><br/></div></div></div></div><div id="40302051" class="c"><input type="checkbox" id="c-40302051" checked=""/><div class="controls bullet"><span class="by">lumb63</span><span>|</span><a href="#40302340">prev</a><span>|</span><a href="#40300433">next</a><span>|</span><label class="collapse" for="c-40302051">[-]</label><label class="expand" for="c-40302051">[1 more]</label></div><br/><div class="children"><div class="content">Would anyone more familiar with the field be able to provide some cursory resources on the protein folding problem? I have a background in computer science and a half a background in biology (took two semesters of OChem, biology, anatomy; didn’t go much further).</div><br/></div></div><div id="40300433" class="c"><input type="checkbox" id="c-40300433" checked=""/><div class="controls bullet"><span class="by">xnx</span><span>|</span><a href="#40302051">prev</a><span>|</span><a href="#40301405">next</a><span>|</span><label class="collapse" for="c-40300433">[-]</label><label class="expand" for="c-40300433">[1 more]</label></div><br/><div class="children"><div class="content">Very cool that anyone can login to <a href="https:&#x2F;&#x2F;golgi.sandbox.google.com&#x2F;" rel="nofollow">https:&#x2F;&#x2F;golgi.sandbox.google.com&#x2F;</a> and check it out</div><br/></div></div><div id="40301405" class="c"><input type="checkbox" id="c-40301405" checked=""/><div class="controls bullet"><span class="by">roody15</span><span>|</span><a href="#40300433">prev</a><span>|</span><a href="#40301441">next</a><span>|</span><label class="collapse" for="c-40301405">[-]</label><label class="expand" for="c-40301405">[1 more]</label></div><br/><div class="children"><div class="content">I wonder in the not too distant future if these AI predictions could be explained back into “humanized” understanding.  Much like ChatGPT can simplify complex topics … cold the model in the future provide feedback to researchers why it is making this prediction?</div><br/></div></div><div id="40301441" class="c"><input type="checkbox" id="c-40301441" checked=""/><div class="controls bullet"><span class="by">reliablereason</span><span>|</span><a href="#40301405">prev</a><span>|</span><a href="#40301452">next</a><span>|</span><label class="collapse" for="c-40301441">[-]</label><label class="expand" for="c-40301441">[1 more]</label></div><br/><div class="children"><div class="content">Would be very useful if one they used it to predict the structure and interaction of the known variants to.<p>Would be very helpful when predicting if a mutation on a protein would lead to loss of function for the protein.</div><br/></div></div><div id="40301452" class="c"><input type="checkbox" id="c-40301452" checked=""/><div class="controls bullet"><span class="by">mfld</span><span>|</span><a href="#40301441">prev</a><span>|</span><a href="#40301856">next</a><span>|</span><label class="collapse" for="c-40301452">[-]</label><label class="expand" for="c-40301452">[1 more]</label></div><br/><div class="children"><div class="content">The improvement on predicting protein&#x2F;RNA&#x2F;ligand interactions might facilitate many commercially relevant use cases. I assume pharma and biotech will eagerly get in line to use this.</div><br/></div></div><div id="40301856" class="c"><input type="checkbox" id="c-40301856" checked=""/><div class="controls bullet"><span class="by">thenerdhead</span><span>|</span><a href="#40301452">prev</a><span>|</span><a href="#40305338">next</a><span>|</span><label class="collapse" for="c-40301856">[-]</label><label class="expand" for="c-40301856">[1 more]</label></div><br/><div class="children"><div class="content">A lot of accelerated article previews as of recently. Seems like humanity is making a lot of breakthroughs.<p>This is nothing short of amazing for all those suffering from disease.</div><br/></div></div><div id="40305338" class="c"><input type="checkbox" id="c-40305338" checked=""/><div class="controls bullet"><span class="by">sidcool</span><span>|</span><a href="#40301856">prev</a><span>|</span><a href="#40299937">next</a><span>|</span><label class="collapse" for="c-40305338">[-]</label><label class="expand" for="c-40305338">[1 more]</label></div><br/><div class="children"><div class="content">And google is giving the service for free.  Pretty good.</div><br/></div></div><div id="40301765" class="c"><input type="checkbox" id="c-40301765" checked=""/><div class="controls bullet"><span class="by">bbstats</span><span>|</span><a href="#40299937">prev</a><span>|</span><a href="#40300819">next</a><span>|</span><label class="collapse" for="c-40301765">[-]</label><label class="expand" for="c-40301765">[1 more]</label></div><br/><div class="children"><div class="content">Zero-shot nearly beating trained catboost is pretty amazing.</div><br/></div></div><div id="40300819" class="c"><input type="checkbox" id="c-40300819" checked=""/><div class="controls bullet"><span class="by">bschmidt1</span><span>|</span><a href="#40301765">prev</a><span>|</span><a href="#40302150">next</a><span>|</span><label class="collapse" for="c-40300819">[-]</label><label class="expand" for="c-40300819">[1 more]</label></div><br/><div class="children"><div class="content">Google&#x27;s Game of Life 3D: Spiral edition</div><br/></div></div><div id="40302150" class="c"><input type="checkbox" id="c-40302150" checked=""/><div class="controls bullet"><span class="by">MPSimmons</span><span>|</span><a href="#40300819">prev</a><span>|</span><a href="#40303115">next</a><span>|</span><label class="collapse" for="c-40302150">[-]</label><label class="expand" for="c-40302150">[1 more]</label></div><br/><div class="children"><div class="content">Not sure why the first thing they point it at wouldn&#x27;t be prions.</div><br/></div></div><div id="40303115" class="c"><input type="checkbox" id="c-40303115" checked=""/><div class="controls bullet"><span class="by">dev1ycan</span><span>|</span><a href="#40302150">prev</a><span>|</span><label class="collapse" for="c-40303115">[-]</label><label class="expand" for="c-40303115">[1 more]</label></div><br/><div class="children"><div class="content">Excited but also it&#x27;s been a fair bit now and I have yet to see something truly remarkable come out of this</div><br/></div></div></div></div></div></div></div></body></html>