<!DOCTYPE html><html lang="en"><head><title>Static News</title><meta charSet="utf-8"/><meta name="description" content="Static delayed Hacker News."/><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"/><meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1d1f21"/><meta name="viewport" content="width=device-width,initial-scale=1.0"/><meta name="application-name" content="Static News"/><meta name="apple-mobile-web-app-title" content="Static News"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="#1d1f21"/><link rel="preload" href="styles.css?v=1722243658788" as="style"/><link rel="stylesheet" href="styles.css?v=1722243658788"/></head><body><div id="container"><div id="inner"><header><a href="/">Static News</a><a href="/about">about</a></header><div id="content"><div><div id="title"><a href="https://github.com/CelVoxes/ceLLama">Show HN: CeLLama – Single cell annotation with local LLMs</a> <span class="domain">(<a href="https://github.com">github.com</a>)</span></div><div class="subtext"><span>celltalk</span> | <span>22 comments</span></div><br/><div><div id="41096481" class="c"><input type="checkbox" id="c-41096481" checked=""/><div class="controls bullet"><span class="by">bob88jg</span><span>|</span><a href="#41095055">next</a><span>|</span><label class="collapse" for="c-41096481">[-]</label><label class="expand" for="c-41096481">[2 more]</label></div><br/><div class="children"><div class="content">Can anyone explain how an LLM is useful here? The clustering is done traditionally right? Then the llm is given the centroids and asked to give a label? Assumption being that the llm corpus already contained some mapping from gene up&#x2F;down regulations to clusters of differentiation?</div><br/><div id="41096715" class="c"><input type="checkbox" id="c-41096715" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41096481">parent</a><span>|</span><a href="#41095055">next</a><span>|</span><label class="collapse" for="c-41096715">[-]</label><label class="expand" for="c-41096715">[1 more]</label></div><br/><div class="children"><div class="content">Yes, it basically automates the cell type annotation process, plus gives a reasoning for the label.</div><br/></div></div></div></div><div id="41095055" class="c"><input type="checkbox" id="c-41095055" checked=""/><div class="controls bullet"><span class="by">ibash</span><span>|</span><a href="#41096481">prev</a><span>|</span><a href="#41094613">next</a><span>|</span><label class="collapse" for="c-41095055">[-]</label><label class="expand" for="c-41095055">[4 more]</label></div><br/><div class="children"><div class="content">How easy is it to check the results of cell annotations for mistakes?<p>Is it easy for a person to do, and this will save them a bunch of time getting a baseline? Or could this lead to a bunch of mislabeled data?</div><br/><div id="41098701" class="c"><input type="checkbox" id="c-41098701" checked=""/><div class="controls bullet"><span class="by">f6v</span><span>|</span><a href="#41095055">parent</a><span>|</span><a href="#41096104">next</a><span>|</span><label class="collapse" for="c-41098701">[-]</label><label class="expand" for="c-41098701">[1 more]</label></div><br/><div class="children"><div class="content">I’ve been doing this for the past three years, it’s very challenging. I think most of these tools do well on very broad cell types like seen on the GitHub page. But the thing is that if you’re working with e.g. immune cell types then you can effortlessly label high-level clusters yourself.<p>The real challenge is identifying fine cell subsets, like different types of CD8 T cells: naïve, central memory, effector memory, Temra, etc. I don’t think it’s a problem that can be solved by a tool though. One issue is that “classical” cell type definitions are based on flow cytometry which uses antibodies to define cell types. These definitions don’t translate that well to scRNA-seq as it’s a completely different protocol. For example, naïve and central memory cells are separated based on CD45 isoforms and this information isn’t available in single cell gene expression with 10x Genomics(most popular protocol).<p>Another issue is that people use ad-hoc cell type definitions. It’s common to come up with a random gene as a cell state marker. Which means the definitions aren’t comparable between studies and a lot of manual curation is required. Which makes some sense because “cell type” is an abstraction. In real world, the cell types and states are much more complex and are often continuous rather than discrete.<p>Taken together, building cell type classifier is a very difficult task that depends much more on the data quality, context(which tissue data comes from), and training labels. You can build a very decent classifier with a regression model if you have good data.</div><br/></div></div><div id="41096104" class="c"><input type="checkbox" id="c-41096104" checked=""/><div class="controls bullet"><span class="by">gww</span><span>|</span><a href="#41095055">parent</a><span>|</span><a href="#41098701">prev</a><span>|</span><a href="#41095396">next</a><span>|</span><label class="collapse" for="c-41096104">[-]</label><label class="expand" for="c-41096104">[1 more]</label></div><br/><div class="children"><div class="content">Users of these kinds of tools should check that their marker genes are associated with the labelled cell types.  There are known markers for many cell types across multiple organisms.</div><br/></div></div><div id="41095396" class="c"><input type="checkbox" id="c-41095396" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41095055">parent</a><span>|</span><a href="#41096104">prev</a><span>|</span><a href="#41094613">next</a><span>|</span><label class="collapse" for="c-41095396">[-]</label><label class="expand" for="c-41095396">[1 more]</label></div><br/><div class="children"><div class="content">It still not 100% accurate but it should be useful for baseline annotations.</div><br/></div></div></div></div><div id="41094613" class="c"><input type="checkbox" id="c-41094613" checked=""/><div class="controls bullet"><span class="by">viraptor</span><span>|</span><a href="#41095055">prev</a><span>|</span><a href="#41094578">next</a><span>|</span><label class="collapse" for="c-41094613">[-]</label><label class="expand" for="c-41094613">[3 more]</label></div><br/><div class="children"><div class="content">I&#x27;m surprised that this is using plain llama3.1 rather than a fine-tune. Have you checked the accuracy of the results on the common benchmarks? Also, given it provides just the answers just based on the up&#x2F;down lists, (or did I miss something?) isn&#x27;t that something that could be extracted into a more efficient lookup with only a 2d grid of weights? (Or 3d if we there are group-of-genes effects)</div><br/><div id="41094827" class="c"><input type="checkbox" id="c-41094827" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41094613">parent</a><span>|</span><a href="#41094578">next</a><span>|</span><label class="collapse" for="c-41094827">[-]</label><label class="expand" for="c-41094827">[2 more]</label></div><br/><div class="children"><div class="content">I don’t have any benchmarking yet, but any help is appreciated. We do have fine-tuned model for anyone interested.</div><br/><div id="41095152" class="c"><input type="checkbox" id="c-41095152" checked=""/><div class="controls bullet"><span class="by">givinguflac</span><span>|</span><a href="#41094613">root</a><span>|</span><a href="#41094827">parent</a><span>|</span><a href="#41094578">next</a><span>|</span><label class="collapse" for="c-41095152">[-]</label><label class="expand" for="c-41095152">[1 more]</label></div><br/><div class="children"><div class="content">This is looks very cool and extremely useful; where can I get hands on the fine-tuned model?</div><br/></div></div></div></div></div></div><div id="41094578" class="c"><input type="checkbox" id="c-41094578" checked=""/><div class="controls bullet"><span class="by">dunomaybe</span><span>|</span><a href="#41094613">prev</a><span>|</span><a href="#41094406">next</a><span>|</span><label class="collapse" for="c-41094578">[-]</label><label class="expand" for="c-41094578">[2 more]</label></div><br/><div class="children"><div class="content">Do you have any benchmark comparisons to e.g. the CellTypist corpus?</div><br/><div id="41094810" class="c"><input type="checkbox" id="c-41094810" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41094578">parent</a><span>|</span><a href="#41094406">next</a><span>|</span><label class="collapse" for="c-41094810">[-]</label><label class="expand" for="c-41094810">[1 more]</label></div><br/><div class="children"><div class="content">No, but the help is appreciated!</div><br/></div></div></div></div><div id="41094406" class="c"><input type="checkbox" id="c-41094406" checked=""/><div class="controls bullet"><span class="by">gww</span><span>|</span><a href="#41094578">prev</a><span>|</span><a href="#41094309">next</a><span>|</span><label class="collapse" for="c-41094406">[-]</label><label class="expand" for="c-41094406">[7 more]</label></div><br/><div class="children"><div class="content">This is really useful thanks for sharing. My students and myself tend to waste a lot of time annotating clusters and have not found a reasonable solution yet. This will be fun to try.</div><br/><div id="41095726" class="c"><input type="checkbox" id="c-41095726" checked=""/><div class="controls bullet"><span class="by">codingfisch</span><span>|</span><a href="#41094406">parent</a><span>|</span><a href="#41094813">next</a><span>|</span><label class="collapse" for="c-41095726">[-]</label><label class="expand" for="c-41095726">[2 more]</label></div><br/><div class="children"><div class="content">I have written a neural network architecture (way smaller than llama) that can be trained to automate this process. Check out the Custom-Data-Tutorial in the repo!<p>GitHub: <a href="https:&#x2F;&#x2F;github.com&#x2F;wwu-mmll&#x2F;gatenet">https:&#x2F;&#x2F;github.com&#x2F;wwu-mmll&#x2F;gatenet</a>
Paper: <a href="https:&#x2F;&#x2F;www.sciencedirect.com&#x2F;science&#x2F;article&#x2F;pii&#x2F;S0010482524009053" rel="nofollow">https:&#x2F;&#x2F;www.sciencedirect.com&#x2F;science&#x2F;article&#x2F;pii&#x2F;S001048252...</a></div><br/><div id="41095886" class="c"><input type="checkbox" id="c-41095886" checked=""/><div class="controls bullet"><span class="by">gww</span><span>|</span><a href="#41094406">root</a><span>|</span><a href="#41095726">parent</a><span>|</span><a href="#41094813">next</a><span>|</span><label class="collapse" for="c-41095886">[-]</label><label class="expand" for="c-41095886">[1 more]</label></div><br/><div class="children"><div class="content">Will check it out. Thanks a lot</div><br/></div></div></div></div><div id="41094813" class="c"><input type="checkbox" id="c-41094813" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41094406">parent</a><span>|</span><a href="#41095726">prev</a><span>|</span><a href="#41094309">next</a><span>|</span><label class="collapse" for="c-41094813">[-]</label><label class="expand" for="c-41094813">[4 more]</label></div><br/><div class="children"><div class="content">Thank you. Hope it is useful!</div><br/><div id="41095108" class="c"><input type="checkbox" id="c-41095108" checked=""/><div class="controls bullet"><span class="by">gww</span><span>|</span><a href="#41094406">root</a><span>|</span><a href="#41094813">parent</a><span>|</span><a href="#41094309">next</a><span>|</span><label class="collapse" for="c-41095108">[-]</label><label class="expand" for="c-41095108">[3 more]</label></div><br/><div class="children"><div class="content">Could this also be adapted for gene set enrichment? For example, if I had a set(s) of genes from an ATAC-seq experiment would it be able to guess their function &#x2F; cell types?</div><br/><div id="41095380" class="c"><input type="checkbox" id="c-41095380" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41094406">root</a><span>|</span><a href="#41095108">parent</a><span>|</span><a href="#41094309">next</a><span>|</span><label class="collapse" for="c-41095380">[-]</label><label class="expand" for="c-41095380">[2 more]</label></div><br/><div class="children"><div class="content">It should be okay if you edit the base prompt properly.</div><br/><div id="41095880" class="c"><input type="checkbox" id="c-41095880" checked=""/><div class="controls bullet"><span class="by">gww</span><span>|</span><a href="#41094406">root</a><span>|</span><a href="#41095380">parent</a><span>|</span><a href="#41094309">next</a><span>|</span><label class="collapse" for="c-41095880">[-]</label><label class="expand" for="c-41095880">[1 more]</label></div><br/><div class="children"><div class="content">Cool thanks</div><br/></div></div></div></div></div></div></div></div></div></div><div id="41094309" class="c"><input type="checkbox" id="c-41094309" checked=""/><div class="controls bullet"><span class="by">j_bum</span><span>|</span><a href="#41094406">prev</a><span>|</span><label class="collapse" for="c-41094309">[-]</label><label class="expand" for="c-41094309">[3 more]</label></div><br/><div class="children"><div class="content">Interesting.<p>Is this deterministic? Any plans for publishing?</div><br/><div id="41094344" class="c"><input type="checkbox" id="c-41094344" checked=""/><div class="controls bullet"><span class="by">celltalk</span><span>|</span><a href="#41094309">parent</a><span>|</span><label class="collapse" for="c-41094344">[-]</label><label class="expand" for="c-41094344">[2 more]</label></div><br/><div class="children"><div class="content">If you set the seed and temperature to 0, it is. I did not have any intentions to publish it, but I might think about a 2-pager Bioinformatics paper if I have time.</div><br/><div id="41096822" class="c"><input type="checkbox" id="c-41096822" checked=""/><div class="controls bullet"><span class="by">j_bum</span><span>|</span><a href="#41094309">root</a><span>|</span><a href="#41094344">parent</a><span>|</span><label class="collapse" for="c-41096822">[-]</label><label class="expand" for="c-41096822">[1 more]</label></div><br/><div class="children"><div class="content">Thanks for sharing!</div><br/></div></div></div></div></div></div></div></div></div></div></div></body></html>